{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "toc": true
   },
   "source": [
    "<h1>Table of Contents<span class=\"tocSkip\"></span></h1>\n",
    "<div class=\"toc\"><ul class=\"toc-item\"><li><span><a href=\"#Goal\" data-toc-modified-id=\"Goal-1\"><span class=\"toc-item-num\">1&nbsp;&nbsp;</span>Goal</a></span></li><li><span><a href=\"#Var\" data-toc-modified-id=\"Var-2\"><span class=\"toc-item-num\">2&nbsp;&nbsp;</span>Var</a></span></li><li><span><a href=\"#Init\" data-toc-modified-id=\"Init-3\"><span class=\"toc-item-num\">3&nbsp;&nbsp;</span>Init</a></span></li><li><span><a href=\"#Load\" data-toc-modified-id=\"Load-4\"><span class=\"toc-item-num\">4&nbsp;&nbsp;</span>Load</a></span></li><li><span><a href=\"#Running-ALE\" data-toc-modified-id=\"Running-ALE-5\"><span class=\"toc-item-num\">5&nbsp;&nbsp;</span>Running ALE</a></span></li><li><span><a href=\"#Summarizing-results\" data-toc-modified-id=\"Summarizing-results-6\"><span class=\"toc-item-num\">6&nbsp;&nbsp;</span>Summarizing results</a></span><ul class=\"toc-item\"><li><span><a href=\"#Writing-combined-ALE-results\" data-toc-modified-id=\"Writing-combined-ALE-results-6.1\"><span class=\"toc-item-num\">6.1&nbsp;&nbsp;</span>Writing combined ALE results</a></span></li></ul></li><li><span><a href=\"#sessionInfo\" data-toc-modified-id=\"sessionInfo-7\"><span class=\"toc-item-num\">7&nbsp;&nbsp;</span>sessionInfo</a></span></li></ul></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Goal\n",
    "\n",
    "* run ALE on simulated training dataset for DeepMAsED \n",
    "  * the ground truth results can be used for selecting cutoffs for ALE"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Var"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T08:36:00.022936Z",
     "start_time": "2019-07-23T08:25:27.489Z"
    }
   },
   "outputs": [],
   "source": [
    "work_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/'\n",
    "\n",
    "# MAGs for genome analysis\n",
    "asmbl_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly/'\n",
    "\n",
    "# bam file dir (mapping of metagenome reads to corresponding MAGs)\n",
    "map_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/'\n",
    "\n",
    "# ALE executable\n",
    "ALE_exe = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE'\n",
    "\n",
    "# params\n",
    "conda_env = 'base'\n",
    "threads = 12"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Init"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T08:20:00.084618Z",
     "start_time": "2019-07-23T08:19:59.566Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading required package: foreach\n",
      "Loading required package: iterators\n",
      "Loading required package: parallel\n"
     ]
    }
   ],
   "source": [
    "library(dplyr)\n",
    "library(tidyr)\n",
    "library(ggplot2)\n",
    "library(data.table)\n",
    "library(doParallel)\n",
    "\n",
    "library(future)\n",
    "library(future.batchtools)\n",
    "library(future.apply)\n",
    "options(future.wait.interval = 2.0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Load"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:38:48.101097Z",
     "start_time": "2019-07-23T07:38:48.013Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 60\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                             </td><td>1                                                                                                                   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta   </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                          </td><td>1                                                                                                                   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta</td></tr>\n",
       "\t<tr><td>megahit                                                                                                             </td><td>10                                                                                                                  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta  </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lll}\n",
       " Assembler & Rep & Fasta\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                        & 1                                                                                                                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/megahit/contigs\\_filtered.fasta   \\\\\n",
       "\t metaspades                                                                                                                     & 1                                                                                                                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/metaspades/contigs\\_filtered.fasta\\\\\n",
       "\t megahit                                                                                                                        & 10                                                                                                                             & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//10/megahit/contigs\\_filtered.fasta  \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                              | 1                                                                                                                    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta    | \n",
       "| metaspades                                                                                                           | 1                                                                                                                    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta | \n",
       "| megahit                                                                                                              | 10                                                                                                                   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta   | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Fasta                                                                                                               \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# adding bam files\n",
    "fasta_files = list.files(asmbl_dir, '*.fasta$', recursive=TRUE, full.names=TRUE)\n",
    "fasta_files = data.frame(Assembler = fasta_files %>% dirname %>% basename,\n",
    "                         Rep = fasta_files %>% dirname %>% dirname %>% basename,\n",
    "                         Fasta = fasta_files)\n",
    "\n",
    "fasta_files %>% nrow %>% print\n",
    "fasta_files %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:39:35.645735Z",
     "start_time": "2019-07-23T07:39:35.563Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 60\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Bam</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                    </td><td>1                                                                                          </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam   </td></tr>\n",
       "\t<tr><td>metaspades                                                                                 </td><td>1                                                                                          </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam</td></tr>\n",
       "\t<tr><td>megahit                                                                                    </td><td>10                                                                                         </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam  </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lll}\n",
       " Assembler & Rep & Bam\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                             & 1                                                                                                   & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/megahit.bam   \\\\\n",
       "\t metaspades                                                                                          & 1                                                                                                   & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/metaspades.bam\\\\\n",
       "\t megahit                                                                                             & 10                                                                                                  & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/10/megahit.bam  \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Bam | \n",
       "|---|---|---|\n",
       "| megahit                                                                                     | 1                                                                                           | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam    | \n",
       "| metaspades                                                                                  | 1                                                                                           | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam | \n",
       "| megahit                                                                                     | 10                                                                                          | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam   | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Bam                                                                                        \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# adding bam files\n",
    "bam_files = list.files(map_dir, '*.bam$', recursive=TRUE, full.names=TRUE)\n",
    "bam_files = data.frame(Assembler = bam_files %>% basename %>% gsub('\\\\.bam$', '', .),\n",
    "                       Rep = bam_files %>% dirname %>% basename,\n",
    "                       Bam = bam_files)\n",
    "\n",
    "bam_files %>% nrow %>% print\n",
    "bam_files %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:40:04.303457Z",
     "start_time": "2019-07-23T07:40:04.226Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 60\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th><th scope=col>Bam</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                             </td><td>1                                                                                                                   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam                            </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                          </td><td>1                                                                                                                   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta</td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam                         </td></tr>\n",
       "\t<tr><td>megahit                                                                                                             </td><td>10                                                                                                                  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam                           </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|llll}\n",
       " Assembler & Rep & Fasta & Bam\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                        & 1                                                                                                                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/megahit.bam                              \\\\\n",
       "\t metaspades                                                                                                                     & 1                                                                                                                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/metaspades/contigs\\_filtered.fasta & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/metaspades.bam                           \\\\\n",
       "\t megahit                                                                                                                        & 10                                                                                                                             & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//10/megahit/contigs\\_filtered.fasta   & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/10/megahit.bam                             \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | Bam | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                              | 1                                                                                                                    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam                             | \n",
       "| metaspades                                                                                                           | 1                                                                                                                    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam                          | \n",
       "| megahit                                                                                                              | 10                                                                                                                   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam                            | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Fasta                                                                                                               \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta  \n",
       "  Bam                                                                                        \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# joining\n",
    "MAGs = fasta_files %>%\n",
    "    inner_join(bam_files, c('Assembler', 'Rep'))\n",
    "MAGs %>% nrow %>% print\n",
    "MAGs %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:42:00.792201Z",
     "start_time": "2019-07-23T07:42:00.725Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th><th scope=col>Bam</th><th scope=col>ALE</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                             </td><td>1                                                                                                                   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam                            </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt                            </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                          </td><td>1                                                                                                                   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta</td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam                         </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt                         </td></tr>\n",
       "\t<tr><td>megahit                                                                                                             </td><td>10                                                                                                                  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam                           </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt                           </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllll}\n",
       " Assembler & Rep & Fasta & Bam & ALE\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                        & 1                                                                                                                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/megahit.bam                               & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/megahit.txt                              \\\\\n",
       "\t metaspades                                                                                                                     & 1                                                                                                                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/metaspades/contigs\\_filtered.fasta & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/metaspades.bam                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/metaspades.txt                           \\\\\n",
       "\t megahit                                                                                                                        & 10                                                                                                                             & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//10/megahit/contigs\\_filtered.fasta   & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/10/megahit.bam                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/megahit.txt                             \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | Bam | ALE | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                              | 1                                                                                                                    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam                             | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt                             | \n",
       "| metaspades                                                                                                           | 1                                                                                                                    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam                          | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt                          | \n",
       "| megahit                                                                                                              | 10                                                                                                                   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam                            | /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt                            | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Fasta                                                                                                               \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta  \n",
       "  Bam                                                                                        \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam  \n",
       "  ALE                                                                                        \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# ALE output file name\n",
    "MAGs = MAGs %>%\n",
    "    mutate(ALE = mapply(function(A, R) file.path(work_dir, R, paste0(A, '.txt')),\n",
    "                        A=Assembler, R=Rep))\n",
    "                        \n",
    "MAGs %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:56:25.645118Z",
     "start_time": "2019-07-23T07:56:25.599Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<dl class=dl-horizontal>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "\t<dt>/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9</dt>\n",
       "\t\t<dd>FALSE</dd>\n",
       "</dl>\n"
      ],
      "text/latex": [
       "\\begin{description*}\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/1] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/10] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/11] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/12] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/13] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/14] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/15] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/16] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/17] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/18] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/19] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/2] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/20] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/21] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/22] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/23] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/24] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/25] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/26] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/27] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/28] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/29] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/3] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/30] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/4] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/5] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/6] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/7] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/8] FALSE\n",
       "\\item[/ebio/abt3\\textbackslash{}\\_projects/databases\\textbackslash{}\\_no-backup/DeepMAsED/train\\textbackslash{}\\_runs/n1000\\textbackslash{}\\_r30/ALE/9] FALSE\n",
       "\\end{description*}\n"
      ],
      "text/markdown": [
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8\n",
       ":   FALSE/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9\n",
       ":   FALSE\n",
       "\n"
      ],
      "text/plain": [
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3 \n",
       "                                                                        FALSE \n",
       "/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8 \n",
       "                                                                        FALSE \n",
       " /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9 \n",
       "                                                                        FALSE "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# making output directories\n",
    "ret = MAGs$ALE %>% \n",
    "    dirname() %>%\n",
    "    unique() %>%\n",
    "    sapply(., function(x) dir.create(x, showWarnings=FALSE))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Running ALE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:56:42.231125Z",
     "start_time": "2019-07-23T07:56:42.196Z"
    }
   },
   "outputs": [],
   "source": [
    "# cluster resources\n",
    "resources = list(h_rt = '00:59:00',\n",
    "                 h_vmem = '8G',\n",
    "                 threads = '1',\n",
    "                 conda.env = 'py3_physeq')     # conda env with batchtools installed\n",
    "plan(batchtools_sge, resources=resources, workers=30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:56:42.579081Z",
     "start_time": "2019-07-23T07:56:42.474Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 60\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "'/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'"
      ],
      "text/latex": [
       "'/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/megahit.bam /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/megahit/contigs\\_filtered.fasta /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/megahit.txt'"
      ],
      "text/markdown": [
       "'/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'"
      ],
      "text/plain": [
       "[1] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "ALE_params = function(x, exe){\n",
    "    cmd = '{exe} {bam} {fasta} {outfile}'\n",
    "    cmd = glue::glue(cmd, exe=exe, bam=x[['Bam']], fasta=x[['Fasta']], outfile=x[['ALE']])\n",
    "    return(cmd)\n",
    "}\n",
    "\n",
    "ALE_cmds = apply(MAGs, 1, ALE_params, exe=ALE_exe) %>%\n",
    "    as.list\n",
    "\n",
    "ALE_cmds %>% length %>% print\n",
    "ALE_cmds[[1]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T07:56:45.024276Z",
     "start_time": "2019-07-23T07:56:44.995Z"
    }
   },
   "outputs": [],
   "source": [
    "bash_job = function(cmd, conda_env, stdout=TRUE, stderr=FALSE){\n",
    "    # cmd : string; commandline job (eg., 'ls -thlc')\n",
    "    # conda_env : string; conda environment name\n",
    "    cmd = sprintf('. ~/.bashrc; conda activate %s; %s', conda_env, cmd)\n",
    "    cmd = sprintf('-c \"%s\"', cmd)\n",
    "    system2('bash', cmd, stdout=stdout, stderr=stderr)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T08:02:23.764694Z",
     "start_time": "2019-07-23T07:56:45.507Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.739318 from 316378 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.908582'</li>\n",
       "\t<li>'There were 330559 total reads, 330559 paired (328316 properly mated), 2105 proper singles, 138 improper reads (128 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 328316 reads, 328316 mapped, 0 unmapped, 328316 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 128 reads, 128 mapped, 0 unmapped, 128 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2105 reads, 2105 mapped, 0 unmapped, 2105 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 330559'</li>\n",
       "\t<li>'Total placed reads: 330559'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 27092'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1293.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -17374082.995081'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.712465 from 306050 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.897022'</li>\n",
       "\t<li>'There were 319723 total reads, 319723 paired (317490 properly mated), 1965 proper singles, 268 improper reads (246 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 317490 reads, 317490 mapped, 0 unmapped, 317490 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1965 reads, 1965 mapped, 0 unmapped, 1965 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 319723'</li>\n",
       "\t<li>'Total placed reads: 319723'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16819'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 736.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16575003.788841'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.763820 from 506326 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.920908'</li>\n",
       "\t<li>'There were 528349 total reads, 528349 paired (525374 properly mated), 2561 proper singles, 414 improper reads (402 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 525374 reads, 525374 mapped, 0 unmapped, 525374 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2559 reads, 2559 mapped, 0 unmapped, 2559 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 528349'</li>\n",
       "\t<li>'Total placed reads: 528349'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 18480'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1187.100000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12015646.535580'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.755345 from 497614 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.923640'</li>\n",
       "\t<li>'There were 519607 total reads, 519607 paired (516280 properly mated), 2649 proper singles, 678 improper reads (662 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 516280 reads, 516280 mapped, 0 unmapped, 516280 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 662 reads, 662 mapped, 0 unmapped, 662 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2643 reads, 2643 mapped, 0 unmapped, 2643 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 519607'</li>\n",
       "\t<li>'Total placed reads: 519607'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15662'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 907.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11418044.195180'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/11/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//11/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.766512 from 368602 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.904772'</li>\n",
       "\t<li>'There were 384711 total reads, 384711 paired (382384 properly mated), 2137 proper singles, 190 improper reads (180 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 382384 reads, 382384 mapped, 0 unmapped, 382384 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 180 reads, 180 mapped, 0 unmapped, 180 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2121 reads, 2121 mapped, 0 unmapped, 2121 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 384711'</li>\n",
       "\t<li>'Total placed reads: 384710'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17799'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 880.502289'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13270694.290886'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/11/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//11/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.771830 from 361546 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.910933'</li>\n",
       "\t<li>'There were 377432 total reads, 377432 paired (375054 properly mated), 2026 proper singles, 352 improper reads (324 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 375054 reads, 375054 mapped, 0 unmapped, 375054 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 324 reads, 324 mapped, 0 unmapped, 324 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2016 reads, 2016 mapped, 0 unmapped, 2016 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 377432'</li>\n",
       "\t<li>'Total placed reads: 377432'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11820'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 694.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12801647.537861'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/12/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//12/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.781898 from 392358 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.905981'</li>\n",
       "\t<li>'There were 409329 total reads, 409329 paired (407022 properly mated), 2087 proper singles, 220 improper reads (192 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 407022 reads, 407022 mapped, 0 unmapped, 407022 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 192 reads, 192 mapped, 0 unmapped, 192 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2075 reads, 2075 mapped, 0 unmapped, 2075 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 409329'</li>\n",
       "\t<li>'Total placed reads: 409329'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 21455'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1214.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16687248.633607'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/12/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//12/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.741528 from 382920 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.895810'</li>\n",
       "\t<li>'There were 399459 total reads, 399459 paired (397146 properly mated), 2007 proper singles, 306 improper reads (280 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 397146 reads, 397146 mapped, 0 unmapped, 397146 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 280 reads, 280 mapped, 0 unmapped, 280 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2005 reads, 2005 mapped, 0 unmapped, 2005 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 399459'</li>\n",
       "\t<li>'Total placed reads: 399459'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16214'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 709.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16026048.041389'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/13/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//13/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.836049 from 225994 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.893290'</li>\n",
       "\t<li>'There were 236258 total reads, 236258 paired (234416 properly mated), 1744 proper singles, 98 improper reads (98 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 234416 reads, 234416 mapped, 0 unmapped, 234416 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 98 reads, 98 mapped, 0 unmapped, 98 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1742 reads, 1742 mapped, 0 unmapped, 1742 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 236258'</li>\n",
       "\t<li>'Total placed reads: 236258'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 20144'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 939.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13441130.446388'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/13/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//13/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.752095 from 218834 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.863300'</li>\n",
       "\t<li>'There were 228674 total reads, 228674 paired (226896 properly mated), 1646 proper singles, 132 improper reads (110 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 226896 reads, 226896 mapped, 0 unmapped, 226896 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 110 reads, 110 mapped, 0 unmapped, 110 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1646 reads, 1646 mapped, 0 unmapped, 1646 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 228674'</li>\n",
       "\t<li>'Total placed reads: 228674'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13893'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 456.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12829391.286242'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/14/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//14/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.757812 from 483128 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.914564'</li>\n",
       "\t<li>'There were 503117 total reads, 503117 paired (501250 properly mated), 1605 proper singles, 262 improper reads (242 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 501250 reads, 501250 mapped, 0 unmapped, 501250 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 242 reads, 242 mapped, 0 unmapped, 242 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 503117'</li>\n",
       "\t<li>'Total placed reads: 503117'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16405'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 833.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13218885.840352'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/14/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//14/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.734157 from 477590 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.908243'</li>\n",
       "\t<li>'There were 497321 total reads, 497321 paired (495422 properly mated), 1527 proper singles, 372 improper reads (360 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 495422 reads, 495422 mapped, 0 unmapped, 495422 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 360 reads, 360 mapped, 0 unmapped, 360 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1527 reads, 1527 mapped, 0 unmapped, 1527 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 497321'</li>\n",
       "\t<li>'Total placed reads: 497320'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9978'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 625.501258'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12874444.057809'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/15/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//15/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.744798 from 573334 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.920983'</li>\n",
       "\t<li>'There were 596785 total reads, 596785 paired (594766 properly mated), 1773 proper singles, 246 improper reads (238 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 594766 reads, 594766 mapped, 0 unmapped, 594765 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 238 reads, 238 mapped, 0 unmapped, 238 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1773 reads, 1773 mapped, 0 unmapped, 1773 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 596785'</li>\n",
       "\t<li>'Total placed reads: 596784'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 18442'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1030.801727'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12399446.201300'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/15/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//15/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.713940 from 565336 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.910377'</li>\n",
       "\t<li>'There were 588560 total reads, 588560 paired (586414 properly mated), 1778 proper singles, 368 improper reads (354 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 586414 reads, 586414 mapped, 0 unmapped, 586413 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1776 reads, 1776 mapped, 0 unmapped, 1776 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 588560'</li>\n",
       "\t<li>'Total placed reads: 588559'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13012'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 728.101237'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11815273.130115'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/16/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//16/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.825277 from 258958 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.906877'</li>\n",
       "\t<li>'There were 270606 total reads, 270606 paired (268648 properly mated), 1852 proper singles, 106 improper reads (92 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 268648 reads, 268648 mapped, 0 unmapped, 268648 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 92 reads, 92 mapped, 0 unmapped, 92 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1852 reads, 1852 mapped, 0 unmapped, 1852 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 270606'</li>\n",
       "\t<li>'Total placed reads: 270606'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 21444'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1181.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14952829.516917'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/16/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//16/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.760708 from 250940 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.880949'</li>\n",
       "\t<li>'There were 261924 total reads, 261924 paired (260222 properly mated), 1560 proper singles, 142 improper reads (124 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 260222 reads, 260222 mapped, 0 unmapped, 260222 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 124 reads, 124 mapped, 0 unmapped, 124 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1560 reads, 1560 mapped, 0 unmapped, 1560 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 261924'</li>\n",
       "\t<li>'Total placed reads: 261924'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14252'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 611.900000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14246895.113395'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/17/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//17/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.741113 from 421342 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.895415'</li>\n",
       "\t<li>'There were 439355 total reads, 439355 paired (437148 properly mated), 1841 proper singles, 366 improper reads (344 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 437148 reads, 437148 mapped, 0 unmapped, 437148 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 344 reads, 344 mapped, 0 unmapped, 344 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1835 reads, 1835 mapped, 0 unmapped, 1835 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 439355'</li>\n",
       "\t<li>'Total placed reads: 439355'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15450'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 978.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15184661.003788'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/17/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//17/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.755768 from 413492 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.903851'</li>\n",
       "\t<li>'There were 431046 total reads, 431046 paired (428998 properly mated), 1648 proper singles, 400 improper reads (382 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 428998 reads, 428998 mapped, 0 unmapped, 428997 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 382 reads, 382 mapped, 0 unmapped, 382 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1648 reads, 1648 mapped, 0 unmapped, 1648 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 431046'</li>\n",
       "\t<li>'Total placed reads: 431045'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12471'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 683.801586'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14668353.704128'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/18/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//18/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.749547 from 757308 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.932785'</li>\n",
       "\t<li>'There were 788345 total reads, 788345 paired (785754 properly mated), 1677 proper singles, 914 improper reads (828 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 785754 reads, 785754 mapped, 0 unmapped, 785754 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 828 reads, 828 mapped, 0 unmapped, 828 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1649 reads, 1649 mapped, 0 unmapped, 1649 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 788345'</li>\n",
       "\t<li>'Total placed reads: 788345'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11841'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1212.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11191721.107405'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/18/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//18/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.721260 from 750586 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.920155'</li>\n",
       "\t<li>'There were 781411 total reads, 781411 paired (778634 properly mated), 2063 proper singles, 714 improper reads (630 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 778634 reads, 778634 mapped, 0 unmapped, 778634 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 630 reads, 630 mapped, 0 unmapped, 630 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2059 reads, 2059 mapped, 0 unmapped, 2059 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 781411'</li>\n",
       "\t<li>'Total placed reads: 781411'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8211'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1198.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -10727524.033928'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/19/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//19/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.710925 from 439506 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.882234'</li>\n",
       "\t<li>'There were 458135 total reads, 458135 paired (455902 properly mated), 1979 proper singles, 254 improper reads (240 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 455902 reads, 455902 mapped, 0 unmapped, 455902 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 240 reads, 240 mapped, 0 unmapped, 240 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1979 reads, 1979 mapped, 0 unmapped, 1979 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 458135'</li>\n",
       "\t<li>'Total placed reads: 458135'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16123'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 998.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12704608.093983'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/19/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//19/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.685501 from 431938 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.869250'</li>\n",
       "\t<li>'There were 450048 total reads, 450048 paired (447962 properly mated), 1666 proper singles, 420 improper reads (404 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 447962 reads, 447962 mapped, 0 unmapped, 447962 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 404 reads, 404 mapped, 0 unmapped, 404 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1662 reads, 1662 mapped, 0 unmapped, 1662 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 450048'</li>\n",
       "\t<li>'Total placed reads: 450048'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11611'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 640.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12183660.987856'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/2/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//2/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.763331 from 852556 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.933910'</li>\n",
       "\t<li>'There were 887324 total reads, 887324 paired (884688 properly mated), 1796 proper singles, 840 improper reads (764 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 884688 reads, 884688 mapped, 0 unmapped, 884688 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 764 reads, 764 mapped, 0 unmapped, 764 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1790 reads, 1790 mapped, 0 unmapped, 1790 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 887324'</li>\n",
       "\t<li>'Total placed reads: 887324'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14078'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1238.900000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -17187997.456175'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/2/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//2/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.757652 from 841888 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.932336'</li>\n",
       "\t<li>'There were 876686 total reads, 876686 paired (873584 properly mated), 2236 proper singles, 866 improper reads (796 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 873584 reads, 873584 mapped, 0 unmapped, 873584 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 796 reads, 796 mapped, 0 unmapped, 796 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2224 reads, 2224 mapped, 0 unmapped, 2224 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 876686'</li>\n",
       "\t<li>'Total placed reads: 876686'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9417'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1354.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16654318.964110'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/20/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//20/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.693648 from 546398 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.887225'</li>\n",
       "\t<li>'There were 568494 total reads, 568494 paired (566614 properly mated), 1594 proper singles, 286 improper reads (270 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 566614 reads, 566614 mapped, 0 unmapped, 566614 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 270 reads, 270 mapped, 0 unmapped, 270 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1578 reads, 1578 mapped, 0 unmapped, 1578 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 568494'</li>\n",
       "\t<li>'Total placed reads: 568494'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13121'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 765.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16995916.356186'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/20/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//20/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.703542 from 539658 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.894556'</li>\n",
       "\t<li>'There were 561765 total reads, 561765 paired (559688 properly mated), 1663 proper singles, 414 improper reads (400 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 559688 reads, 559688 mapped, 0 unmapped, 559688 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 400 reads, 400 mapped, 0 unmapped, 400 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1663 reads, 1663 mapped, 0 unmapped, 1663 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 561765'</li>\n",
       "\t<li>'Total placed reads: 561765'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8043'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 857.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16611079.347268'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/21/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//21/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.770890 from 698024 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.935357'</li>\n",
       "\t<li>'There were 725969 total reads, 725969 paired (724254 properly mated), 1479 proper singles, 236 improper reads (214 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 724254 reads, 724254 mapped, 0 unmapped, 724253 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 214 reads, 214 mapped, 0 unmapped, 214 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1475 reads, 1475 mapped, 0 unmapped, 1475 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 725969'</li>\n",
       "\t<li>'Total placed reads: 725968'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8492'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 661.400911'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15127024.327711'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/21/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//21/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.749190 from 693936 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.925717'</li>\n",
       "\t<li>'There were 721773 total reads, 721773 paired (719942 properly mated), 1481 proper singles, 350 improper reads (336 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 719942 reads, 719942 mapped, 0 unmapped, 719942 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1473 reads, 1473 mapped, 0 unmapped, 1473 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 721773'</li>\n",
       "\t<li>'Total placed reads: 721773'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 5549'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 570.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14809653.791573'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/22/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//22/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.741647 from 369982 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.879979'</li>\n",
       "\t<li>'There were 386029 total reads, 386029 paired (383718 properly mated), 2143 proper singles, 168 improper reads (142 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 383718 reads, 383718 mapped, 0 unmapped, 383718 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 142 reads, 142 mapped, 0 unmapped, 142 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2143 reads, 2143 mapped, 0 unmapped, 2143 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 386029'</li>\n",
       "\t<li>'Total placed reads: 386029'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17603'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 839.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12932315.161128'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/22/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//22/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.720616 from 364072 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.877073'</li>\n",
       "\t<li>'There were 379994 total reads, 379994 paired (377602 properly mated), 2056 proper singles, 336 improper reads (308 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 377602 reads, 377602 mapped, 0 unmapped, 377602 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 15 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 308 reads, 308 mapped, 0 unmapped, 308 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2056 reads, 2056 mapped, 0 unmapped, 2056 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 379994'</li>\n",
       "\t<li>'Total placed reads: 379993'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11788'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 577.401520'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12470140.178258'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/23/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//23/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.677136 from 516186 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.887594'</li>\n",
       "\t<li>'There were 537539 total reads, 537539 paired (535338 properly mated), 1915 proper singles, 286 improper reads (246 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 535338 reads, 535338 mapped, 0 unmapped, 535337 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1897 reads, 1897 mapped, 0 unmapped, 1897 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 537539'</li>\n",
       "\t<li>'Total placed reads: 537538'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12951'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 844.901572'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13346764.513687'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/23/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//23/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.670893 from 510296 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.883901'</li>\n",
       "\t<li>'There were 531447 total reads, 531447 paired (529170 properly mated), 1821 proper singles, 456 improper reads (428 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 529170 reads, 529170 mapped, 0 unmapped, 529170 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 428 reads, 428 mapped, 0 unmapped, 428 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1807 reads, 1807 mapped, 0 unmapped, 1807 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 531447'</li>\n",
       "\t<li>'Total placed reads: 531447'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 10525'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 683.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12996454.555808'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/24/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//24/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.677348 from 323618 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.860010'</li>\n",
       "\t<li>'There were 337511 total reads, 337511 paired (335530 properly mated), 1627 proper singles, 354 improper reads (326 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 335530 reads, 335530 mapped, 0 unmapped, 335530 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1627 reads, 1627 mapped, 0 unmapped, 1627 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 337511'</li>\n",
       "\t<li>'Total placed reads: 337511'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11254'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 842.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -9919433.184678'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/24/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//24/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.714823 from 316610 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.875752'</li>\n",
       "\t<li>'There were 330503 total reads, 330503 paired (328358 properly mated), 1739 proper singles, 406 improper reads (370 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 328358 reads, 328358 mapped, 0 unmapped, 328358 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 370 reads, 370 mapped, 0 unmapped, 370 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1739 reads, 1739 mapped, 0 unmapped, 1739 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 330503'</li>\n",
       "\t<li>'Total placed reads: 330503'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 10498'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 813.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -9515257.556246'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/25/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//25/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.700994 from 572804 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.896120'</li>\n",
       "\t<li>'There were 596261 total reads, 596261 paired (594004 properly mated), 1559 proper singles, 698 improper reads (676 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 594004 reads, 594004 mapped, 0 unmapped, 594004 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 676 reads, 676 mapped, 0 unmapped, 676 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1545 reads, 1545 mapped, 0 unmapped, 1545 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 596261'</li>\n",
       "\t<li>'Total placed reads: 596261'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12886'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1044.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14853488.488456'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/25/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//25/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.703648 from 565516 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.903196'</li>\n",
       "\t<li>'There were 589534 total reads, 589534 paired (586468 properly mated), 2148 proper singles, 918 improper reads (872 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 586468 reads, 586468 mapped, 0 unmapped, 586468 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 872 reads, 872 mapped, 0 unmapped, 872 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2142 reads, 2142 mapped, 0 unmapped, 2142 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 589534'</li>\n",
       "\t<li>'Total placed reads: 589534'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 10919'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1113.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14503625.790964'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/26/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//26/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.848518 from 762942 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.981232'</li>\n",
       "\t<li>'There were 794498 total reads, 794498 paired (791910 properly mated), 1582 proper singles, 1006 improper reads (860 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 791910 reads, 791910 mapped, 0 unmapped, 791903 placed, 7 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 88 reads, 88 mapped, 0 unmapped, 88 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1524 reads, 1524 mapped, 0 unmapped, 1524 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 794498'</li>\n",
       "\t<li>'Total placed reads: 794491'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 7'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 7176'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1039.751222'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14384733.330159'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/26/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//26/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.824811 from 758038 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.970790'</li>\n",
       "\t<li>'There were 790175 total reads, 790175 paired (786864 properly mated), 2387 proper singles, 924 improper reads (744 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 786864 reads, 786864 mapped, 0 unmapped, 786857 placed, 7 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 96 reads, 96 mapped, 0 unmapped, 96 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 744 reads, 744 mapped, 0 unmapped, 744 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2343 reads, 2343 mapped, 0 unmapped, 2343 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 790175'</li>\n",
       "\t<li>'Total placed reads: 790168'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 7'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 5558'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1245.682832'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14218317.825355'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/27/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//27/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.747729 from 484106 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.900189'</li>\n",
       "\t<li>'There were 504159 total reads, 504159 paired (502188 properly mated), 1653 proper singles, 318 improper reads (306 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 502188 reads, 502188 mapped, 0 unmapped, 502188 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 306 reads, 306 mapped, 0 unmapped, 306 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1651 reads, 1651 mapped, 0 unmapped, 1651 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 504159'</li>\n",
       "\t<li>'Total placed reads: 504159'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17255'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1064.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13516775.165159'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/27/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//27/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.716492 from 478160 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.887961'</li>\n",
       "\t<li>'There were 497996 total reads, 497996 paired (495980 properly mated), 1548 proper singles, 468 improper reads (448 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 495980 reads, 495980 mapped, 0 unmapped, 495980 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1548 reads, 1548 mapped, 0 unmapped, 1548 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 497996'</li>\n",
       "\t<li>'Total placed reads: 497996'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12965'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 694.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13118587.682537'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/28/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//28/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.771064 from 271272 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.901245'</li>\n",
       "\t<li>'There were 283284 total reads, 283284 paired (281252 properly mated), 1732 proper singles, 300 improper reads (300 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 281252 reads, 281252 mapped, 0 unmapped, 281252 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 300 reads, 300 mapped, 0 unmapped, 300 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1732 reads, 1732 mapped, 0 unmapped, 1732 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 283284'</li>\n",
       "\t<li>'Total placed reads: 283284'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12386'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 784.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11165800.783435'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/28/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//28/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.749976 from 266078 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.891963'</li>\n",
       "\t<li>'There were 277610 total reads, 277610 paired (275828 properly mated), 1510 proper singles, 272 improper reads (264 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 275828 reads, 275828 mapped, 0 unmapped, 275828 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 264 reads, 264 mapped, 0 unmapped, 264 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1510 reads, 1510 mapped, 0 unmapped, 1510 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 277610'</li>\n",
       "\t<li>'Total placed reads: 277610'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9038'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 552.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -10806835.625985'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/29/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//29/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.730651 from 533398 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.910921'</li>\n",
       "\t<li>'There were 555957 total reads, 555957 paired (553314 properly mated), 1853 proper singles, 790 improper reads (718 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 553314 reads, 553314 mapped, 0 unmapped, 553311 placed, 3 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 718 reads, 718 mapped, 0 unmapped, 718 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1825 reads, 1825 mapped, 0 unmapped, 1825 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 555957'</li>\n",
       "\t<li>'Total placed reads: 555954'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 3'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15709'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1088.605874'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11178993.697992'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/29/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//29/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.714380 from 527428 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.904824'</li>\n",
       "\t<li>'There were 549525 total reads, 549525 paired (547114 properly mated), 1749 proper singles, 662 improper reads (604 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 547114 reads, 547114 mapped, 0 unmapped, 547111 placed, 3 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1705 reads, 1705 mapped, 0 unmapped, 1705 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 549525'</li>\n",
       "\t<li>'Total placed reads: 549522'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 3'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11532'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 873.104767'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -10770544.377923'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/3/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//3/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.748898 from 235904 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.865444'</li>\n",
       "\t<li>'There were 246262 total reads, 246262 paired (244704 properly mated), 1486 proper singles, 72 improper reads (66 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 244704 reads, 244704 mapped, 0 unmapped, 244704 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 66 reads, 66 mapped, 0 unmapped, 66 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1486 reads, 1486 mapped, 0 unmapped, 1486 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 246262'</li>\n",
       "\t<li>'Total placed reads: 246262'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13408'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 750.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12545384.151465'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/3/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//3/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.732209 from 229306 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.856694'</li>\n",
       "\t<li>'There were 239119 total reads, 239119 paired (237844 properly mated), 1169 proper singles, 106 improper reads (106 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 237844 reads, 237844 mapped, 0 unmapped, 237844 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 106 reads, 106 mapped, 0 unmapped, 106 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1167 reads, 1167 mapped, 0 unmapped, 1167 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 239119'</li>\n",
       "\t<li>'Total placed reads: 239119'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9975'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 368.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12030126.110063'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/30/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//30/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.773617 from 607112 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.937958'</li>\n",
       "\t<li>'There were 631675 total reads, 631675 paired (629870 properly mated), 1685 proper singles, 120 improper reads (100 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 629870 reads, 629870 mapped, 0 unmapped, 629870 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 100 reads, 100 mapped, 0 unmapped, 100 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1683 reads, 1683 mapped, 0 unmapped, 1683 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 631675'</li>\n",
       "\t<li>'Total placed reads: 631675'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14930'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 830.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -26596142.638070'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/30/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//30/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.743582 from 600208 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.925605'</li>\n",
       "\t<li>'There were 624155 total reads, 624155 paired (622624 properly mated), 1393 proper singles, 138 improper reads (126 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 622624 reads, 622624 mapped, 0 unmapped, 622624 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 126 reads, 126 mapped, 0 unmapped, 126 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1391 reads, 1391 mapped, 0 unmapped, 1391 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 624155'</li>\n",
       "\t<li>'Total placed reads: 624155'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8854'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 510.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -25965655.555041'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/4/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//4/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.713039 from 379564 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.879604'</li>\n",
       "\t<li>'There were 396198 total reads, 396198 paired (393586 properly mated), 2148 proper singles, 464 improper reads (442 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 393586 reads, 393586 mapped, 0 unmapped, 393586 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 442 reads, 442 mapped, 0 unmapped, 442 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2144 reads, 2144 mapped, 0 unmapped, 2144 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 396198'</li>\n",
       "\t<li>'Total placed reads: 396198'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17254'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1255.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13656553.006202'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/4/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//4/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.707399 from 373622 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.880365'</li>\n",
       "\t<li>'There were 389836 total reads, 389836 paired (387414 properly mated), 1922 proper singles, 500 improper reads (484 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 387414 reads, 387414 mapped, 0 unmapped, 387414 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 484 reads, 484 mapped, 0 unmapped, 484 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1902 reads, 1902 mapped, 0 unmapped, 1902 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 389836'</li>\n",
       "\t<li>'Total placed reads: 389836'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13180'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 872.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13194840.370122'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/5/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//5/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.696256 from 250718 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.875842'</li>\n",
       "\t<li>'There were 261393 total reads, 261393 paired (260022 properly mated), 1297 proper singles, 74 improper reads (62 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 260022 reads, 260022 mapped, 0 unmapped, 260022 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1293 reads, 1293 mapped, 0 unmapped, 1293 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 261393'</li>\n",
       "\t<li>'Total placed reads: 261393'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9797'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 616.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -9418117.983413'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/5/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//5/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.662871 from 246096 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.866825'</li>\n",
       "\t<li>'There were 256213 total reads, 256213 paired (255122 properly mated), 963 proper singles, 128 improper reads (112 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 255122 reads, 255122 mapped, 0 unmapped, 255122 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 112 reads, 112 mapped, 0 unmapped, 112 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 963 reads, 963 mapped, 0 unmapped, 963 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 256213'</li>\n",
       "\t<li>'Total placed reads: 256213'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 6760'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 407.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -9083426.562538'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/6/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//6/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.767056 from 712204 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.927949'</li>\n",
       "\t<li>'There were 741088 total reads, 741088 paired (739100 properly mated), 1598 proper singles, 390 improper reads (332 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 739100 reads, 739100 mapped, 0 unmapped, 739100 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 332 reads, 332 mapped, 0 unmapped, 332 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1576 reads, 1576 mapped, 0 unmapped, 1576 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 741088'</li>\n",
       "\t<li>'Total placed reads: 741088'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11967'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 798.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16130340.567633'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/6/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//6/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.755147 from 707362 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.925815'</li>\n",
       "\t<li>'There were 736000 total reads, 736000 paired (734046 properly mated), 1576 proper singles, 378 improper reads (364 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 734046 reads, 734046 mapped, 0 unmapped, 734046 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 364 reads, 364 mapped, 0 unmapped, 364 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1572 reads, 1572 mapped, 0 unmapped, 1572 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 736000'</li>\n",
       "\t<li>'Total placed reads: 736000'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8499'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 662.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15828310.751221'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/7/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//7/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.764770 from 572266 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.929275'</li>\n",
       "\t<li>'There were 595605 total reads, 595605 paired (593784 properly mated), 1605 proper singles, 216 improper reads (206 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 593784 reads, 593784 mapped, 0 unmapped, 593784 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 206 reads, 206 mapped, 0 unmapped, 206 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 595605'</li>\n",
       "\t<li>'Total placed reads: 595605'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13246'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 866.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13895857.636032'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/7/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//7/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.758549 from 567304 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.929236'</li>\n",
       "\t<li>'There were 590509 total reads, 590509 paired (588602 properly mated), 1555 proper singles, 352 improper reads (336 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 588602 reads, 588602 mapped, 0 unmapped, 588602 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1549 reads, 1549 mapped, 0 unmapped, 1549 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 590509'</li>\n",
       "\t<li>'Total placed reads: 590509'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9107'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 595.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13582663.661134'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/8/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//8/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.789475 from 407582 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.908351'</li>\n",
       "\t<li>'There were 425560 total reads, 425560 paired (422904 properly mated), 2166 proper singles, 490 improper reads (456 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 422904 reads, 422904 mapped, 0 unmapped, 422904 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 456 reads, 456 mapped, 0 unmapped, 456 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2164 reads, 2164 mapped, 0 unmapped, 2164 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 425560'</li>\n",
       "\t<li>'Total placed reads: 425560'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15790'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1097.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13191614.984628'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/8/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//8/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.792128 from 398812 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.916498'</li>\n",
       "\t<li>'There were 416603 total reads, 416603 paired (413802 properly mated), 2269 proper singles, 532 improper reads (494 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 413802 reads, 413802 mapped, 0 unmapped, 413802 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 494 reads, 494 mapped, 0 unmapped, 494 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2267 reads, 2267 mapped, 0 unmapped, 2267 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 416603'</li>\n",
       "\t<li>'Total placed reads: 416603'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11518'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 926.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12711677.097005'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/9/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//9/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.705491 from 520154 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.900413'</li>\n",
       "\t<li>'There were 541958 total reads, 541958 paired (539498 properly mated), 1940 proper singles, 520 improper reads (470 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 539498 reads, 539498 mapped, 0 unmapped, 539497 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 470 reads, 470 mapped, 0 unmapped, 470 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1930 reads, 1930 mapped, 0 unmapped, 1930 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 541958'</li>\n",
       "\t<li>'Total placed reads: 541957'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15487'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 994.001834'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13988489.574853'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/9/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//9/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.722684 from 511928 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.907540'</li>\n",
       "\t<li>'There were 533364 total reads, 533364 paired (531020 properly mated), 1812 proper singles, 532 improper reads (506 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 531020 reads, 531020 mapped, 0 unmapped, 531020 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 506 reads, 506 mapped, 0 unmapped, 506 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1806 reads, 1806 mapped, 0 unmapped, 1806 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 533364'</li>\n",
       "\t<li>'Total placed reads: 533364'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13274'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 848.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13635354.189521'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate}\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.739318 from 316378 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.908582'\n",
       "\\item 'There were 330559 total reads, 330559 paired (328316 properly mated), 2105 proper singles, 138 improper reads (128 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 328316 reads, 328316 mapped, 0 unmapped, 328316 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 128 reads, 128 mapped, 0 unmapped, 128 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2105 reads, 2105 mapped, 0 unmapped, 2105 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 330559'\n",
       "\\item 'Total placed reads: 330559'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 27092'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1293.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -17374082.995081'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/1/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//1/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.712465 from 306050 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.897022'\n",
       "\\item 'There were 319723 total reads, 319723 paired (317490 properly mated), 1965 proper singles, 268 improper reads (246 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 317490 reads, 317490 mapped, 0 unmapped, 317490 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1965 reads, 1965 mapped, 0 unmapped, 1965 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 319723'\n",
       "\\item 'Total placed reads: 319723'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16819'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 736.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16575003.788841'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/10/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//10/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.763820 from 506326 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.920908'\n",
       "\\item 'There were 528349 total reads, 528349 paired (525374 properly mated), 2561 proper singles, 414 improper reads (402 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 525374 reads, 525374 mapped, 0 unmapped, 525374 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2559 reads, 2559 mapped, 0 unmapped, 2559 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 528349'\n",
       "\\item 'Total placed reads: 528349'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 18480'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1187.100000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12015646.535580'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/10/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//10/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.755345 from 497614 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.923640'\n",
       "\\item 'There were 519607 total reads, 519607 paired (516280 properly mated), 2649 proper singles, 678 improper reads (662 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 516280 reads, 516280 mapped, 0 unmapped, 516280 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 662 reads, 662 mapped, 0 unmapped, 662 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2643 reads, 2643 mapped, 0 unmapped, 2643 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 519607'\n",
       "\\item 'Total placed reads: 519607'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15662'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 907.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11418044.195180'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/11/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//11/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.766512 from 368602 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.904772'\n",
       "\\item 'There were 384711 total reads, 384711 paired (382384 properly mated), 2137 proper singles, 190 improper reads (180 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 382384 reads, 382384 mapped, 0 unmapped, 382384 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 180 reads, 180 mapped, 0 unmapped, 180 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2121 reads, 2121 mapped, 0 unmapped, 2121 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 384711'\n",
       "\\item 'Total placed reads: 384710'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17799'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 880.502289'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13270694.290886'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/11/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//11/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.771830 from 361546 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.910933'\n",
       "\\item 'There were 377432 total reads, 377432 paired (375054 properly mated), 2026 proper singles, 352 improper reads (324 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 375054 reads, 375054 mapped, 0 unmapped, 375054 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 324 reads, 324 mapped, 0 unmapped, 324 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2016 reads, 2016 mapped, 0 unmapped, 2016 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 377432'\n",
       "\\item 'Total placed reads: 377432'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11820'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 694.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12801647.537861'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/12/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//12/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/12/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.781898 from 392358 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.905981'\n",
       "\\item 'There were 409329 total reads, 409329 paired (407022 properly mated), 2087 proper singles, 220 improper reads (192 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/12/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 407022 reads, 407022 mapped, 0 unmapped, 407022 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 192 reads, 192 mapped, 0 unmapped, 192 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2075 reads, 2075 mapped, 0 unmapped, 2075 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 409329'\n",
       "\\item 'Total placed reads: 409329'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 21455'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1214.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16687248.633607'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/12/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/12/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//12/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/12/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.741528 from 382920 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.895810'\n",
       "\\item 'There were 399459 total reads, 399459 paired (397146 properly mated), 2007 proper singles, 306 improper reads (280 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/12/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 397146 reads, 397146 mapped, 0 unmapped, 397146 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 280 reads, 280 mapped, 0 unmapped, 280 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2005 reads, 2005 mapped, 0 unmapped, 2005 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 399459'\n",
       "\\item 'Total placed reads: 399459'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16214'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 709.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16026048.041389'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/12/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/13/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//13/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/13/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.836049 from 225994 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.893290'\n",
       "\\item 'There were 236258 total reads, 236258 paired (234416 properly mated), 1744 proper singles, 98 improper reads (98 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/13/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 234416 reads, 234416 mapped, 0 unmapped, 234416 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 98 reads, 98 mapped, 0 unmapped, 98 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1742 reads, 1742 mapped, 0 unmapped, 1742 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 236258'\n",
       "\\item 'Total placed reads: 236258'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 20144'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 939.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13441130.446388'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/13/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/13/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//13/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/13/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.752095 from 218834 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.863300'\n",
       "\\item 'There were 228674 total reads, 228674 paired (226896 properly mated), 1646 proper singles, 132 improper reads (110 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/13/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 226896 reads, 226896 mapped, 0 unmapped, 226896 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 110 reads, 110 mapped, 0 unmapped, 110 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1646 reads, 1646 mapped, 0 unmapped, 1646 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 228674'\n",
       "\\item 'Total placed reads: 228674'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13893'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 456.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12829391.286242'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/13/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/14/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//14/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/14/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.757812 from 483128 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.914564'\n",
       "\\item 'There were 503117 total reads, 503117 paired (501250 properly mated), 1605 proper singles, 262 improper reads (242 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/14/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 501250 reads, 501250 mapped, 0 unmapped, 501250 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 242 reads, 242 mapped, 0 unmapped, 242 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 503117'\n",
       "\\item 'Total placed reads: 503117'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16405'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 833.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13218885.840352'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/14/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/14/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//14/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/14/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.734157 from 477590 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.908243'\n",
       "\\item 'There were 497321 total reads, 497321 paired (495422 properly mated), 1527 proper singles, 372 improper reads (360 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/14/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 495422 reads, 495422 mapped, 0 unmapped, 495422 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 360 reads, 360 mapped, 0 unmapped, 360 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1527 reads, 1527 mapped, 0 unmapped, 1527 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 497321'\n",
       "\\item 'Total placed reads: 497320'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9978'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 625.501258'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12874444.057809'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/14/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/15/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//15/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/15/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.744798 from 573334 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.920983'\n",
       "\\item 'There were 596785 total reads, 596785 paired (594766 properly mated), 1773 proper singles, 246 improper reads (238 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/15/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 594766 reads, 594766 mapped, 0 unmapped, 594765 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 238 reads, 238 mapped, 0 unmapped, 238 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1773 reads, 1773 mapped, 0 unmapped, 1773 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 596785'\n",
       "\\item 'Total placed reads: 596784'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 18442'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1030.801727'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12399446.201300'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/15/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/15/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//15/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/15/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.713940 from 565336 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.910377'\n",
       "\\item 'There were 588560 total reads, 588560 paired (586414 properly mated), 1778 proper singles, 368 improper reads (354 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/15/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 586414 reads, 586414 mapped, 0 unmapped, 586413 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1776 reads, 1776 mapped, 0 unmapped, 1776 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 588560'\n",
       "\\item 'Total placed reads: 588559'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13012'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 728.101237'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11815273.130115'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/15/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/16/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//16/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/16/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.825277 from 258958 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.906877'\n",
       "\\item 'There were 270606 total reads, 270606 paired (268648 properly mated), 1852 proper singles, 106 improper reads (92 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/16/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 268648 reads, 268648 mapped, 0 unmapped, 268648 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 92 reads, 92 mapped, 0 unmapped, 92 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1852 reads, 1852 mapped, 0 unmapped, 1852 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 270606'\n",
       "\\item 'Total placed reads: 270606'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 21444'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1181.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14952829.516917'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/16/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/16/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//16/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/16/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.760708 from 250940 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.880949'\n",
       "\\item 'There were 261924 total reads, 261924 paired (260222 properly mated), 1560 proper singles, 142 improper reads (124 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/16/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 260222 reads, 260222 mapped, 0 unmapped, 260222 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 124 reads, 124 mapped, 0 unmapped, 124 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1560 reads, 1560 mapped, 0 unmapped, 1560 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 261924'\n",
       "\\item 'Total placed reads: 261924'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14252'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 611.900000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14246895.113395'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/16/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/17/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//17/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/17/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.741113 from 421342 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.895415'\n",
       "\\item 'There were 439355 total reads, 439355 paired (437148 properly mated), 1841 proper singles, 366 improper reads (344 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/17/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 437148 reads, 437148 mapped, 0 unmapped, 437148 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 344 reads, 344 mapped, 0 unmapped, 344 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1835 reads, 1835 mapped, 0 unmapped, 1835 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 439355'\n",
       "\\item 'Total placed reads: 439355'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15450'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 978.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15184661.003788'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/17/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/17/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//17/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/17/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.755768 from 413492 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.903851'\n",
       "\\item 'There were 431046 total reads, 431046 paired (428998 properly mated), 1648 proper singles, 400 improper reads (382 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/17/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 428998 reads, 428998 mapped, 0 unmapped, 428997 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 382 reads, 382 mapped, 0 unmapped, 382 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1648 reads, 1648 mapped, 0 unmapped, 1648 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 431046'\n",
       "\\item 'Total placed reads: 431045'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12471'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 683.801586'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14668353.704128'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/17/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/18/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//18/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/18/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.749547 from 757308 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.932785'\n",
       "\\item 'There were 788345 total reads, 788345 paired (785754 properly mated), 1677 proper singles, 914 improper reads (828 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/18/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 785754 reads, 785754 mapped, 0 unmapped, 785754 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 828 reads, 828 mapped, 0 unmapped, 828 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1649 reads, 1649 mapped, 0 unmapped, 1649 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 788345'\n",
       "\\item 'Total placed reads: 788345'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11841'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1212.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11191721.107405'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/18/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/18/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//18/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/18/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.721260 from 750586 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.920155'\n",
       "\\item 'There were 781411 total reads, 781411 paired (778634 properly mated), 2063 proper singles, 714 improper reads (630 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/18/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 778634 reads, 778634 mapped, 0 unmapped, 778634 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 630 reads, 630 mapped, 0 unmapped, 630 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2059 reads, 2059 mapped, 0 unmapped, 2059 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 781411'\n",
       "\\item 'Total placed reads: 781411'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8211'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1198.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -10727524.033928'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/18/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/19/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//19/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/19/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.710925 from 439506 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.882234'\n",
       "\\item 'There were 458135 total reads, 458135 paired (455902 properly mated), 1979 proper singles, 254 improper reads (240 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/19/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 455902 reads, 455902 mapped, 0 unmapped, 455902 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 240 reads, 240 mapped, 0 unmapped, 240 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1979 reads, 1979 mapped, 0 unmapped, 1979 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 458135'\n",
       "\\item 'Total placed reads: 458135'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16123'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 998.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12704608.093983'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/19/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/19/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//19/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/19/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.685501 from 431938 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.869250'\n",
       "\\item 'There were 450048 total reads, 450048 paired (447962 properly mated), 1666 proper singles, 420 improper reads (404 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/19/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 447962 reads, 447962 mapped, 0 unmapped, 447962 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 404 reads, 404 mapped, 0 unmapped, 404 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1662 reads, 1662 mapped, 0 unmapped, 1662 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 450048'\n",
       "\\item 'Total placed reads: 450048'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11611'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 640.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12183660.987856'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/19/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/2/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//2/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/2/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.763331 from 852556 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.933910'\n",
       "\\item 'There were 887324 total reads, 887324 paired (884688 properly mated), 1796 proper singles, 840 improper reads (764 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/2/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 884688 reads, 884688 mapped, 0 unmapped, 884688 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 764 reads, 764 mapped, 0 unmapped, 764 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1790 reads, 1790 mapped, 0 unmapped, 1790 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 887324'\n",
       "\\item 'Total placed reads: 887324'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14078'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1238.900000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -17187997.456175'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/2/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/2/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//2/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/2/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.757652 from 841888 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.932336'\n",
       "\\item 'There were 876686 total reads, 876686 paired (873584 properly mated), 2236 proper singles, 866 improper reads (796 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/2/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 873584 reads, 873584 mapped, 0 unmapped, 873584 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 796 reads, 796 mapped, 0 unmapped, 796 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2224 reads, 2224 mapped, 0 unmapped, 2224 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 876686'\n",
       "\\item 'Total placed reads: 876686'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9417'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1354.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16654318.964110'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/2/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/20/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//20/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/20/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.693648 from 546398 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.887225'\n",
       "\\item 'There were 568494 total reads, 568494 paired (566614 properly mated), 1594 proper singles, 286 improper reads (270 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/20/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 566614 reads, 566614 mapped, 0 unmapped, 566614 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 270 reads, 270 mapped, 0 unmapped, 270 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1578 reads, 1578 mapped, 0 unmapped, 1578 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 568494'\n",
       "\\item 'Total placed reads: 568494'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13121'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 765.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16995916.356186'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/20/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/20/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//20/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/20/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.703542 from 539658 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.894556'\n",
       "\\item 'There were 561765 total reads, 561765 paired (559688 properly mated), 1663 proper singles, 414 improper reads (400 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/20/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 559688 reads, 559688 mapped, 0 unmapped, 559688 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 400 reads, 400 mapped, 0 unmapped, 400 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1663 reads, 1663 mapped, 0 unmapped, 1663 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 561765'\n",
       "\\item 'Total placed reads: 561765'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8043'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 857.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16611079.347268'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/20/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/21/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//21/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/21/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.770890 from 698024 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.935357'\n",
       "\\item 'There were 725969 total reads, 725969 paired (724254 properly mated), 1479 proper singles, 236 improper reads (214 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/21/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 724254 reads, 724254 mapped, 0 unmapped, 724253 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 214 reads, 214 mapped, 0 unmapped, 214 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1475 reads, 1475 mapped, 0 unmapped, 1475 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 725969'\n",
       "\\item 'Total placed reads: 725968'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8492'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 661.400911'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15127024.327711'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/21/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/21/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//21/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/21/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.749190 from 693936 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.925717'\n",
       "\\item 'There were 721773 total reads, 721773 paired (719942 properly mated), 1481 proper singles, 350 improper reads (336 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/21/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 719942 reads, 719942 mapped, 0 unmapped, 719942 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1473 reads, 1473 mapped, 0 unmapped, 1473 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 721773'\n",
       "\\item 'Total placed reads: 721773'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 5549'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 570.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14809653.791573'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/21/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/22/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//22/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/22/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.741647 from 369982 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.879979'\n",
       "\\item 'There were 386029 total reads, 386029 paired (383718 properly mated), 2143 proper singles, 168 improper reads (142 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/22/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 383718 reads, 383718 mapped, 0 unmapped, 383718 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 142 reads, 142 mapped, 0 unmapped, 142 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2143 reads, 2143 mapped, 0 unmapped, 2143 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 386029'\n",
       "\\item 'Total placed reads: 386029'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17603'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 839.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12932315.161128'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/22/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/22/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//22/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/22/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.720616 from 364072 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.877073'\n",
       "\\item 'There were 379994 total reads, 379994 paired (377602 properly mated), 2056 proper singles, 336 improper reads (308 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/22/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 377602 reads, 377602 mapped, 0 unmapped, 377602 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 15 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 308 reads, 308 mapped, 0 unmapped, 308 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2056 reads, 2056 mapped, 0 unmapped, 2056 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 379994'\n",
       "\\item 'Total placed reads: 379993'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11788'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 577.401520'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12470140.178258'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/22/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/23/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//23/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/23/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.677136 from 516186 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.887594'\n",
       "\\item 'There were 537539 total reads, 537539 paired (535338 properly mated), 1915 proper singles, 286 improper reads (246 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/23/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 535338 reads, 535338 mapped, 0 unmapped, 535337 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1897 reads, 1897 mapped, 0 unmapped, 1897 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 537539'\n",
       "\\item 'Total placed reads: 537538'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12951'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 844.901572'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13346764.513687'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/23/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/23/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//23/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/23/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.670893 from 510296 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.883901'\n",
       "\\item 'There were 531447 total reads, 531447 paired (529170 properly mated), 1821 proper singles, 456 improper reads (428 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/23/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 529170 reads, 529170 mapped, 0 unmapped, 529170 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 428 reads, 428 mapped, 0 unmapped, 428 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1807 reads, 1807 mapped, 0 unmapped, 1807 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 531447'\n",
       "\\item 'Total placed reads: 531447'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 10525'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 683.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12996454.555808'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/23/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/24/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//24/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/24/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.677348 from 323618 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.860010'\n",
       "\\item 'There were 337511 total reads, 337511 paired (335530 properly mated), 1627 proper singles, 354 improper reads (326 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/24/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 335530 reads, 335530 mapped, 0 unmapped, 335530 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1627 reads, 1627 mapped, 0 unmapped, 1627 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 337511'\n",
       "\\item 'Total placed reads: 337511'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11254'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 842.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -9919433.184678'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/24/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/24/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//24/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/24/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.714823 from 316610 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.875752'\n",
       "\\item 'There were 330503 total reads, 330503 paired (328358 properly mated), 1739 proper singles, 406 improper reads (370 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/24/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 328358 reads, 328358 mapped, 0 unmapped, 328358 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 370 reads, 370 mapped, 0 unmapped, 370 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1739 reads, 1739 mapped, 0 unmapped, 1739 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 330503'\n",
       "\\item 'Total placed reads: 330503'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 10498'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 813.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -9515257.556246'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/24/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/25/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//25/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/25/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.700994 from 572804 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.896120'\n",
       "\\item 'There were 596261 total reads, 596261 paired (594004 properly mated), 1559 proper singles, 698 improper reads (676 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/25/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 594004 reads, 594004 mapped, 0 unmapped, 594004 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 676 reads, 676 mapped, 0 unmapped, 676 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1545 reads, 1545 mapped, 0 unmapped, 1545 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 596261'\n",
       "\\item 'Total placed reads: 596261'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12886'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1044.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14853488.488456'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/25/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/25/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//25/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/25/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.703648 from 565516 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.903196'\n",
       "\\item 'There were 589534 total reads, 589534 paired (586468 properly mated), 2148 proper singles, 918 improper reads (872 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/25/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 586468 reads, 586468 mapped, 0 unmapped, 586468 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 872 reads, 872 mapped, 0 unmapped, 872 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2142 reads, 2142 mapped, 0 unmapped, 2142 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 589534'\n",
       "\\item 'Total placed reads: 589534'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 10919'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1113.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14503625.790964'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/25/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/26/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//26/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/26/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.848518 from 762942 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.981232'\n",
       "\\item 'There were 794498 total reads, 794498 paired (791910 properly mated), 1582 proper singles, 1006 improper reads (860 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/26/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 791910 reads, 791910 mapped, 0 unmapped, 791903 placed, 7 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 88 reads, 88 mapped, 0 unmapped, 88 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1524 reads, 1524 mapped, 0 unmapped, 1524 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 794498'\n",
       "\\item 'Total placed reads: 794491'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 7'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 7176'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1039.751222'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14384733.330159'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/26/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/26/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//26/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/26/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.824811 from 758038 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.970790'\n",
       "\\item 'There were 790175 total reads, 790175 paired (786864 properly mated), 2387 proper singles, 924 improper reads (744 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/26/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 786864 reads, 786864 mapped, 0 unmapped, 786857 placed, 7 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 96 reads, 96 mapped, 0 unmapped, 96 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 744 reads, 744 mapped, 0 unmapped, 744 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2343 reads, 2343 mapped, 0 unmapped, 2343 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 790175'\n",
       "\\item 'Total placed reads: 790168'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 7'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 5558'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1245.682832'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14218317.825355'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/26/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/27/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//27/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/27/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.747729 from 484106 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.900189'\n",
       "\\item 'There were 504159 total reads, 504159 paired (502188 properly mated), 1653 proper singles, 318 improper reads (306 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/27/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 502188 reads, 502188 mapped, 0 unmapped, 502188 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 306 reads, 306 mapped, 0 unmapped, 306 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1651 reads, 1651 mapped, 0 unmapped, 1651 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 504159'\n",
       "\\item 'Total placed reads: 504159'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17255'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1064.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13516775.165159'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/27/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/27/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//27/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/27/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.716492 from 478160 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.887961'\n",
       "\\item 'There were 497996 total reads, 497996 paired (495980 properly mated), 1548 proper singles, 468 improper reads (448 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/27/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 495980 reads, 495980 mapped, 0 unmapped, 495980 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1548 reads, 1548 mapped, 0 unmapped, 1548 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 497996'\n",
       "\\item 'Total placed reads: 497996'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12965'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 694.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13118587.682537'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/27/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/28/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//28/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/28/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.771064 from 271272 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.901245'\n",
       "\\item 'There were 283284 total reads, 283284 paired (281252 properly mated), 1732 proper singles, 300 improper reads (300 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/28/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 281252 reads, 281252 mapped, 0 unmapped, 281252 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 300 reads, 300 mapped, 0 unmapped, 300 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1732 reads, 1732 mapped, 0 unmapped, 1732 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 283284'\n",
       "\\item 'Total placed reads: 283284'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12386'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 784.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11165800.783435'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/28/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/28/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//28/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/28/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.749976 from 266078 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.891963'\n",
       "\\item 'There were 277610 total reads, 277610 paired (275828 properly mated), 1510 proper singles, 272 improper reads (264 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/28/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 275828 reads, 275828 mapped, 0 unmapped, 275828 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 264 reads, 264 mapped, 0 unmapped, 264 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1510 reads, 1510 mapped, 0 unmapped, 1510 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 277610'\n",
       "\\item 'Total placed reads: 277610'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9038'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 552.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -10806835.625985'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/28/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/29/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//29/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/29/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.730651 from 533398 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.910921'\n",
       "\\item 'There were 555957 total reads, 555957 paired (553314 properly mated), 1853 proper singles, 790 improper reads (718 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/29/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 553314 reads, 553314 mapped, 0 unmapped, 553311 placed, 3 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 718 reads, 718 mapped, 0 unmapped, 718 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1825 reads, 1825 mapped, 0 unmapped, 1825 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 555957'\n",
       "\\item 'Total placed reads: 555954'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 3'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15709'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1088.605874'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11178993.697992'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/29/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/29/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//29/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/29/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.714380 from 527428 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.904824'\n",
       "\\item 'There were 549525 total reads, 549525 paired (547114 properly mated), 1749 proper singles, 662 improper reads (604 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/29/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 547114 reads, 547114 mapped, 0 unmapped, 547111 placed, 3 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1705 reads, 1705 mapped, 0 unmapped, 1705 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 549525'\n",
       "\\item 'Total placed reads: 549522'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 3'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11532'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 873.104767'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -10770544.377923'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/29/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/3/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//3/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/3/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.748898 from 235904 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.865444'\n",
       "\\item 'There were 246262 total reads, 246262 paired (244704 properly mated), 1486 proper singles, 72 improper reads (66 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/3/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 244704 reads, 244704 mapped, 0 unmapped, 244704 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 66 reads, 66 mapped, 0 unmapped, 66 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1486 reads, 1486 mapped, 0 unmapped, 1486 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 246262'\n",
       "\\item 'Total placed reads: 246262'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13408'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 750.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12545384.151465'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/3/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/3/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//3/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/3/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.732209 from 229306 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.856694'\n",
       "\\item 'There were 239119 total reads, 239119 paired (237844 properly mated), 1169 proper singles, 106 improper reads (106 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/3/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 237844 reads, 237844 mapped, 0 unmapped, 237844 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 106 reads, 106 mapped, 0 unmapped, 106 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1167 reads, 1167 mapped, 0 unmapped, 1167 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 239119'\n",
       "\\item 'Total placed reads: 239119'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9975'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 368.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12030126.110063'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/3/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/30/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//30/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/30/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.773617 from 607112 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.937958'\n",
       "\\item 'There were 631675 total reads, 631675 paired (629870 properly mated), 1685 proper singles, 120 improper reads (100 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/30/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 629870 reads, 629870 mapped, 0 unmapped, 629870 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 100 reads, 100 mapped, 0 unmapped, 100 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1683 reads, 1683 mapped, 0 unmapped, 1683 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 631675'\n",
       "\\item 'Total placed reads: 631675'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14930'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 830.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -26596142.638070'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/30/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/30/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//30/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/30/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.743582 from 600208 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.925605'\n",
       "\\item 'There were 624155 total reads, 624155 paired (622624 properly mated), 1393 proper singles, 138 improper reads (126 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/30/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 622624 reads, 622624 mapped, 0 unmapped, 622624 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 126 reads, 126 mapped, 0 unmapped, 126 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1391 reads, 1391 mapped, 0 unmapped, 1391 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 624155'\n",
       "\\item 'Total placed reads: 624155'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8854'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 510.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -25965655.555041'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/30/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/4/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//4/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/4/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.713039 from 379564 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.879604'\n",
       "\\item 'There were 396198 total reads, 396198 paired (393586 properly mated), 2148 proper singles, 464 improper reads (442 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/4/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 393586 reads, 393586 mapped, 0 unmapped, 393586 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 442 reads, 442 mapped, 0 unmapped, 442 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2144 reads, 2144 mapped, 0 unmapped, 2144 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 396198'\n",
       "\\item 'Total placed reads: 396198'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17254'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1255.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13656553.006202'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/4/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/4/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//4/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/4/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.707399 from 373622 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.880365'\n",
       "\\item 'There were 389836 total reads, 389836 paired (387414 properly mated), 1922 proper singles, 500 improper reads (484 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/4/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 387414 reads, 387414 mapped, 0 unmapped, 387414 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 484 reads, 484 mapped, 0 unmapped, 484 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1902 reads, 1902 mapped, 0 unmapped, 1902 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 389836'\n",
       "\\item 'Total placed reads: 389836'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13180'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 872.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13194840.370122'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/4/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/5/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//5/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/5/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.696256 from 250718 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.875842'\n",
       "\\item 'There were 261393 total reads, 261393 paired (260022 properly mated), 1297 proper singles, 74 improper reads (62 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/5/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 260022 reads, 260022 mapped, 0 unmapped, 260022 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1293 reads, 1293 mapped, 0 unmapped, 1293 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 261393'\n",
       "\\item 'Total placed reads: 261393'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9797'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 616.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -9418117.983413'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/5/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/5/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//5/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/5/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.662871 from 246096 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.866825'\n",
       "\\item 'There were 256213 total reads, 256213 paired (255122 properly mated), 963 proper singles, 128 improper reads (112 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/5/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 255122 reads, 255122 mapped, 0 unmapped, 255122 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 112 reads, 112 mapped, 0 unmapped, 112 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 963 reads, 963 mapped, 0 unmapped, 963 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 256213'\n",
       "\\item 'Total placed reads: 256213'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 6760'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 407.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -9083426.562538'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/5/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/6/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//6/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/6/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.767056 from 712204 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.927949'\n",
       "\\item 'There were 741088 total reads, 741088 paired (739100 properly mated), 1598 proper singles, 390 improper reads (332 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/6/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 739100 reads, 739100 mapped, 0 unmapped, 739100 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 332 reads, 332 mapped, 0 unmapped, 332 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1576 reads, 1576 mapped, 0 unmapped, 1576 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 741088'\n",
       "\\item 'Total placed reads: 741088'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11967'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 798.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16130340.567633'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/6/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/6/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//6/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/6/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.755147 from 707362 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.925815'\n",
       "\\item 'There were 736000 total reads, 736000 paired (734046 properly mated), 1576 proper singles, 378 improper reads (364 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/6/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 734046 reads, 734046 mapped, 0 unmapped, 734046 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 364 reads, 364 mapped, 0 unmapped, 364 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1572 reads, 1572 mapped, 0 unmapped, 1572 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 736000'\n",
       "\\item 'Total placed reads: 736000'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8499'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 662.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15828310.751221'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/6/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/7/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//7/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/7/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.764770 from 572266 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.929275'\n",
       "\\item 'There were 595605 total reads, 595605 paired (593784 properly mated), 1605 proper singles, 216 improper reads (206 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/7/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 593784 reads, 593784 mapped, 0 unmapped, 593784 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 206 reads, 206 mapped, 0 unmapped, 206 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 595605'\n",
       "\\item 'Total placed reads: 595605'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13246'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 866.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13895857.636032'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/7/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/7/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//7/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/7/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.758549 from 567304 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.929236'\n",
       "\\item 'There were 590509 total reads, 590509 paired (588602 properly mated), 1555 proper singles, 352 improper reads (336 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/7/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 588602 reads, 588602 mapped, 0 unmapped, 588602 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1549 reads, 1549 mapped, 0 unmapped, 1549 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 590509'\n",
       "\\item 'Total placed reads: 590509'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9107'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 595.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13582663.661134'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/7/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/8/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//8/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/8/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.789475 from 407582 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.908351'\n",
       "\\item 'There were 425560 total reads, 425560 paired (422904 properly mated), 2166 proper singles, 490 improper reads (456 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/8/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 422904 reads, 422904 mapped, 0 unmapped, 422904 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 456 reads, 456 mapped, 0 unmapped, 456 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2164 reads, 2164 mapped, 0 unmapped, 2164 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 425560'\n",
       "\\item 'Total placed reads: 425560'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15790'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1097.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13191614.984628'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/8/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/8/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//8/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/8/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.792128 from 398812 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.916498'\n",
       "\\item 'There were 416603 total reads, 416603 paired (413802 properly mated), 2269 proper singles, 532 improper reads (494 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/8/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 413802 reads, 413802 mapped, 0 unmapped, 413802 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 494 reads, 494 mapped, 0 unmapped, 494 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2267 reads, 2267 mapped, 0 unmapped, 2267 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 416603'\n",
       "\\item 'Total placed reads: 416603'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11518'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 926.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12711677.097005'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/8/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/9/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//9/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/9/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.705491 from 520154 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.900413'\n",
       "\\item 'There were 541958 total reads, 541958 paired (539498 properly mated), 1940 proper singles, 520 improper reads (470 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/9/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 539498 reads, 539498 mapped, 0 unmapped, 539497 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 470 reads, 470 mapped, 0 unmapped, 470 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1930 reads, 1930 mapped, 0 unmapped, 1930 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 541958'\n",
       "\\item 'Total placed reads: 541957'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15487'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 994.001834'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13988489.574853'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/9/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/map/9/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/assembly//9/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/9/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.722684 from 511928 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.907540'\n",
       "\\item 'There were 533364 total reads, 533364 paired (531020 properly mated), 1812 proper singles, 532 improper reads (506 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/9/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 531020 reads, 531020 mapped, 0 unmapped, 531020 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 506 reads, 506 mapped, 0 unmapped, 506 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1806 reads, 1806 mapped, 0 unmapped, 1806 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 533364'\n",
       "\\item 'Total placed reads: 533364'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13274'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 848.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13635354.189521'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/9/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\end{enumerate}\n"
      ],
      "text/markdown": [
       "1. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.739318 from 316378 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.908582'\n",
       "9. 'There were 330559 total reads, 330559 paired (328316 properly mated), 2105 proper singles, 138 improper reads (128 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 328316 reads, 328316 mapped, 0 unmapped, 328316 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 128 reads, 128 mapped, 0 unmapped, 128 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2105 reads, 2105 mapped, 0 unmapped, 2105 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 330559'\n",
       "27. 'Total placed reads: 330559'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 27092'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1293.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -17374082.995081'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "2. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.712465 from 306050 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.897022'\n",
       "9. 'There were 319723 total reads, 319723 paired (317490 properly mated), 1965 proper singles, 268 improper reads (246 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 317490 reads, 317490 mapped, 0 unmapped, 317490 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1965 reads, 1965 mapped, 0 unmapped, 1965 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 319723'\n",
       "27. 'Total placed reads: 319723'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 16819'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 736.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16575003.788841'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "3. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.763820 from 506326 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.920908'\n",
       "9. 'There were 528349 total reads, 528349 paired (525374 properly mated), 2561 proper singles, 414 improper reads (402 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 525374 reads, 525374 mapped, 0 unmapped, 525374 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2559 reads, 2559 mapped, 0 unmapped, 2559 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 528349'\n",
       "27. 'Total placed reads: 528349'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 18480'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1187.100000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12015646.535580'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "4. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.755345 from 497614 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.923640'\n",
       "9. 'There were 519607 total reads, 519607 paired (516280 properly mated), 2649 proper singles, 678 improper reads (662 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 516280 reads, 516280 mapped, 0 unmapped, 516280 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 662 reads, 662 mapped, 0 unmapped, 662 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2643 reads, 2643 mapped, 0 unmapped, 2643 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 519607'\n",
       "27. 'Total placed reads: 519607'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 15662'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 907.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -11418044.195180'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "5. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/11/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//11/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.766512 from 368602 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.904772'\n",
       "9. 'There were 384711 total reads, 384711 paired (382384 properly mated), 2137 proper singles, 190 improper reads (180 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 382384 reads, 382384 mapped, 0 unmapped, 382384 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 180 reads, 180 mapped, 0 unmapped, 180 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2121 reads, 2121 mapped, 0 unmapped, 2121 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 384711'\n",
       "27. 'Total placed reads: 384710'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 17799'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 880.502289'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13270694.290886'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "6. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/11/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//11/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.771830 from 361546 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.910933'\n",
       "9. 'There were 377432 total reads, 377432 paired (375054 properly mated), 2026 proper singles, 352 improper reads (324 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 375054 reads, 375054 mapped, 0 unmapped, 375054 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 324 reads, 324 mapped, 0 unmapped, 324 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2016 reads, 2016 mapped, 0 unmapped, 2016 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 377432'\n",
       "27. 'Total placed reads: 377432'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11820'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 694.200000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12801647.537861'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "7. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/12/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//12/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.781898 from 392358 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.905981'\n",
       "9. 'There were 409329 total reads, 409329 paired (407022 properly mated), 2087 proper singles, 220 improper reads (192 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 407022 reads, 407022 mapped, 0 unmapped, 407022 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 192 reads, 192 mapped, 0 unmapped, 192 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2075 reads, 2075 mapped, 0 unmapped, 2075 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 409329'\n",
       "27. 'Total placed reads: 409329'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 21455'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1214.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16687248.633607'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "8. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/12/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//12/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.741528 from 382920 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.895810'\n",
       "9. 'There were 399459 total reads, 399459 paired (397146 properly mated), 2007 proper singles, 306 improper reads (280 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 397146 reads, 397146 mapped, 0 unmapped, 397146 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 280 reads, 280 mapped, 0 unmapped, 280 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2005 reads, 2005 mapped, 0 unmapped, 2005 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 399459'\n",
       "27. 'Total placed reads: 399459'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 16214'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 709.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16026048.041389'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "9. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/13/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//13/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.836049 from 225994 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.893290'\n",
       "9. 'There were 236258 total reads, 236258 paired (234416 properly mated), 1744 proper singles, 98 improper reads (98 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 234416 reads, 234416 mapped, 0 unmapped, 234416 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 98 reads, 98 mapped, 0 unmapped, 98 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1742 reads, 1742 mapped, 0 unmapped, 1742 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 236258'\n",
       "27. 'Total placed reads: 236258'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 20144'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 939.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13441130.446388'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "10. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/13/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//13/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.752095 from 218834 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.863300'\n",
       "9. 'There were 228674 total reads, 228674 paired (226896 properly mated), 1646 proper singles, 132 improper reads (110 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 226896 reads, 226896 mapped, 0 unmapped, 226896 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 110 reads, 110 mapped, 0 unmapped, 110 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1646 reads, 1646 mapped, 0 unmapped, 1646 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 228674'\n",
       "27. 'Total placed reads: 228674'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13893'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 456.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12829391.286242'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "11. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/14/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//14/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.757812 from 483128 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.914564'\n",
       "9. 'There were 503117 total reads, 503117 paired (501250 properly mated), 1605 proper singles, 262 improper reads (242 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 501250 reads, 501250 mapped, 0 unmapped, 501250 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 242 reads, 242 mapped, 0 unmapped, 242 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 503117'\n",
       "27. 'Total placed reads: 503117'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 16405'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 833.200000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13218885.840352'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "12. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/14/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//14/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.734157 from 477590 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.908243'\n",
       "9. 'There were 497321 total reads, 497321 paired (495422 properly mated), 1527 proper singles, 372 improper reads (360 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 495422 reads, 495422 mapped, 0 unmapped, 495422 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 360 reads, 360 mapped, 0 unmapped, 360 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1527 reads, 1527 mapped, 0 unmapped, 1527 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 497321'\n",
       "27. 'Total placed reads: 497320'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 9978'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 625.501258'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12874444.057809'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "13. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/15/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//15/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.744798 from 573334 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.920983'\n",
       "9. 'There were 596785 total reads, 596785 paired (594766 properly mated), 1773 proper singles, 246 improper reads (238 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 594766 reads, 594766 mapped, 0 unmapped, 594765 placed, 1 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 238 reads, 238 mapped, 0 unmapped, 238 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1773 reads, 1773 mapped, 0 unmapped, 1773 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 596785'\n",
       "27. 'Total placed reads: 596784'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 18442'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1030.801727'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12399446.201300'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "14. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/15/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//15/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.713940 from 565336 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.910377'\n",
       "9. 'There were 588560 total reads, 588560 paired (586414 properly mated), 1778 proper singles, 368 improper reads (354 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 586414 reads, 586414 mapped, 0 unmapped, 586413 placed, 1 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1776 reads, 1776 mapped, 0 unmapped, 1776 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 588560'\n",
       "27. 'Total placed reads: 588559'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13012'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 728.101237'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -11815273.130115'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "15. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/16/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//16/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.825277 from 258958 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.906877'\n",
       "9. 'There were 270606 total reads, 270606 paired (268648 properly mated), 1852 proper singles, 106 improper reads (92 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 268648 reads, 268648 mapped, 0 unmapped, 268648 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 92 reads, 92 mapped, 0 unmapped, 92 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1852 reads, 1852 mapped, 0 unmapped, 1852 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 270606'\n",
       "27. 'Total placed reads: 270606'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 21444'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1181.000000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14952829.516917'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "16. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/16/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//16/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.760708 from 250940 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.880949'\n",
       "9. 'There were 261924 total reads, 261924 paired (260222 properly mated), 1560 proper singles, 142 improper reads (124 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 260222 reads, 260222 mapped, 0 unmapped, 260222 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 124 reads, 124 mapped, 0 unmapped, 124 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1560 reads, 1560 mapped, 0 unmapped, 1560 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 261924'\n",
       "27. 'Total placed reads: 261924'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 14252'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 611.900000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14246895.113395'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "17. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/17/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//17/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.741113 from 421342 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.895415'\n",
       "9. 'There were 439355 total reads, 439355 paired (437148 properly mated), 1841 proper singles, 366 improper reads (344 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 437148 reads, 437148 mapped, 0 unmapped, 437148 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 344 reads, 344 mapped, 0 unmapped, 344 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1835 reads, 1835 mapped, 0 unmapped, 1835 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 439355'\n",
       "27. 'Total placed reads: 439355'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 15450'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 978.200000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -15184661.003788'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "18. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/17/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//17/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.755768 from 413492 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.903851'\n",
       "9. 'There were 431046 total reads, 431046 paired (428998 properly mated), 1648 proper singles, 400 improper reads (382 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 428998 reads, 428998 mapped, 0 unmapped, 428997 placed, 1 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 382 reads, 382 mapped, 0 unmapped, 382 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1648 reads, 1648 mapped, 0 unmapped, 1648 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 431046'\n",
       "27. 'Total placed reads: 431045'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 12471'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 683.801586'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14668353.704128'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "19. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/18/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//18/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.749547 from 757308 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.932785'\n",
       "9. 'There were 788345 total reads, 788345 paired (785754 properly mated), 1677 proper singles, 914 improper reads (828 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 785754 reads, 785754 mapped, 0 unmapped, 785754 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 828 reads, 828 mapped, 0 unmapped, 828 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1649 reads, 1649 mapped, 0 unmapped, 1649 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 788345'\n",
       "27. 'Total placed reads: 788345'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11841'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1212.800000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -11191721.107405'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "20. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/18/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//18/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.721260 from 750586 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.920155'\n",
       "9. 'There were 781411 total reads, 781411 paired (778634 properly mated), 2063 proper singles, 714 improper reads (630 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 778634 reads, 778634 mapped, 0 unmapped, 778634 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 630 reads, 630 mapped, 0 unmapped, 630 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2059 reads, 2059 mapped, 0 unmapped, 2059 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 781411'\n",
       "27. 'Total placed reads: 781411'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 8211'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1198.400000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -10727524.033928'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "21. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/19/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//19/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.710925 from 439506 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.882234'\n",
       "9. 'There were 458135 total reads, 458135 paired (455902 properly mated), 1979 proper singles, 254 improper reads (240 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 455902 reads, 455902 mapped, 0 unmapped, 455902 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 240 reads, 240 mapped, 0 unmapped, 240 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1979 reads, 1979 mapped, 0 unmapped, 1979 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 458135'\n",
       "27. 'Total placed reads: 458135'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 16123'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 998.700000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12704608.093983'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "22. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/19/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//19/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.685501 from 431938 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.869250'\n",
       "9. 'There were 450048 total reads, 450048 paired (447962 properly mated), 1666 proper singles, 420 improper reads (404 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 447962 reads, 447962 mapped, 0 unmapped, 447962 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 404 reads, 404 mapped, 0 unmapped, 404 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1662 reads, 1662 mapped, 0 unmapped, 1662 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 450048'\n",
       "27. 'Total placed reads: 450048'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11611'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 640.800000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12183660.987856'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "23. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/2/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//2/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.763331 from 852556 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.933910'\n",
       "9. 'There were 887324 total reads, 887324 paired (884688 properly mated), 1796 proper singles, 840 improper reads (764 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 884688 reads, 884688 mapped, 0 unmapped, 884688 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 764 reads, 764 mapped, 0 unmapped, 764 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1790 reads, 1790 mapped, 0 unmapped, 1790 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 887324'\n",
       "27. 'Total placed reads: 887324'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 14078'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1238.900000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -17187997.456175'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "24. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/2/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//2/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.757652 from 841888 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.932336'\n",
       "9. 'There were 876686 total reads, 876686 paired (873584 properly mated), 2236 proper singles, 866 improper reads (796 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 873584 reads, 873584 mapped, 0 unmapped, 873584 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 796 reads, 796 mapped, 0 unmapped, 796 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2224 reads, 2224 mapped, 0 unmapped, 2224 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 876686'\n",
       "27. 'Total placed reads: 876686'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 9417'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1354.400000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16654318.964110'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "25. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/20/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//20/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.693648 from 546398 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.887225'\n",
       "9. 'There were 568494 total reads, 568494 paired (566614 properly mated), 1594 proper singles, 286 improper reads (270 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 566614 reads, 566614 mapped, 0 unmapped, 566614 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 270 reads, 270 mapped, 0 unmapped, 270 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1578 reads, 1578 mapped, 0 unmapped, 1578 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 568494'\n",
       "27. 'Total placed reads: 568494'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13121'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 765.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16995916.356186'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "26. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/20/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//20/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.703542 from 539658 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.894556'\n",
       "9. 'There were 561765 total reads, 561765 paired (559688 properly mated), 1663 proper singles, 414 improper reads (400 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 559688 reads, 559688 mapped, 0 unmapped, 559688 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 400 reads, 400 mapped, 0 unmapped, 400 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1663 reads, 1663 mapped, 0 unmapped, 1663 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 561765'\n",
       "27. 'Total placed reads: 561765'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 8043'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 857.700000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16611079.347268'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "27. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/21/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//21/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.770890 from 698024 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.935357'\n",
       "9. 'There were 725969 total reads, 725969 paired (724254 properly mated), 1479 proper singles, 236 improper reads (214 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 724254 reads, 724254 mapped, 0 unmapped, 724253 placed, 1 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 214 reads, 214 mapped, 0 unmapped, 214 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1475 reads, 1475 mapped, 0 unmapped, 1475 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 725969'\n",
       "27. 'Total placed reads: 725968'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 8492'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 661.400911'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -15127024.327711'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "28. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/21/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//21/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.749190 from 693936 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.925717'\n",
       "9. 'There were 721773 total reads, 721773 paired (719942 properly mated), 1481 proper singles, 350 improper reads (336 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 719942 reads, 719942 mapped, 0 unmapped, 719942 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1473 reads, 1473 mapped, 0 unmapped, 1473 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 721773'\n",
       "27. 'Total placed reads: 721773'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 5549'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 570.200000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14809653.791573'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "29. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/22/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//22/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.741647 from 369982 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.879979'\n",
       "9. 'There were 386029 total reads, 386029 paired (383718 properly mated), 2143 proper singles, 168 improper reads (142 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 383718 reads, 383718 mapped, 0 unmapped, 383718 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 142 reads, 142 mapped, 0 unmapped, 142 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2143 reads, 2143 mapped, 0 unmapped, 2143 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 386029'\n",
       "27. 'Total placed reads: 386029'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 17603'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 839.200000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12932315.161128'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "30. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/22/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//22/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.720616 from 364072 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.877073'\n",
       "9. 'There were 379994 total reads, 379994 paired (377602 properly mated), 2056 proper singles, 336 improper reads (308 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 377602 reads, 377602 mapped, 0 unmapped, 377602 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 15 placed, 1 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 308 reads, 308 mapped, 0 unmapped, 308 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2056 reads, 2056 mapped, 0 unmapped, 2056 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 379994'\n",
       "27. 'Total placed reads: 379993'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11788'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 577.401520'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12470140.178258'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "31. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/23/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//23/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.677136 from 516186 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.887594'\n",
       "9. 'There were 537539 total reads, 537539 paired (535338 properly mated), 1915 proper singles, 286 improper reads (246 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 535338 reads, 535338 mapped, 0 unmapped, 535337 placed, 1 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1897 reads, 1897 mapped, 0 unmapped, 1897 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 537539'\n",
       "27. 'Total placed reads: 537538'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 12951'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 844.901572'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13346764.513687'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "32. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/23/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//23/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.670893 from 510296 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.883901'\n",
       "9. 'There were 531447 total reads, 531447 paired (529170 properly mated), 1821 proper singles, 456 improper reads (428 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 529170 reads, 529170 mapped, 0 unmapped, 529170 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 428 reads, 428 mapped, 0 unmapped, 428 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1807 reads, 1807 mapped, 0 unmapped, 1807 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 531447'\n",
       "27. 'Total placed reads: 531447'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 10525'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 683.400000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12996454.555808'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "33. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/24/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//24/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.677348 from 323618 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.860010'\n",
       "9. 'There were 337511 total reads, 337511 paired (335530 properly mated), 1627 proper singles, 354 improper reads (326 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 335530 reads, 335530 mapped, 0 unmapped, 335530 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1627 reads, 1627 mapped, 0 unmapped, 1627 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 337511'\n",
       "27. 'Total placed reads: 337511'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11254'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 842.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -9919433.184678'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "34. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/24/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//24/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.714823 from 316610 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.875752'\n",
       "9. 'There were 330503 total reads, 330503 paired (328358 properly mated), 1739 proper singles, 406 improper reads (370 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 328358 reads, 328358 mapped, 0 unmapped, 328358 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 370 reads, 370 mapped, 0 unmapped, 370 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1739 reads, 1739 mapped, 0 unmapped, 1739 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 330503'\n",
       "27. 'Total placed reads: 330503'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 10498'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 813.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -9515257.556246'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "35. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/25/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//25/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.700994 from 572804 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.896120'\n",
       "9. 'There were 596261 total reads, 596261 paired (594004 properly mated), 1559 proper singles, 698 improper reads (676 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 594004 reads, 594004 mapped, 0 unmapped, 594004 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 676 reads, 676 mapped, 0 unmapped, 676 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1545 reads, 1545 mapped, 0 unmapped, 1545 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 596261'\n",
       "27. 'Total placed reads: 596261'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 12886'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1044.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14853488.488456'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "36. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/25/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//25/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.703648 from 565516 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.903196'\n",
       "9. 'There were 589534 total reads, 589534 paired (586468 properly mated), 2148 proper singles, 918 improper reads (872 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 586468 reads, 586468 mapped, 0 unmapped, 586468 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 872 reads, 872 mapped, 0 unmapped, 872 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2142 reads, 2142 mapped, 0 unmapped, 2142 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 589534'\n",
       "27. 'Total placed reads: 589534'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 10919'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1113.800000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14503625.790964'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "37. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/26/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//26/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.848518 from 762942 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.981232'\n",
       "9. 'There were 794498 total reads, 794498 paired (791910 properly mated), 1582 proper singles, 1006 improper reads (860 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 791910 reads, 791910 mapped, 0 unmapped, 791903 placed, 7 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 88 reads, 88 mapped, 0 unmapped, 88 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1524 reads, 1524 mapped, 0 unmapped, 1524 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 794498'\n",
       "27. 'Total placed reads: 794491'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 7'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 7176'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1039.751222'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14384733.330159'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "38. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/26/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//26/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.824811 from 758038 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.970790'\n",
       "9. 'There were 790175 total reads, 790175 paired (786864 properly mated), 2387 proper singles, 924 improper reads (744 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 786864 reads, 786864 mapped, 0 unmapped, 786857 placed, 7 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 96 reads, 96 mapped, 0 unmapped, 96 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 744 reads, 744 mapped, 0 unmapped, 744 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2343 reads, 2343 mapped, 0 unmapped, 2343 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 790175'\n",
       "27. 'Total placed reads: 790168'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 7'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 5558'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1245.682832'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -14218317.825355'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "39. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/27/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//27/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.747729 from 484106 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.900189'\n",
       "9. 'There were 504159 total reads, 504159 paired (502188 properly mated), 1653 proper singles, 318 improper reads (306 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 502188 reads, 502188 mapped, 0 unmapped, 502188 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 306 reads, 306 mapped, 0 unmapped, 306 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1651 reads, 1651 mapped, 0 unmapped, 1651 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 504159'\n",
       "27. 'Total placed reads: 504159'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 17255'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1064.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13516775.165159'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "40. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/27/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//27/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.716492 from 478160 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.887961'\n",
       "9. 'There were 497996 total reads, 497996 paired (495980 properly mated), 1548 proper singles, 468 improper reads (448 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 495980 reads, 495980 mapped, 0 unmapped, 495980 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1548 reads, 1548 mapped, 0 unmapped, 1548 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 497996'\n",
       "27. 'Total placed reads: 497996'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 12965'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 694.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13118587.682537'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "41. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/28/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//28/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.771064 from 271272 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.901245'\n",
       "9. 'There were 283284 total reads, 283284 paired (281252 properly mated), 1732 proper singles, 300 improper reads (300 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 281252 reads, 281252 mapped, 0 unmapped, 281252 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 300 reads, 300 mapped, 0 unmapped, 300 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1732 reads, 1732 mapped, 0 unmapped, 1732 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 283284'\n",
       "27. 'Total placed reads: 283284'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 12386'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 784.700000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -11165800.783435'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "42. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/28/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//28/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.749976 from 266078 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.891963'\n",
       "9. 'There were 277610 total reads, 277610 paired (275828 properly mated), 1510 proper singles, 272 improper reads (264 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 275828 reads, 275828 mapped, 0 unmapped, 275828 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 264 reads, 264 mapped, 0 unmapped, 264 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1510 reads, 1510 mapped, 0 unmapped, 1510 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 277610'\n",
       "27. 'Total placed reads: 277610'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 9038'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 552.400000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -10806835.625985'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "43. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/29/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//29/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.730651 from 533398 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.910921'\n",
       "9. 'There were 555957 total reads, 555957 paired (553314 properly mated), 1853 proper singles, 790 improper reads (718 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 553314 reads, 553314 mapped, 0 unmapped, 553311 placed, 3 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 718 reads, 718 mapped, 0 unmapped, 718 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1825 reads, 1825 mapped, 0 unmapped, 1825 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 555957'\n",
       "27. 'Total placed reads: 555954'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 3'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 15709'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1088.605874'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -11178993.697992'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "44. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/29/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//29/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.714380 from 527428 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.904824'\n",
       "9. 'There were 549525 total reads, 549525 paired (547114 properly mated), 1749 proper singles, 662 improper reads (604 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 547114 reads, 547114 mapped, 0 unmapped, 547111 placed, 3 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1705 reads, 1705 mapped, 0 unmapped, 1705 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 549525'\n",
       "27. 'Total placed reads: 549522'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 3'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11532'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 873.104767'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -10770544.377923'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "45. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/3/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//3/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.748898 from 235904 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.865444'\n",
       "9. 'There were 246262 total reads, 246262 paired (244704 properly mated), 1486 proper singles, 72 improper reads (66 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 244704 reads, 244704 mapped, 0 unmapped, 244704 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 66 reads, 66 mapped, 0 unmapped, 66 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1486 reads, 1486 mapped, 0 unmapped, 1486 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 246262'\n",
       "27. 'Total placed reads: 246262'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13408'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 750.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12545384.151465'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "46. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/3/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//3/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.732209 from 229306 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.856694'\n",
       "9. 'There were 239119 total reads, 239119 paired (237844 properly mated), 1169 proper singles, 106 improper reads (106 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 237844 reads, 237844 mapped, 0 unmapped, 237844 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 106 reads, 106 mapped, 0 unmapped, 106 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1167 reads, 1167 mapped, 0 unmapped, 1167 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 239119'\n",
       "27. 'Total placed reads: 239119'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 9975'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 368.200000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12030126.110063'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "47. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/30/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//30/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.773617 from 607112 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.937958'\n",
       "9. 'There were 631675 total reads, 631675 paired (629870 properly mated), 1685 proper singles, 120 improper reads (100 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 629870 reads, 629870 mapped, 0 unmapped, 629870 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 100 reads, 100 mapped, 0 unmapped, 100 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1683 reads, 1683 mapped, 0 unmapped, 1683 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 631675'\n",
       "27. 'Total placed reads: 631675'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 14930'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 830.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -26596142.638070'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "48. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/30/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//30/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.743582 from 600208 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.925605'\n",
       "9. 'There were 624155 total reads, 624155 paired (622624 properly mated), 1393 proper singles, 138 improper reads (126 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 622624 reads, 622624 mapped, 0 unmapped, 622624 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 126 reads, 126 mapped, 0 unmapped, 126 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1391 reads, 1391 mapped, 0 unmapped, 1391 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 624155'\n",
       "27. 'Total placed reads: 624155'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 8854'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 510.800000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -25965655.555041'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "49. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/4/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//4/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.713039 from 379564 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.879604'\n",
       "9. 'There were 396198 total reads, 396198 paired (393586 properly mated), 2148 proper singles, 464 improper reads (442 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 393586 reads, 393586 mapped, 0 unmapped, 393586 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 442 reads, 442 mapped, 0 unmapped, 442 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2144 reads, 2144 mapped, 0 unmapped, 2144 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 396198'\n",
       "27. 'Total placed reads: 396198'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 17254'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1255.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13656553.006202'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "50. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/4/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//4/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.707399 from 373622 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.880365'\n",
       "9. 'There were 389836 total reads, 389836 paired (387414 properly mated), 1922 proper singles, 500 improper reads (484 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 387414 reads, 387414 mapped, 0 unmapped, 387414 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 484 reads, 484 mapped, 0 unmapped, 484 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1902 reads, 1902 mapped, 0 unmapped, 1902 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 389836'\n",
       "27. 'Total placed reads: 389836'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13180'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 872.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13194840.370122'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "51. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/5/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//5/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.696256 from 250718 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.875842'\n",
       "9. 'There were 261393 total reads, 261393 paired (260022 properly mated), 1297 proper singles, 74 improper reads (62 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 260022 reads, 260022 mapped, 0 unmapped, 260022 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1293 reads, 1293 mapped, 0 unmapped, 1293 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 261393'\n",
       "27. 'Total placed reads: 261393'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 9797'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 616.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -9418117.983413'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "52. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/5/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//5/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.662871 from 246096 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.866825'\n",
       "9. 'There were 256213 total reads, 256213 paired (255122 properly mated), 963 proper singles, 128 improper reads (112 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 255122 reads, 255122 mapped, 0 unmapped, 255122 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 112 reads, 112 mapped, 0 unmapped, 112 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 963 reads, 963 mapped, 0 unmapped, 963 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 256213'\n",
       "27. 'Total placed reads: 256213'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 6760'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 407.000000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -9083426.562538'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "53. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/6/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//6/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.767056 from 712204 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.927949'\n",
       "9. 'There were 741088 total reads, 741088 paired (739100 properly mated), 1598 proper singles, 390 improper reads (332 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 739100 reads, 739100 mapped, 0 unmapped, 739100 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 332 reads, 332 mapped, 0 unmapped, 332 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1576 reads, 1576 mapped, 0 unmapped, 1576 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 741088'\n",
       "27. 'Total placed reads: 741088'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11967'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 798.000000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -16130340.567633'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "54. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/6/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//6/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.755147 from 707362 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.925815'\n",
       "9. 'There were 736000 total reads, 736000 paired (734046 properly mated), 1576 proper singles, 378 improper reads (364 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 734046 reads, 734046 mapped, 0 unmapped, 734046 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 364 reads, 364 mapped, 0 unmapped, 364 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1572 reads, 1572 mapped, 0 unmapped, 1572 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 736000'\n",
       "27. 'Total placed reads: 736000'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 8499'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 662.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -15828310.751221'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "55. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/7/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//7/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.764770 from 572266 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.929275'\n",
       "9. 'There were 595605 total reads, 595605 paired (593784 properly mated), 1605 proper singles, 216 improper reads (206 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 593784 reads, 593784 mapped, 0 unmapped, 593784 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 206 reads, 206 mapped, 0 unmapped, 206 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 595605'\n",
       "27. 'Total placed reads: 595605'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13246'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 866.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13895857.636032'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "56. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/7/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//7/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.758549 from 567304 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.929236'\n",
       "9. 'There were 590509 total reads, 590509 paired (588602 properly mated), 1555 proper singles, 352 improper reads (336 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 588602 reads, 588602 mapped, 0 unmapped, 588602 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1549 reads, 1549 mapped, 0 unmapped, 1549 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 590509'\n",
       "27. 'Total placed reads: 590509'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 9107'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 595.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13582663.661134'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "57. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/8/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//8/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.789475 from 407582 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.908351'\n",
       "9. 'There were 425560 total reads, 425560 paired (422904 properly mated), 2166 proper singles, 490 improper reads (456 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 422904 reads, 422904 mapped, 0 unmapped, 422904 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 456 reads, 456 mapped, 0 unmapped, 456 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2164 reads, 2164 mapped, 0 unmapped, 2164 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 425560'\n",
       "27. 'Total placed reads: 425560'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 15790'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1097.700000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13191614.984628'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "58. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/8/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//8/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.792128 from 398812 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.916498'\n",
       "9. 'There were 416603 total reads, 416603 paired (413802 properly mated), 2269 proper singles, 532 improper reads (494 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 413802 reads, 413802 mapped, 0 unmapped, 413802 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 494 reads, 494 mapped, 0 unmapped, 494 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 2267 reads, 2267 mapped, 0 unmapped, 2267 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 416603'\n",
       "27. 'Total placed reads: 416603'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 11518'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 926.600000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -12711677.097005'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "59. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/9/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//9/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.705491 from 520154 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.900413'\n",
       "9. 'There were 541958 total reads, 541958 paired (539498 properly mated), 1940 proper singles, 520 improper reads (470 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 539498 reads, 539498 mapped, 0 unmapped, 539497 placed, 1 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 470 reads, 470 mapped, 0 unmapped, 470 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1930 reads, 1930 mapped, 0 unmapped, 1930 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 541958'\n",
       "27. 'Total placed reads: 541957'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 15487'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 994.001834'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13988489.574853'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "60. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/9/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//9/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.722684 from 511928 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.907540'\n",
       "9. 'There were 533364 total reads, 533364 paired (531020 properly mated), 1812 proper singles, 532 improper reads (506 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 531020 reads, 531020 mapped, 0 unmapped, 531020 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 506 reads, 506 mapped, 0 unmapped, 506 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1806 reads, 1806 mapped, 0 unmapped, 1806 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 533364'\n",
       "27. 'Total placed reads: 533364'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13274'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 848.700000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -13635354.189521'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[[1]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.739318 from 316378 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.908582\"                                                                                                     \n",
       " [9] \"There were 330559 total reads, 330559 paired (328316 properly mated), 2105 proper singles, 138 improper reads (128 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 328316 reads, 328316 mapped, 0 unmapped, 328316 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 128 reads, 128 mapped, 0 unmapped, 128 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2105 reads, 2105 mapped, 0 unmapped, 2105 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 330559\"                                                                                                                           \n",
       "[27] \"Total placed reads: 330559\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 27092\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1293.600000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -17374082.995081\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[2]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/1/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//1/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.712465 from 306050 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.897022\"                                                                                                     \n",
       " [9] \"There were 319723 total reads, 319723 paired (317490 properly mated), 1965 proper singles, 268 improper reads (246 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 317490 reads, 317490 mapped, 0 unmapped, 317490 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1965 reads, 1965 mapped, 0 unmapped, 1965 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 319723\"                                                                                                                           \n",
       "[27] \"Total placed reads: 319723\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 16819\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 736.500000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16575003.788841\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[3]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.763820 from 506326 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.920908\"                                                                                                     \n",
       " [9] \"There were 528349 total reads, 528349 paired (525374 properly mated), 2561 proper singles, 414 improper reads (402 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 525374 reads, 525374 mapped, 0 unmapped, 525374 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2559 reads, 2559 mapped, 0 unmapped, 2559 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 528349\"                                                                                                                           \n",
       "[27] \"Total placed reads: 528349\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 18480\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1187.100000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12015646.535580\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[4]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/10/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//10/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.755345 from 497614 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.923640\"                                                                                                     \n",
       " [9] \"There were 519607 total reads, 519607 paired (516280 properly mated), 2649 proper singles, 678 improper reads (662 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 516280 reads, 516280 mapped, 0 unmapped, 516280 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 662 reads, 662 mapped, 0 unmapped, 662 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2643 reads, 2643 mapped, 0 unmapped, 2643 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 519607\"                                                                                                                           \n",
       "[27] \"Total placed reads: 519607\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 15662\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 907.500000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -11418044.195180\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[5]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/11/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//11/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.766512 from 368602 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.904772\"                                                                                                     \n",
       " [9] \"There were 384711 total reads, 384711 paired (382384 properly mated), 2137 proper singles, 190 improper reads (180 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 382384 reads, 382384 mapped, 0 unmapped, 382384 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 180 reads, 180 mapped, 0 unmapped, 180 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2121 reads, 2121 mapped, 0 unmapped, 2121 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 384711\"                                                                                                                           \n",
       "[27] \"Total placed reads: 384710\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 17799\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 880.502289\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13270694.290886\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[6]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/11/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//11/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.771830 from 361546 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.910933\"                                                                                                     \n",
       " [9] \"There were 377432 total reads, 377432 paired (375054 properly mated), 2026 proper singles, 352 improper reads (324 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 375054 reads, 375054 mapped, 0 unmapped, 375054 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 324 reads, 324 mapped, 0 unmapped, 324 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2016 reads, 2016 mapped, 0 unmapped, 2016 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 377432\"                                                                                                                           \n",
       "[27] \"Total placed reads: 377432\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11820\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 694.200000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12801647.537861\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[7]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/12/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//12/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.781898 from 392358 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.905981\"                                                                                                     \n",
       " [9] \"There were 409329 total reads, 409329 paired (407022 properly mated), 2087 proper singles, 220 improper reads (192 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 407022 reads, 407022 mapped, 0 unmapped, 407022 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 192 reads, 192 mapped, 0 unmapped, 192 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2075 reads, 2075 mapped, 0 unmapped, 2075 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 409329\"                                                                                                                           \n",
       "[27] \"Total placed reads: 409329\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 21455\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1214.500000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16687248.633607\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[8]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/12/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//12/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.741528 from 382920 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.895810\"                                                                                                     \n",
       " [9] \"There were 399459 total reads, 399459 paired (397146 properly mated), 2007 proper singles, 306 improper reads (280 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 397146 reads, 397146 mapped, 0 unmapped, 397146 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 280 reads, 280 mapped, 0 unmapped, 280 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2005 reads, 2005 mapped, 0 unmapped, 2005 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 399459\"                                                                                                                           \n",
       "[27] \"Total placed reads: 399459\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 16214\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 709.600000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16026048.041389\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/12/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[9]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/13/megahit.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//13/megahit/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                             \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                     \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                \n",
       " [7] \"Found FR sample avg insert length to be 253.836049 from 225994 mapped reads\"                                                                        \n",
       " [8] \"Found FR sample insert length std to be 9.893290\"                                                                                                   \n",
       " [9] \"There were 236258 total reads, 236258 paired (234416 properly mated), 1744 proper singles, 98 improper reads (98 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                               \n",
       "[12] \"FR orientation with 234416 reads, 234416 mapped, 0 unmapped, 234416 placed, 0 mappedButNotPlaced\"                                                   \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[19] \"CHIMER orientation with 98 reads, 98 mapped, 0 unmapped, 98 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1742 reads, 1742 mapped, 0 unmapped, 1742 placed, 0 mappedButNotPlaced\"                                       \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[26] \"Total mapped reads: 236258\"                                                                                                                         \n",
       "[27] \"Total placed reads: 236258\"                                                                                                                         \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                            \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                            \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                      \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                   \n",
       "[32] \"Computing depth statistics...\"                                                                                                                      \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                        \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                     \n",
       "[35] \"bases with 0 coverage: 20144\"                                                                                                                       \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                   \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                        \n",
       "[38] \"Expected extra length: 939.300000\"                                                                                                                  \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                     \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                          \n",
       "[41] \"Total ALE Score: -13441130.446388\"                                                                                                                  \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/megahit.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                 \n",
       "\n",
       "[[10]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/13/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//13/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.752095 from 218834 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.863300\"                                                                                                     \n",
       " [9] \"There were 228674 total reads, 228674 paired (226896 properly mated), 1646 proper singles, 132 improper reads (110 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 226896 reads, 226896 mapped, 0 unmapped, 226896 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 110 reads, 110 mapped, 0 unmapped, 110 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1646 reads, 1646 mapped, 0 unmapped, 1646 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 228674\"                                                                                                                           \n",
       "[27] \"Total placed reads: 228674\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13893\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 456.300000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12829391.286242\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/13/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[11]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/14/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//14/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.757812 from 483128 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.914564\"                                                                                                     \n",
       " [9] \"There were 503117 total reads, 503117 paired (501250 properly mated), 1605 proper singles, 262 improper reads (242 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 501250 reads, 501250 mapped, 0 unmapped, 501250 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 242 reads, 242 mapped, 0 unmapped, 242 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 503117\"                                                                                                                           \n",
       "[27] \"Total placed reads: 503117\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 16405\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 833.200000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13218885.840352\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[12]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/14/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//14/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.734157 from 477590 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.908243\"                                                                                                     \n",
       " [9] \"There were 497321 total reads, 497321 paired (495422 properly mated), 1527 proper singles, 372 improper reads (360 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 495422 reads, 495422 mapped, 0 unmapped, 495422 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 5 placed, 1 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 360 reads, 360 mapped, 0 unmapped, 360 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1527 reads, 1527 mapped, 0 unmapped, 1527 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 497321\"                                                                                                                           \n",
       "[27] \"Total placed reads: 497320\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 9978\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 625.501258\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12874444.057809\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/14/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[13]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/15/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//15/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.744798 from 573334 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.920983\"                                                                                                     \n",
       " [9] \"There were 596785 total reads, 596785 paired (594766 properly mated), 1773 proper singles, 246 improper reads (238 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 594766 reads, 594766 mapped, 0 unmapped, 594765 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 238 reads, 238 mapped, 0 unmapped, 238 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1773 reads, 1773 mapped, 0 unmapped, 1773 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 596785\"                                                                                                                           \n",
       "[27] \"Total placed reads: 596784\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 18442\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1030.801727\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12399446.201300\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[14]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/15/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//15/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.713940 from 565336 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.910377\"                                                                                                     \n",
       " [9] \"There were 588560 total reads, 588560 paired (586414 properly mated), 1778 proper singles, 368 improper reads (354 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 586414 reads, 586414 mapped, 0 unmapped, 586413 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1776 reads, 1776 mapped, 0 unmapped, 1776 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 588560\"                                                                                                                           \n",
       "[27] \"Total placed reads: 588559\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13012\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 728.101237\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -11815273.130115\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/15/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[15]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/16/megahit.bam\"                                                 \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//16/megahit/contigs_filtered.fasta\"             \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt\"                                          \n",
       " [4] \"Reading in assembly...\"                                                                                                                              \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                      \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                 \n",
       " [7] \"Found FR sample avg insert length to be 253.825277 from 258958 mapped reads\"                                                                         \n",
       " [8] \"Found FR sample insert length std to be 9.906877\"                                                                                                    \n",
       " [9] \"There were 270606 total reads, 270606 paired (268648 properly mated), 1852 proper singles, 106 improper reads (92 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt.param\"                         \n",
       "[11] \"Computing read placements and depths\"                                                                                                                \n",
       "[12] \"FR orientation with 268648 reads, 268648 mapped, 0 unmapped, 268648 placed, 0 mappedButNotPlaced\"                                                    \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[15] \"NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[19] \"CHIMER orientation with 92 reads, 92 mapped, 0 unmapped, 92 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1852 reads, 1852 mapped, 0 unmapped, 1852 placed, 0 mappedButNotPlaced\"                                        \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"Total mapped reads: 270606\"                                                                                                                          \n",
       "[27] \"Total placed reads: 270606\"                                                                                                                          \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                             \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                             \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                       \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                    \n",
       "[32] \"Computing depth statistics...\"                                                                                                                       \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                         \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                      \n",
       "[35] \"bases with 0 coverage: 21444\"                                                                                                                        \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                    \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                         \n",
       "[38] \"Expected extra length: 1181.000000\"                                                                                                                  \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                      \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                           \n",
       "[41] \"Total ALE Score: -14952829.516917\"                                                                                                                   \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/megahit.txt\"                                        \n",
       "[43] \"Closing input file\"                                                                                                                                  \n",
       "\n",
       "[[16]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/16/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//16/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.760708 from 250940 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.880949\"                                                                                                     \n",
       " [9] \"There were 261924 total reads, 261924 paired (260222 properly mated), 1560 proper singles, 142 improper reads (124 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 260222 reads, 260222 mapped, 0 unmapped, 260222 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 124 reads, 124 mapped, 0 unmapped, 124 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1560 reads, 1560 mapped, 0 unmapped, 1560 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 261924\"                                                                                                                           \n",
       "[27] \"Total placed reads: 261924\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 14252\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 611.900000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -14246895.113395\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/16/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[17]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/17/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//17/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.741113 from 421342 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.895415\"                                                                                                     \n",
       " [9] \"There were 439355 total reads, 439355 paired (437148 properly mated), 1841 proper singles, 366 improper reads (344 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 437148 reads, 437148 mapped, 0 unmapped, 437148 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 344 reads, 344 mapped, 0 unmapped, 344 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1835 reads, 1835 mapped, 0 unmapped, 1835 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 439355\"                                                                                                                           \n",
       "[27] \"Total placed reads: 439355\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 15450\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 978.200000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -15184661.003788\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[18]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/17/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//17/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.755768 from 413492 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.903851\"                                                                                                     \n",
       " [9] \"There were 431046 total reads, 431046 paired (428998 properly mated), 1648 proper singles, 400 improper reads (382 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 428998 reads, 428998 mapped, 0 unmapped, 428997 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 382 reads, 382 mapped, 0 unmapped, 382 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1648 reads, 1648 mapped, 0 unmapped, 1648 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 431046\"                                                                                                                           \n",
       "[27] \"Total placed reads: 431045\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 12471\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 683.801586\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -14668353.704128\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/17/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[19]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/18/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//18/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.749547 from 757308 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.932785\"                                                                                                     \n",
       " [9] \"There were 788345 total reads, 788345 paired (785754 properly mated), 1677 proper singles, 914 improper reads (828 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 785754 reads, 785754 mapped, 0 unmapped, 785754 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 828 reads, 828 mapped, 0 unmapped, 828 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1649 reads, 1649 mapped, 0 unmapped, 1649 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 788345\"                                                                                                                           \n",
       "[27] \"Total placed reads: 788345\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11841\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1212.800000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -11191721.107405\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[20]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/18/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//18/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.721260 from 750586 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.920155\"                                                                                                     \n",
       " [9] \"There were 781411 total reads, 781411 paired (778634 properly mated), 2063 proper singles, 714 improper reads (630 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 778634 reads, 778634 mapped, 0 unmapped, 778634 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 630 reads, 630 mapped, 0 unmapped, 630 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2059 reads, 2059 mapped, 0 unmapped, 2059 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 781411\"                                                                                                                           \n",
       "[27] \"Total placed reads: 781411\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 8211\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1198.400000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -10727524.033928\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/18/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[21]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/19/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//19/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.710925 from 439506 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.882234\"                                                                                                     \n",
       " [9] \"There were 458135 total reads, 458135 paired (455902 properly mated), 1979 proper singles, 254 improper reads (240 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 455902 reads, 455902 mapped, 0 unmapped, 455902 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 240 reads, 240 mapped, 0 unmapped, 240 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1979 reads, 1979 mapped, 0 unmapped, 1979 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 458135\"                                                                                                                           \n",
       "[27] \"Total placed reads: 458135\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 16123\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 998.700000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12704608.093983\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[22]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/19/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//19/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.685501 from 431938 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.869250\"                                                                                                     \n",
       " [9] \"There were 450048 total reads, 450048 paired (447962 properly mated), 1666 proper singles, 420 improper reads (404 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 447962 reads, 447962 mapped, 0 unmapped, 447962 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 404 reads, 404 mapped, 0 unmapped, 404 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1662 reads, 1662 mapped, 0 unmapped, 1662 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 450048\"                                                                                                                           \n",
       "[27] \"Total placed reads: 450048\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11611\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 640.800000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12183660.987856\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/19/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[23]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/2/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//2/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.763331 from 852556 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.933910\"                                                                                                     \n",
       " [9] \"There were 887324 total reads, 887324 paired (884688 properly mated), 1796 proper singles, 840 improper reads (764 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 884688 reads, 884688 mapped, 0 unmapped, 884688 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 764 reads, 764 mapped, 0 unmapped, 764 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1790 reads, 1790 mapped, 0 unmapped, 1790 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 887324\"                                                                                                                           \n",
       "[27] \"Total placed reads: 887324\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 14078\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1238.900000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -17187997.456175\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[24]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/2/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//2/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.757652 from 841888 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.932336\"                                                                                                     \n",
       " [9] \"There were 876686 total reads, 876686 paired (873584 properly mated), 2236 proper singles, 866 improper reads (796 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 873584 reads, 873584 mapped, 0 unmapped, 873584 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 796 reads, 796 mapped, 0 unmapped, 796 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2224 reads, 2224 mapped, 0 unmapped, 2224 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 876686\"                                                                                                                           \n",
       "[27] \"Total placed reads: 876686\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 9417\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1354.400000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16654318.964110\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/2/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[25]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/20/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//20/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.693648 from 546398 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.887225\"                                                                                                     \n",
       " [9] \"There were 568494 total reads, 568494 paired (566614 properly mated), 1594 proper singles, 286 improper reads (270 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 566614 reads, 566614 mapped, 0 unmapped, 566614 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 270 reads, 270 mapped, 0 unmapped, 270 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1578 reads, 1578 mapped, 0 unmapped, 1578 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 568494\"                                                                                                                           \n",
       "[27] \"Total placed reads: 568494\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13121\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 765.300000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16995916.356186\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[26]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/20/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//20/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.703542 from 539658 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.894556\"                                                                                                     \n",
       " [9] \"There were 561765 total reads, 561765 paired (559688 properly mated), 1663 proper singles, 414 improper reads (400 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 559688 reads, 559688 mapped, 0 unmapped, 559688 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 400 reads, 400 mapped, 0 unmapped, 400 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1663 reads, 1663 mapped, 0 unmapped, 1663 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 561765\"                                                                                                                           \n",
       "[27] \"Total placed reads: 561765\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 8043\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 857.700000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16611079.347268\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/20/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[27]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/21/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//21/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.770890 from 698024 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.935357\"                                                                                                     \n",
       " [9] \"There were 725969 total reads, 725969 paired (724254 properly mated), 1479 proper singles, 236 improper reads (214 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 724254 reads, 724254 mapped, 0 unmapped, 724253 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 214 reads, 214 mapped, 0 unmapped, 214 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1475 reads, 1475 mapped, 0 unmapped, 1475 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 725969\"                                                                                                                           \n",
       "[27] \"Total placed reads: 725968\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 8492\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 661.400911\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -15127024.327711\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[28]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/21/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//21/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.749190 from 693936 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.925717\"                                                                                                     \n",
       " [9] \"There were 721773 total reads, 721773 paired (719942 properly mated), 1481 proper singles, 350 improper reads (336 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 719942 reads, 719942 mapped, 0 unmapped, 719942 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1473 reads, 1473 mapped, 0 unmapped, 1473 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 721773\"                                                                                                                           \n",
       "[27] \"Total placed reads: 721773\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 5549\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 570.200000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -14809653.791573\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/21/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[29]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/22/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//22/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.741647 from 369982 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.879979\"                                                                                                     \n",
       " [9] \"There were 386029 total reads, 386029 paired (383718 properly mated), 2143 proper singles, 168 improper reads (142 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 383718 reads, 383718 mapped, 0 unmapped, 383718 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 142 reads, 142 mapped, 0 unmapped, 142 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2143 reads, 2143 mapped, 0 unmapped, 2143 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 386029\"                                                                                                                           \n",
       "[27] \"Total placed reads: 386029\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 17603\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 839.200000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12932315.161128\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[30]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/22/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//22/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.720616 from 364072 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.877073\"                                                                                                     \n",
       " [9] \"There were 379994 total reads, 379994 paired (377602 properly mated), 2056 proper singles, 336 improper reads (308 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 377602 reads, 377602 mapped, 0 unmapped, 377602 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 15 placed, 1 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 308 reads, 308 mapped, 0 unmapped, 308 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2056 reads, 2056 mapped, 0 unmapped, 2056 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 379994\"                                                                                                                           \n",
       "[27] \"Total placed reads: 379993\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11788\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 577.401520\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12470140.178258\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/22/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[31]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/23/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//23/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.677136 from 516186 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.887594\"                                                                                                     \n",
       " [9] \"There were 537539 total reads, 537539 paired (535338 properly mated), 1915 proper singles, 286 improper reads (246 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 535338 reads, 535338 mapped, 0 unmapped, 535337 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1897 reads, 1897 mapped, 0 unmapped, 1897 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 537539\"                                                                                                                           \n",
       "[27] \"Total placed reads: 537538\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 12951\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 844.901572\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13346764.513687\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[32]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/23/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//23/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.670893 from 510296 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.883901\"                                                                                                     \n",
       " [9] \"There were 531447 total reads, 531447 paired (529170 properly mated), 1821 proper singles, 456 improper reads (428 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 529170 reads, 529170 mapped, 0 unmapped, 529170 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 428 reads, 428 mapped, 0 unmapped, 428 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1807 reads, 1807 mapped, 0 unmapped, 1807 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 531447\"                                                                                                                           \n",
       "[27] \"Total placed reads: 531447\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 10525\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 683.400000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12996454.555808\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/23/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[33]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/24/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//24/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.677348 from 323618 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.860010\"                                                                                                     \n",
       " [9] \"There were 337511 total reads, 337511 paired (335530 properly mated), 1627 proper singles, 354 improper reads (326 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 335530 reads, 335530 mapped, 0 unmapped, 335530 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1627 reads, 1627 mapped, 0 unmapped, 1627 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 337511\"                                                                                                                           \n",
       "[27] \"Total placed reads: 337511\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11254\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 842.300000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -9919433.184678\"                                                                                                                     \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[34]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/24/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//24/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.714823 from 316610 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.875752\"                                                                                                     \n",
       " [9] \"There were 330503 total reads, 330503 paired (328358 properly mated), 1739 proper singles, 406 improper reads (370 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 328358 reads, 328358 mapped, 0 unmapped, 328358 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 370 reads, 370 mapped, 0 unmapped, 370 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1739 reads, 1739 mapped, 0 unmapped, 1739 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 330503\"                                                                                                                           \n",
       "[27] \"Total placed reads: 330503\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 10498\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 813.600000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -9515257.556246\"                                                                                                                     \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/24/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[35]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/25/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//25/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.700994 from 572804 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.896120\"                                                                                                     \n",
       " [9] \"There were 596261 total reads, 596261 paired (594004 properly mated), 1559 proper singles, 698 improper reads (676 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 594004 reads, 594004 mapped, 0 unmapped, 594004 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 676 reads, 676 mapped, 0 unmapped, 676 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1545 reads, 1545 mapped, 0 unmapped, 1545 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 596261\"                                                                                                                           \n",
       "[27] \"Total placed reads: 596261\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 12886\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1044.600000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -14853488.488456\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[36]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/25/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//25/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.703648 from 565516 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.903196\"                                                                                                     \n",
       " [9] \"There were 589534 total reads, 589534 paired (586468 properly mated), 2148 proper singles, 918 improper reads (872 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 586468 reads, 586468 mapped, 0 unmapped, 586468 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 872 reads, 872 mapped, 0 unmapped, 872 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2142 reads, 2142 mapped, 0 unmapped, 2142 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 589534\"                                                                                                                           \n",
       "[27] \"Total placed reads: 589534\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 10919\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1113.800000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -14503625.790964\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/25/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[37]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/26/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//26/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                        \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                   \n",
       " [7] \"Found FR sample avg insert length to be 253.848518 from 762942 mapped reads\"                                                                           \n",
       " [8] \"Found FR sample insert length std to be 9.981232\"                                                                                                      \n",
       " [9] \"There were 794498 total reads, 794498 paired (791910 properly mated), 1582 proper singles, 1006 improper reads (860 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                  \n",
       "[12] \"FR orientation with 791910 reads, 791910 mapped, 0 unmapped, 791903 placed, 7 mappedButNotPlaced\"                                                      \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                     \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                     \n",
       "[15] \"NOT_PROPER_FR orientation with 88 reads, 88 mapped, 0 unmapped, 88 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[16] \"NOT_PROPER_RF orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[17] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[19] \"CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1524 reads, 1524 mapped, 0 unmapped, 1524 placed, 0 mappedButNotPlaced\"                                          \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[23] \"SINGLE_READ orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                               \n",
       "[26] \"Total mapped reads: 794498\"                                                                                                                            \n",
       "[27] \"Total placed reads: 794491\"                                                                                                                            \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                               \n",
       "[29] \"Total unplaced reads: 7\"                                                                                                                               \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                         \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                      \n",
       "[32] \"Computing depth statistics...\"                                                                                                                         \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                           \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                        \n",
       "[35] \"bases with 0 coverage: 7176\"                                                                                                                           \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                      \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                           \n",
       "[38] \"Expected extra length: 1039.751222\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                        \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                             \n",
       "[41] \"Total ALE Score: -14384733.330159\"                                                                                                                     \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                    \n",
       "\n",
       "[[38]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/26/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//26/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.824811 from 758038 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.970790\"                                                                                                     \n",
       " [9] \"There were 790175 total reads, 790175 paired (786864 properly mated), 2387 proper singles, 924 improper reads (744 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 786864 reads, 786864 mapped, 0 unmapped, 786857 placed, 7 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 96 reads, 96 mapped, 0 unmapped, 96 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 744 reads, 744 mapped, 0 unmapped, 744 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2343 reads, 2343 mapped, 0 unmapped, 2343 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 790175\"                                                                                                                           \n",
       "[27] \"Total placed reads: 790168\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 7\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 5558\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1245.682832\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -14218317.825355\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/26/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[39]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/27/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//27/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.747729 from 484106 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.900189\"                                                                                                     \n",
       " [9] \"There were 504159 total reads, 504159 paired (502188 properly mated), 1653 proper singles, 318 improper reads (306 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 502188 reads, 502188 mapped, 0 unmapped, 502188 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 306 reads, 306 mapped, 0 unmapped, 306 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1651 reads, 1651 mapped, 0 unmapped, 1651 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 504159\"                                                                                                                           \n",
       "[27] \"Total placed reads: 504159\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 17255\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1064.600000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13516775.165159\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[40]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/27/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//27/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.716492 from 478160 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.887961\"                                                                                                     \n",
       " [9] \"There were 497996 total reads, 497996 paired (495980 properly mated), 1548 proper singles, 468 improper reads (448 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 495980 reads, 495980 mapped, 0 unmapped, 495980 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1548 reads, 1548 mapped, 0 unmapped, 1548 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 497996\"                                                                                                                           \n",
       "[27] \"Total placed reads: 497996\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 12965\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 694.300000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13118587.682537\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/27/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[41]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/28/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//28/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.771064 from 271272 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.901245\"                                                                                                     \n",
       " [9] \"There were 283284 total reads, 283284 paired (281252 properly mated), 1732 proper singles, 300 improper reads (300 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 281252 reads, 281252 mapped, 0 unmapped, 281252 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 300 reads, 300 mapped, 0 unmapped, 300 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1732 reads, 1732 mapped, 0 unmapped, 1732 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 283284\"                                                                                                                           \n",
       "[27] \"Total placed reads: 283284\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 12386\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 784.700000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -11165800.783435\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[42]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/28/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//28/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.749976 from 266078 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.891963\"                                                                                                     \n",
       " [9] \"There were 277610 total reads, 277610 paired (275828 properly mated), 1510 proper singles, 272 improper reads (264 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 275828 reads, 275828 mapped, 0 unmapped, 275828 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 264 reads, 264 mapped, 0 unmapped, 264 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1510 reads, 1510 mapped, 0 unmapped, 1510 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 277610\"                                                                                                                           \n",
       "[27] \"Total placed reads: 277610\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 9038\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 552.400000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -10806835.625985\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/28/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[43]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/29/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//29/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.730651 from 533398 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.910921\"                                                                                                     \n",
       " [9] \"There were 555957 total reads, 555957 paired (553314 properly mated), 1853 proper singles, 790 improper reads (718 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 553314 reads, 553314 mapped, 0 unmapped, 553311 placed, 3 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 718 reads, 718 mapped, 0 unmapped, 718 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1825 reads, 1825 mapped, 0 unmapped, 1825 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 555957\"                                                                                                                           \n",
       "[27] \"Total placed reads: 555954\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 3\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 15709\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1088.605874\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -11178993.697992\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[44]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/29/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//29/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.714380 from 527428 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.904824\"                                                                                                     \n",
       " [9] \"There were 549525 total reads, 549525 paired (547114 properly mated), 1749 proper singles, 662 improper reads (604 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 547114 reads, 547114 mapped, 0 unmapped, 547111 placed, 3 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1705 reads, 1705 mapped, 0 unmapped, 1705 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 549525\"                                                                                                                           \n",
       "[27] \"Total placed reads: 549522\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 3\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11532\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 873.104767\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -10770544.377923\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/29/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[45]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/3/megahit.bam\"                                                 \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//3/megahit/contigs_filtered.fasta\"             \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt\"                                          \n",
       " [4] \"Reading in assembly...\"                                                                                                                             \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                     \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                \n",
       " [7] \"Found FR sample avg insert length to be 253.748898 from 235904 mapped reads\"                                                                        \n",
       " [8] \"Found FR sample insert length std to be 9.865444\"                                                                                                   \n",
       " [9] \"There were 246262 total reads, 246262 paired (244704 properly mated), 1486 proper singles, 72 improper reads (66 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt.param\"                         \n",
       "[11] \"Computing read placements and depths\"                                                                                                               \n",
       "[12] \"FR orientation with 244704 reads, 244704 mapped, 0 unmapped, 244704 placed, 0 mappedButNotPlaced\"                                                   \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[15] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[16] \"NOT_PROPER_RF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[19] \"CHIMER orientation with 66 reads, 66 mapped, 0 unmapped, 66 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1486 reads, 1486 mapped, 0 unmapped, 1486 placed, 0 mappedButNotPlaced\"                                       \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[26] \"Total mapped reads: 246262\"                                                                                                                         \n",
       "[27] \"Total placed reads: 246262\"                                                                                                                         \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                            \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                            \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                      \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                   \n",
       "[32] \"Computing depth statistics...\"                                                                                                                      \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                        \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                     \n",
       "[35] \"bases with 0 coverage: 13408\"                                                                                                                       \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                   \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                        \n",
       "[38] \"Expected extra length: 750.300000\"                                                                                                                  \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                     \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                          \n",
       "[41] \"Total ALE Score: -12545384.151465\"                                                                                                                  \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/megahit.txt\"                                        \n",
       "[43] \"Closing input file\"                                                                                                                                 \n",
       "\n",
       "[[46]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/3/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//3/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.732209 from 229306 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.856694\"                                                                                                     \n",
       " [9] \"There were 239119 total reads, 239119 paired (237844 properly mated), 1169 proper singles, 106 improper reads (106 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 237844 reads, 237844 mapped, 0 unmapped, 237844 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 106 reads, 106 mapped, 0 unmapped, 106 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1167 reads, 1167 mapped, 0 unmapped, 1167 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 239119\"                                                                                                                           \n",
       "[27] \"Total placed reads: 239119\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 9975\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 368.200000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12030126.110063\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/3/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[47]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/30/megahit.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//30/megahit/contigs_filtered.fasta\"              \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.773617 from 607112 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.937958\"                                                                                                     \n",
       " [9] \"There were 631675 total reads, 631675 paired (629870 properly mated), 1685 proper singles, 120 improper reads (100 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 629870 reads, 629870 mapped, 0 unmapped, 629870 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 100 reads, 100 mapped, 0 unmapped, 100 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1683 reads, 1683 mapped, 0 unmapped, 1683 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 631675\"                                                                                                                           \n",
       "[27] \"Total placed reads: 631675\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 14930\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 830.300000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -26596142.638070\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/megahit.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[48]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/30/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//30/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.743582 from 600208 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.925605\"                                                                                                     \n",
       " [9] \"There were 624155 total reads, 624155 paired (622624 properly mated), 1393 proper singles, 138 improper reads (126 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 622624 reads, 622624 mapped, 0 unmapped, 622624 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 126 reads, 126 mapped, 0 unmapped, 126 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1391 reads, 1391 mapped, 0 unmapped, 1391 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 624155\"                                                                                                                           \n",
       "[27] \"Total placed reads: 624155\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 8854\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 510.800000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -25965655.555041\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/30/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[49]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/4/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//4/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.713039 from 379564 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.879604\"                                                                                                     \n",
       " [9] \"There were 396198 total reads, 396198 paired (393586 properly mated), 2148 proper singles, 464 improper reads (442 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 393586 reads, 393586 mapped, 0 unmapped, 393586 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 442 reads, 442 mapped, 0 unmapped, 442 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2144 reads, 2144 mapped, 0 unmapped, 2144 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 396198\"                                                                                                                           \n",
       "[27] \"Total placed reads: 396198\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 17254\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1255.500000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13656553.006202\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[50]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/4/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//4/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.707399 from 373622 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.880365\"                                                                                                     \n",
       " [9] \"There were 389836 total reads, 389836 paired (387414 properly mated), 1922 proper singles, 500 improper reads (484 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 387414 reads, 387414 mapped, 0 unmapped, 387414 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 484 reads, 484 mapped, 0 unmapped, 484 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1902 reads, 1902 mapped, 0 unmapped, 1902 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 389836\"                                                                                                                           \n",
       "[27] \"Total placed reads: 389836\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13180\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 872.500000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13194840.370122\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/4/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[51]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/5/megahit.bam\"                                                 \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//5/megahit/contigs_filtered.fasta\"             \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt\"                                          \n",
       " [4] \"Reading in assembly...\"                                                                                                                             \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                     \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                \n",
       " [7] \"Found FR sample avg insert length to be 253.696256 from 250718 mapped reads\"                                                                        \n",
       " [8] \"Found FR sample insert length std to be 9.875842\"                                                                                                   \n",
       " [9] \"There were 261393 total reads, 261393 paired (260022 properly mated), 1297 proper singles, 74 improper reads (62 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt.param\"                         \n",
       "[11] \"Computing read placements and depths\"                                                                                                               \n",
       "[12] \"FR orientation with 260022 reads, 260022 mapped, 0 unmapped, 260022 placed, 0 mappedButNotPlaced\"                                                   \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[15] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                    \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[19] \"CHIMER orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1293 reads, 1293 mapped, 0 unmapped, 1293 placed, 0 mappedButNotPlaced\"                                       \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[26] \"Total mapped reads: 261393\"                                                                                                                         \n",
       "[27] \"Total placed reads: 261393\"                                                                                                                         \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                            \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                            \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                      \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                   \n",
       "[32] \"Computing depth statistics...\"                                                                                                                      \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                        \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                     \n",
       "[35] \"bases with 0 coverage: 9797\"                                                                                                                        \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                   \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                        \n",
       "[38] \"Expected extra length: 616.500000\"                                                                                                                  \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                     \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                          \n",
       "[41] \"Total ALE Score: -9418117.983413\"                                                                                                                   \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/megahit.txt\"                                        \n",
       "[43] \"Closing input file\"                                                                                                                                 \n",
       "\n",
       "[[52]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/5/metaspades.bam\"                                               \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//5/metaspades/contigs_filtered.fasta\"           \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt\"                                        \n",
       " [4] \"Reading in assembly...\"                                                                                                                              \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                      \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                 \n",
       " [7] \"Found FR sample avg insert length to be 253.662871 from 246096 mapped reads\"                                                                         \n",
       " [8] \"Found FR sample insert length std to be 9.866825\"                                                                                                    \n",
       " [9] \"There were 256213 total reads, 256213 paired (255122 properly mated), 963 proper singles, 128 improper reads (112 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt.param\"                       \n",
       "[11] \"Computing read placements and depths\"                                                                                                                \n",
       "[12] \"FR orientation with 255122 reads, 255122 mapped, 0 unmapped, 255122 placed, 0 mappedButNotPlaced\"                                                    \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[15] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[16] \"NOT_PROPER_RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[19] \"CHIMER orientation with 112 reads, 112 mapped, 0 unmapped, 112 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 963 reads, 963 mapped, 0 unmapped, 963 placed, 0 mappedButNotPlaced\"                                           \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[23] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"Total mapped reads: 256213\"                                                                                                                          \n",
       "[27] \"Total placed reads: 256213\"                                                                                                                          \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                             \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                             \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                       \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                    \n",
       "[32] \"Computing depth statistics...\"                                                                                                                       \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                         \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                      \n",
       "[35] \"bases with 0 coverage: 6760\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                    \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                         \n",
       "[38] \"Expected extra length: 407.000000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                      \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                           \n",
       "[41] \"Total ALE Score: -9083426.562538\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/5/metaspades.txt\"                                      \n",
       "[43] \"Closing input file\"                                                                                                                                  \n",
       "\n",
       "[[53]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/6/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//6/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.767056 from 712204 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.927949\"                                                                                                     \n",
       " [9] \"There were 741088 total reads, 741088 paired (739100 properly mated), 1598 proper singles, 390 improper reads (332 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 739100 reads, 739100 mapped, 0 unmapped, 739100 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 332 reads, 332 mapped, 0 unmapped, 332 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1576 reads, 1576 mapped, 0 unmapped, 1576 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 741088\"                                                                                                                           \n",
       "[27] \"Total placed reads: 741088\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11967\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 798.000000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -16130340.567633\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[54]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/6/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//6/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.755147 from 707362 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.925815\"                                                                                                     \n",
       " [9] \"There were 736000 total reads, 736000 paired (734046 properly mated), 1576 proper singles, 378 improper reads (364 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 734046 reads, 734046 mapped, 0 unmapped, 734046 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 364 reads, 364 mapped, 0 unmapped, 364 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1572 reads, 1572 mapped, 0 unmapped, 1572 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 736000\"                                                                                                                           \n",
       "[27] \"Total placed reads: 736000\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 8499\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 662.500000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -15828310.751221\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/6/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[55]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/7/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//7/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.764770 from 572266 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.929275\"                                                                                                     \n",
       " [9] \"There were 595605 total reads, 595605 paired (593784 properly mated), 1605 proper singles, 216 improper reads (206 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 593784 reads, 593784 mapped, 0 unmapped, 593784 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 206 reads, 206 mapped, 0 unmapped, 206 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1603 reads, 1603 mapped, 0 unmapped, 1603 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 595605\"                                                                                                                           \n",
       "[27] \"Total placed reads: 595605\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13246\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 866.500000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13895857.636032\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[56]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/7/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//7/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.758549 from 567304 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.929236\"                                                                                                     \n",
       " [9] \"There were 590509 total reads, 590509 paired (588602 properly mated), 1555 proper singles, 352 improper reads (336 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 588602 reads, 588602 mapped, 0 unmapped, 588602 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 336 reads, 336 mapped, 0 unmapped, 336 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1549 reads, 1549 mapped, 0 unmapped, 1549 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 590509\"                                                                                                                           \n",
       "[27] \"Total placed reads: 590509\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 9107\"                                                                                                                          \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 595.600000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13582663.661134\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/7/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[57]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/8/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//8/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.789475 from 407582 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.908351\"                                                                                                     \n",
       " [9] \"There were 425560 total reads, 425560 paired (422904 properly mated), 2166 proper singles, 490 improper reads (456 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 422904 reads, 422904 mapped, 0 unmapped, 422904 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 456 reads, 456 mapped, 0 unmapped, 456 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2164 reads, 2164 mapped, 0 unmapped, 2164 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 425560\"                                                                                                                           \n",
       "[27] \"Total placed reads: 425560\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 15790\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1097.700000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13191614.984628\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[58]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/8/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//8/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.792128 from 398812 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.916498\"                                                                                                     \n",
       " [9] \"There were 416603 total reads, 416603 paired (413802 properly mated), 2269 proper singles, 532 improper reads (494 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 413802 reads, 413802 mapped, 0 unmapped, 413802 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 494 reads, 494 mapped, 0 unmapped, 494 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 2267 reads, 2267 mapped, 0 unmapped, 2267 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 416603\"                                                                                                                           \n",
       "[27] \"Total placed reads: 416603\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 11518\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 926.600000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -12711677.097005\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/8/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[59]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/9/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//9/megahit/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.705491 from 520154 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.900413\"                                                                                                     \n",
       " [9] \"There were 541958 total reads, 541958 paired (539498 properly mated), 1940 proper singles, 520 improper reads (470 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 539498 reads, 539498 mapped, 0 unmapped, 539497 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 470 reads, 470 mapped, 0 unmapped, 470 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1930 reads, 1930 mapped, 0 unmapped, 1930 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 541958\"                                                                                                                           \n",
       "[27] \"Total placed reads: 541957\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 15487\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 994.001834\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13988489.574853\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[60]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/map/9/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/assembly//9/metaspades/contigs_filtered.fasta\"            \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.722684 from 511928 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.907540\"                                                                                                     \n",
       " [9] \"There were 533364 total reads, 533364 paired (531020 properly mated), 1812 proper singles, 532 improper reads (506 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 531020 reads, 531020 mapped, 0 unmapped, 531020 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 506 reads, 506 mapped, 0 unmapped, 506 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1806 reads, 1806 mapped, 0 unmapped, 1806 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 533364\"                                                                                                                           \n",
       "[27] \"Total placed reads: 533364\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13274\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 848.700000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -13635354.189521\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/9/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# apply function on cluster\n",
    "job_ret = future_lapply(ALE_cmds, FUN = function(x) bash_job(x, conda_env=conda_env))\n",
    "job_ret"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Summarizing results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T08:05:14.240964Z",
     "start_time": "2019-07-23T08:05:14.170Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 60\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/1/metaspades.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/10/metaspades.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/train\\_runs/n1000\\_r30/ALE/11/metaspades.txt'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt'\n",
       "2. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt'\n",
       "3. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt'\n",
       "4. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt'\n",
       "5. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt'\n",
       "6. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/megahit.txt\"    \n",
       "[2] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/1/metaspades.txt\" \n",
       "[3] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/megahit.txt\"   \n",
       "[4] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/10/metaspades.txt\"\n",
       "[5] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/megahit.txt\"   \n",
       "[6] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE/11/metaspades.txt\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "ALE_res_files = list.files(work_dir, '*.txt$', recursive=TRUE, full.names=TRUE)\n",
    "ALE_res_files %>% length %>% print\n",
    "ALE_res_files %>% head"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T08:22:55.143491Z",
     "start_time": "2019-07-23T08:22:55.112Z"
    }
   },
   "outputs": [],
   "source": [
    "read_ALE_files = function(in_file){\n",
    "    # input command\n",
    "    cmd = 'egrep -v \"^#\" {in_file}'\n",
    "    cmd = glue::glue(cmd, in_file=in_file)\n",
    "    # reading in table\n",
    "    df = fread(cmd, sep=' ', header=FALSE)\n",
    "    # formatting\n",
    "    colnames(df) = c('contig', 'position', 'depth', \n",
    "                     'ln__depthLike', 'ln__placeLike', \n",
    "                     'ln__insertLike', 'ln__kmerLike')\n",
    "    df$Assembler = in_file %>% basename %>% gsub('\\\\.txt$', '', .)\n",
    "    df$Rep = in_file %>% dirname %>% basename\n",
    "    return(df)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T08:35:59.974150Z",
     "start_time": "2019-07-23T08:24:10.517Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 849228827\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>contig</th><th scope=col>position</th><th scope=col>depth</th><th scope=col>ln__depthLike</th><th scope=col>ln__placeLike</th><th scope=col>ln__insertLike</th><th scope=col>ln__kmerLike</th><th scope=col>Assembler</th><th scope=col>Rep</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>0      </td><td>0      </td><td>1      </td><td>-2.601 </td><td>-0.007 </td><td>0.32   </td><td>-0.198 </td><td>megahit</td><td>1      </td></tr>\n",
       "\t<tr><td>0      </td><td>1      </td><td>1      </td><td>-2.601 </td><td>-0.007 </td><td>0.32   </td><td>-0.420 </td><td>megahit</td><td>1      </td></tr>\n",
       "\t<tr><td>0      </td><td>2      </td><td>1      </td><td>-2.601 </td><td>-0.007 </td><td>0.32   </td><td>-0.393 </td><td>megahit</td><td>1      </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllllllll}\n",
       " contig & position & depth & ln\\_\\_depthLike & ln\\_\\_placeLike & ln\\_\\_insertLike & ln\\_\\_kmerLike & Assembler & Rep\\\\\n",
       "\\hline\n",
       "\t 0       & 0       & 1       & -2.601  & -0.007  & 0.32    & -0.198  & megahit & 1      \\\\\n",
       "\t 0       & 1       & 1       & -2.601  & -0.007  & 0.32    & -0.420  & megahit & 1      \\\\\n",
       "\t 0       & 2       & 1       & -2.601  & -0.007  & 0.32    & -0.393  & megahit & 1      \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "contig | position | depth | ln__depthLike | ln__placeLike | ln__insertLike | ln__kmerLike | Assembler | Rep | \n",
       "|---|---|---|\n",
       "| 0       | 0       | 1       | -2.601  | -0.007  | 0.32    | -0.198  | megahit | 1       | \n",
       "| 0       | 1       | 1       | -2.601  | -0.007  | 0.32    | -0.420  | megahit | 1       | \n",
       "| 0       | 2       | 1       | -2.601  | -0.007  | 0.32    | -0.393  | megahit | 1       | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  contig position depth ln__depthLike ln__placeLike ln__insertLike ln__kmerLike\n",
       "1 0      0        1     -2.601        -0.007        0.32           -0.198      \n",
       "2 0      1        1     -2.601        -0.007        0.32           -0.420      \n",
       "3 0      2        1     -2.601        -0.007        0.32           -0.393      \n",
       "  Assembler Rep\n",
       "1 megahit   1  \n",
       "2 megahit   1  \n",
       "3 megahit   1  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "doParallel::registerDoParallel(threads)\n",
    "ALE_res = plyr::llply(as.list(ALE_res_files), read_ALE_files, .parallel=TRUE) %>%\n",
    "    do.call(rbind, .)\n",
    "\n",
    "ALE_res %>% nrow %>% print\n",
    "ALE_res %>% head(n=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Writing combined ALE results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:51:54.598741Z",
     "start_time": "2019-07-23T08:25:47.056Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File written: /ebio/abt3_projects/databases_no-backup/DeepMAsED/train_runs/n1000_r30/ALE//ALE_results.tsv \n"
     ]
    }
   ],
   "source": [
    "out_file = file.path(work_dir, 'ALE_results.tsv')\n",
    "ALE_res %>%\n",
    "    write.table(file=out_file, sep='\\t', quote=FALSE, row.names=FALSE)\n",
    "cat('File written:', out_file, '\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# sessionInfo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:54:20.003343Z",
     "start_time": "2019-07-23T10:54:06.979Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 3.4.1 (2017-06-30)\n",
       "Platform: x86_64-pc-linux-gnu (64-bit)\n",
       "Running under: Ubuntu 18.04.1 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS: /ebio/abt3_projects/software/miniconda3_gt4.4/envs/py3_physeq/lib/R/lib/libRblas.so\n",
       "LAPACK: /ebio/abt3_projects/software/miniconda3_gt4.4/envs/py3_physeq/lib/R/lib/libRlapack.so\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats     graphics  grDevices utils     datasets  methods  \n",
       "[8] base     \n",
       "\n",
       "other attached packages:\n",
       " [1] doParallel_1.0.11       iterators_1.0.10        foreach_1.4.4          \n",
       " [4] data.table_1.11.4       bindrcpp_0.2.2          future.apply_1.0.0     \n",
       " [7] future.batchtools_0.7.1 future_1.9.0            ggplot2_3.0.0          \n",
       "[10] tidyr_0.8.1             dplyr_0.7.6            \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       " [1] pbdZMQ_0.3-2      progress_1.2.0    tidyselect_0.2.4  repr_0.15.0      \n",
       " [5] purrr_0.2.4       listenv_0.7.0     colorspace_1.3-2  htmltools_0.3.6  \n",
       " [9] base64enc_0.1-3   rlang_0.2.1       pillar_1.2.2      glue_1.3.0       \n",
       "[13] withr_2.1.1       rappdirs_0.3.1    uuid_0.1-2        bindr_0.1.1      \n",
       "[17] plyr_1.8.4        stringr_1.3.1     munsell_0.5.0     gtable_0.2.0     \n",
       "[21] codetools_0.2-15  evaluate_0.10.1   batchtools_0.9.11 IRdisplay_0.4.4  \n",
       "[25] Rcpp_0.12.17      scales_0.5.0      backports_1.1.2   checkmate_1.8.5  \n",
       "[29] IRkernel_0.8.12   jsonlite_1.5      fs_1.2.6          brew_1.0-6       \n",
       "[33] hms_0.4.2         digest_0.6.15     stringi_1.2.3     grid_3.4.1       \n",
       "[37] tools_3.4.1       magrittr_1.5      base64url_1.4     lazyeval_0.2.1   \n",
       "[41] tibble_1.4.2      crayon_1.3.4      pkgconfig_2.0.1   prettyunits_1.0.2\n",
       "[45] assertthat_0.2.0  R6_2.2.2          globals_0.12.2    compiler_3.4.1   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R [conda env:py3_physeq]",
   "language": "R",
   "name": "conda-env-py3_physeq-r"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.4.1"
  },
  "notify_time": "30",
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": true,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
