{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "toc": true
   },
   "source": [
    "<h1>Table of Contents<span class=\"tocSkip\"></span></h1>\n",
    "<div class=\"toc\"><ul class=\"toc-item\"><li><span><a href=\"#Goal\" data-toc-modified-id=\"Goal-1\"><span class=\"toc-item-num\">1&nbsp;&nbsp;</span>Goal</a></span></li><li><span><a href=\"#Var\" data-toc-modified-id=\"Var-2\"><span class=\"toc-item-num\">2&nbsp;&nbsp;</span>Var</a></span></li><li><span><a href=\"#Init\" data-toc-modified-id=\"Init-3\"><span class=\"toc-item-num\">3&nbsp;&nbsp;</span>Init</a></span></li><li><span><a href=\"#Load\" data-toc-modified-id=\"Load-4\"><span class=\"toc-item-num\">4&nbsp;&nbsp;</span>Load</a></span></li><li><span><a href=\"#Running-ALE\" data-toc-modified-id=\"Running-ALE-5\"><span class=\"toc-item-num\">5&nbsp;&nbsp;</span>Running ALE</a></span></li><li><span><a href=\"#Summarizing-results\" data-toc-modified-id=\"Summarizing-results-6\"><span class=\"toc-item-num\">6&nbsp;&nbsp;</span>Summarizing results</a></span><ul class=\"toc-item\"><li><span><a href=\"#Writing-combined-ALE-results\" data-toc-modified-id=\"Writing-combined-ALE-results-6.1\"><span class=\"toc-item-num\">6.1&nbsp;&nbsp;</span>Writing combined ALE results</a></span></li></ul></li><li><span><a href=\"#---WAITING---\" data-toc-modified-id=\"---WAITING----7\"><span class=\"toc-item-num\">7&nbsp;&nbsp;</span>-- WAITING --</a></span></li><li><span><a href=\"#sessionInfo\" data-toc-modified-id=\"sessionInfo-8\"><span class=\"toc-item-num\">8&nbsp;&nbsp;</span>sessionInfo</a></span></li></ul></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Goal\n",
    "\n",
    "* run ALE on simulated test dataset for DeepMAsED \n",
    "  * the ground truth results can be used for selecting cutoffs for ALE and assessing accuracy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Var"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:56:54.977052Z",
     "start_time": "2019-07-23T10:56:54.509Z"
    }
   },
   "outputs": [],
   "source": [
    "work_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE/'\n",
    "\n",
    "# MAGs for genome analysis\n",
    "asmbl_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly/'\n",
    "\n",
    "# bam file dir (mapping of metagenome reads to corresponding MAGs)\n",
    "map_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map/'\n",
    "\n",
    "# ALE executable\n",
    "ALE_exe = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE'\n",
    "\n",
    "# params\n",
    "conda_env = 'base'\n",
    "threads = 12"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Init"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:56:57.597629Z",
     "start_time": "2019-07-23T10:56:55.530Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "Attaching package: ‘dplyr’\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    filter, lag\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    intersect, setdiff, setequal, union\n",
      "\n",
      "\n",
      "Attaching package: ‘data.table’\n",
      "\n",
      "The following objects are masked from ‘package:dplyr’:\n",
      "\n",
      "    between, first, last\n",
      "\n",
      "Loading required package: foreach\n",
      "Loading required package: iterators\n",
      "Loading required package: parallel\n",
      "\n",
      "Attaching package: ‘future.apply’\n",
      "\n",
      "The following object is masked from ‘package:future’:\n",
      "\n",
      "    future_lapply\n",
      "\n"
     ]
    }
   ],
   "source": [
    "library(dplyr)\n",
    "library(tidyr)\n",
    "library(ggplot2)\n",
    "library(data.table)\n",
    "library(doParallel)\n",
    "\n",
    "library(future)\n",
    "library(future.batchtools)\n",
    "library(future.apply)\n",
    "options(future.wait.interval = 2.0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Load"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:56:58.195427Z",
     "start_time": "2019-07-23T10:56:56.685Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 10\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                          </td><td>1                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta   </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                       </td><td>1                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta</td></tr>\n",
       "\t<tr><td>megahit                                                                                                          </td><td>2                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta   </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lll}\n",
       " Assembler & Rep & Fasta\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                     & 1                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/megahit/contigs\\_filtered.fasta   \\\\\n",
       "\t metaspades                                                                                                                  & 1                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/metaspades/contigs\\_filtered.fasta\\\\\n",
       "\t megahit                                                                                                                     & 2                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//2/megahit/contigs\\_filtered.fasta   \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                           | 1                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta    | \n",
       "| metaspades                                                                                                        | 1                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta | \n",
       "| megahit                                                                                                           | 2                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta    | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    2  \n",
       "  Fasta                                                                                                            \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# adding bam files\n",
    "fasta_files = list.files(asmbl_dir, '*.fasta$', recursive=TRUE, full.names=TRUE)\n",
    "fasta_files = data.frame(Assembler = fasta_files %>% dirname %>% basename,\n",
    "                         Rep = fasta_files %>% dirname %>% dirname %>% basename,\n",
    "                         Fasta = fasta_files)\n",
    "\n",
    "fasta_files %>% nrow %>% print\n",
    "fasta_files %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:56:58.474709Z",
     "start_time": "2019-07-23T10:56:57.489Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 10\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Bam</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                  </td><td>1                                                                                        </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam   </td></tr>\n",
       "\t<tr><td>metaspades                                                                               </td><td>1                                                                                        </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam</td></tr>\n",
       "\t<tr><td>megahit                                                                                  </td><td>2                                                                                        </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam   </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lll}\n",
       " Assembler & Rep & Bam\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                           & 1                                                                                                 & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/megahit.bam   \\\\\n",
       "\t metaspades                                                                                        & 1                                                                                                 & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/metaspades.bam\\\\\n",
       "\t megahit                                                                                           & 2                                                                                                 & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//2/megahit.bam   \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Bam | \n",
       "|---|---|---|\n",
       "| megahit                                                                                   | 1                                                                                         | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam    | \n",
       "| metaspades                                                                                | 1                                                                                         | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam | \n",
       "| megahit                                                                                   | 2                                                                                         | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam    | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    2  \n",
       "  Bam                                                                                      \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# adding bam files\n",
    "bam_files = list.files(map_dir, '*.bam$', recursive=TRUE, full.names=TRUE)\n",
    "bam_files = data.frame(Assembler = bam_files %>% basename %>% gsub('\\\\.bam$', '', .),\n",
    "                       Rep = bam_files %>% dirname %>% basename,\n",
    "                       Bam = bam_files)\n",
    "\n",
    "bam_files %>% nrow %>% print\n",
    "bam_files %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:56:58.575424Z",
     "start_time": "2019-07-23T10:56:57.762Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 10\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th><th scope=col>Bam</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                          </td><td>1                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam                           </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                       </td><td>1                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta</td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam                        </td></tr>\n",
       "\t<tr><td>megahit                                                                                                          </td><td>2                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam                           </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|llll}\n",
       " Assembler & Rep & Fasta & Bam\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                     & 1                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/megahit.bam                             \\\\\n",
       "\t metaspades                                                                                                                  & 1                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/metaspades/contigs\\_filtered.fasta & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/metaspades.bam                          \\\\\n",
       "\t megahit                                                                                                                     & 2                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//2/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//2/megahit.bam                             \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | Bam | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                           | 1                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam                            | \n",
       "| metaspades                                                                                                        | 1                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam                         | \n",
       "| megahit                                                                                                           | 2                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam                            | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    2  \n",
       "  Fasta                                                                                                            \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta   \n",
       "  Bam                                                                                      \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# joining\n",
    "MAGs = fasta_files %>%\n",
    "    inner_join(bam_files, c('Assembler', 'Rep'))\n",
    "MAGs %>% nrow %>% print\n",
    "MAGs %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:56:58.922789Z",
     "start_time": "2019-07-23T10:56:58.772Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th><th scope=col>Bam</th><th scope=col>ALE</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                          </td><td>1                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam                           </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt                           </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                       </td><td>1                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta</td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam                        </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt                        </td></tr>\n",
       "\t<tr><td>megahit                                                                                                          </td><td>2                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam                           </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt                           </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllll}\n",
       " Assembler & Rep & Fasta & Bam & ALE\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                     & 1                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/megahit.bam                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/megahit.txt                             \\\\\n",
       "\t metaspades                                                                                                                  & 1                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/metaspades/contigs\\_filtered.fasta & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/metaspades.bam                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/metaspades.txt                          \\\\\n",
       "\t megahit                                                                                                                     & 2                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//2/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//2/megahit.bam                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/megahit.txt                             \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | Bam | ALE | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                           | 1                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam                            | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt                            | \n",
       "| metaspades                                                                                                        | 1                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam                         | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt                         | \n",
       "| megahit                                                                                                           | 2                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam                            | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt                            | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    2  \n",
       "  Fasta                                                                                                            \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta   \n",
       "  Bam                                                                                      \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam   \n",
       "  ALE                                                                                      \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# ALE output file name\n",
    "MAGs = MAGs %>%\n",
    "    mutate(ALE = mapply(function(A, R) file.path(work_dir, R, paste0(A, '.txt')),\n",
    "                        A=Assembler, R=Rep))\n",
    "                        \n",
    "MAGs %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:57:00.379470Z",
     "start_time": "2019-07-23T10:57:00.347Z"
    }
   },
   "outputs": [],
   "source": [
    "# making output directories\n",
    "ret = MAGs$ALE %>% \n",
    "    dirname() %>%\n",
    "    unique() %>%\n",
    "    sapply(., function(x) dir.create(x, showWarnings=FALSE))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Running ALE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:57:24.975302Z",
     "start_time": "2019-07-23T10:57:24.891Z"
    }
   },
   "outputs": [],
   "source": [
    "# cluster resources\n",
    "resources = list(h_rt = '00:59:00',\n",
    "                 h_vmem = '8G',\n",
    "                 threads = '1',\n",
    "                 conda.env = 'py3_physeq')     # conda env with batchtools installed\n",
    "plan(batchtools_sge, resources=resources, workers=30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:57:25.185773Z",
     "start_time": "2019-07-23T10:57:25.089Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 10\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "'/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'"
      ],
      "text/latex": [
       "'/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/megahit.bam /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/megahit/contigs\\_filtered.fasta /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/megahit.txt'"
      ],
      "text/markdown": [
       "'/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'"
      ],
      "text/plain": [
       "[1] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# creating commands for calling ALE \n",
    "ALE_params = function(x, exe){\n",
    "    cmd = '{exe} {bam} {fasta} {outfile}'\n",
    "    cmd = glue::glue(cmd, exe=exe, bam=x[['Bam']], fasta=x[['Fasta']], outfile=x[['ALE']])\n",
    "    return(cmd)\n",
    "}\n",
    "\n",
    "ALE_cmds = apply(MAGs, 1, ALE_params, exe=ALE_exe) %>%\n",
    "    as.list\n",
    "\n",
    "ALE_cmds %>% length %>% print\n",
    "ALE_cmds[[1]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T10:57:25.323203Z",
     "start_time": "2019-07-23T10:57:25.289Z"
    }
   },
   "outputs": [],
   "source": [
    "# general bash job function\n",
    "bash_job = function(cmd, conda_env, stdout=TRUE, stderr=FALSE){\n",
    "    # cmd : string; commandline job (eg., 'ls -thlc')\n",
    "    # conda_env : string; conda environment name\n",
    "    cmd = sprintf('. ~/.bashrc; conda activate %s; %s', conda_env, cmd)\n",
    "    cmd = sprintf('-c \"%s\"', cmd)\n",
    "    system2('bash', cmd, stdout=stdout, stderr=stderr)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T11:01:09.403432Z",
     "start_time": "2019-07-23T10:57:25.722Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.728772 from 1090912 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.916796'</li>\n",
       "\t<li>'There were 1135136 total reads, 1135136 paired (1131604 properly mated), 2300 proper singles, 1232 improper reads (1190 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1131604 reads, 1131604 mapped, 0 unmapped, 1131604 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1190 reads, 1190 mapped, 0 unmapped, 1190 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2278 reads, 2278 mapped, 0 unmapped, 2278 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1135136'</li>\n",
       "\t<li>'Total placed reads: 1135136'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19449'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1725.100000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21341424.829069'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.732094 from 1076982 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.919110'</li>\n",
       "\t<li>'There were 1121157 total reads, 1121157 paired (1117228 properly mated), 2707 proper singles, 1222 improper reads (1182 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1117228 reads, 1117228 mapped, 0 unmapped, 1117228 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1182 reads, 1182 mapped, 0 unmapped, 1182 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2705 reads, 2705 mapped, 0 unmapped, 2705 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1121157'</li>\n",
       "\t<li>'Total placed reads: 1121157'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14176'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1816.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -20576586.864308'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.733571 from 928224 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.907996'</li>\n",
       "\t<li>'There were 966493 total reads, 966493 paired (962814 properly mated), 3041 proper singles, 638 improper reads (592 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 962814 reads, 962814 mapped, 0 unmapped, 962814 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 592 reads, 592 mapped, 0 unmapped, 592 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3031 reads, 3031 mapped, 0 unmapped, 3031 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 966493'</li>\n",
       "\t<li>'Total placed reads: 966493'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 28327'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1641.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -25478193.858699'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.703599 from 913356 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.897229'</li>\n",
       "\t<li>'There were 951219 total reads, 951219 paired (947314 properly mated), 3097 proper singles, 808 improper reads (760 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 947314 reads, 947314 mapped, 0 unmapped, 947314 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 29 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 760 reads, 760 mapped, 0 unmapped, 760 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3093 reads, 3093 mapped, 0 unmapped, 3093 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 951219'</li>\n",
       "\t<li>'Total placed reads: 951218'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19554'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1253.001317'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -24557003.672984'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//3/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//3/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.723209 from 1264320 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.916738'</li>\n",
       "\t<li>'There were 1314913 total reads, 1314913 paired (1311512 properly mated), 2789 proper singles, 612 improper reads (552 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1311512 reads, 1311512 mapped, 0 unmapped, 1311512 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 552 reads, 552 mapped, 0 unmapped, 552 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2769 reads, 2769 mapped, 0 unmapped, 2769 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1314913'</li>\n",
       "\t<li>'Total placed reads: 1314913'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 33103'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1860.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -23376993.706206'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//3/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//3/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.700152 from 1250640 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.905588'</li>\n",
       "\t<li>'There were 1300973 total reads, 1300973 paired (1297176 properly mated), 2867 proper singles, 930 improper reads (836 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1297176 reads, 1297176 mapped, 0 unmapped, 1297176 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2857 reads, 2857 mapped, 0 unmapped, 2857 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1300973'</li>\n",
       "\t<li>'Total placed reads: 1300973'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19826'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1405.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22645031.366919'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//4/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//4/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.733832 from 1154228 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.930490'</li>\n",
       "\t<li>'There were 1200825 total reads, 1200825 paired (1197536 properly mated), 2001 proper singles, 1288 improper reads (1204 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1197536 reads, 1197536 mapped, 0 unmapped, 1197536 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1985 reads, 1985 mapped, 0 unmapped, 1985 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1200825'</li>\n",
       "\t<li>'Total placed reads: 1200825'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16930'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1568.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15306701.076942'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//4/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//4/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.739542 from 1143486 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.934216'</li>\n",
       "\t<li>'There were 1190695 total reads, 1190695 paired (1186380 properly mated), 2967 proper singles, 1348 improper reads (1240 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1186380 reads, 1186380 mapped, 0 unmapped, 1186380 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1240 reads, 1240 mapped, 0 unmapped, 1240 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2949 reads, 2949 mapped, 0 unmapped, 2949 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1190695'</li>\n",
       "\t<li>'Total placed reads: 1190695'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11768'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1899.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14888272.486862'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//5/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//5/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.737512 from 1062958 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.924341'</li>\n",
       "\t<li>'There were 1105900 total reads, 1105900 paired (1102664 properly mated), 2398 proper singles, 838 improper reads (782 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1102664 reads, 1102664 mapped, 0 unmapped, 1102664 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 782 reads, 782 mapped, 0 unmapped, 782 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2392 reads, 2392 mapped, 0 unmapped, 2392 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1105900'</li>\n",
       "\t<li>'Total placed reads: 1105900'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15980'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1359.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -25110464.260699'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//5/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//5/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.753972 from 1049684 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.931368'</li>\n",
       "\t<li>'There were 1092573 total reads, 1092573 paired (1089006 properly mated), 2821 proper singles, 746 improper reads (694 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1089006 reads, 1089006 mapped, 0 unmapped, 1089006 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 694 reads, 694 mapped, 0 unmapped, 694 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2813 reads, 2813 mapped, 0 unmapped, 2813 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1092573'</li>\n",
       "\t<li>'Total placed reads: 1092573'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 10146'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1370.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -24422782.140910'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate}\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.728772 from 1090912 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.916796'\n",
       "\\item 'There were 1135136 total reads, 1135136 paired (1131604 properly mated), 2300 proper singles, 1232 improper reads (1190 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1131604 reads, 1131604 mapped, 0 unmapped, 1131604 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1190 reads, 1190 mapped, 0 unmapped, 1190 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2278 reads, 2278 mapped, 0 unmapped, 2278 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1135136'\n",
       "\\item 'Total placed reads: 1135136'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19449'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1725.100000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21341424.829069'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//1/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//1/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.732094 from 1076982 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.919110'\n",
       "\\item 'There were 1121157 total reads, 1121157 paired (1117228 properly mated), 2707 proper singles, 1222 improper reads (1182 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1117228 reads, 1117228 mapped, 0 unmapped, 1117228 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1182 reads, 1182 mapped, 0 unmapped, 1182 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2705 reads, 2705 mapped, 0 unmapped, 2705 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1121157'\n",
       "\\item 'Total placed reads: 1121157'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14176'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1816.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -20576586.864308'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//2/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//2/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.733571 from 928224 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.907996'\n",
       "\\item 'There were 966493 total reads, 966493 paired (962814 properly mated), 3041 proper singles, 638 improper reads (592 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 962814 reads, 962814 mapped, 0 unmapped, 962814 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 592 reads, 592 mapped, 0 unmapped, 592 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3031 reads, 3031 mapped, 0 unmapped, 3031 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 966493'\n",
       "\\item 'Total placed reads: 966493'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 28327'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1641.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -25478193.858699'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//2/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//2/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.703599 from 913356 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.897229'\n",
       "\\item 'There were 951219 total reads, 951219 paired (947314 properly mated), 3097 proper singles, 808 improper reads (760 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 947314 reads, 947314 mapped, 0 unmapped, 947314 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 30 reads, 30 mapped, 0 unmapped, 29 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 760 reads, 760 mapped, 0 unmapped, 760 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3093 reads, 3093 mapped, 0 unmapped, 3093 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 951219'\n",
       "\\item 'Total placed reads: 951218'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19554'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1253.001317'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -24557003.672984'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//3/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//3/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.723209 from 1264320 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.916738'\n",
       "\\item 'There were 1314913 total reads, 1314913 paired (1311512 properly mated), 2789 proper singles, 612 improper reads (552 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1311512 reads, 1311512 mapped, 0 unmapped, 1311512 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 552 reads, 552 mapped, 0 unmapped, 552 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2769 reads, 2769 mapped, 0 unmapped, 2769 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1314913'\n",
       "\\item 'Total placed reads: 1314913'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 33103'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1860.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -23376993.706206'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//3/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//3/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.700152 from 1250640 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.905588'\n",
       "\\item 'There were 1300973 total reads, 1300973 paired (1297176 properly mated), 2867 proper singles, 930 improper reads (836 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1297176 reads, 1297176 mapped, 0 unmapped, 1297176 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2857 reads, 2857 mapped, 0 unmapped, 2857 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1300973'\n",
       "\\item 'Total placed reads: 1300973'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19826'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1405.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22645031.366919'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//4/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//4/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//4/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.733832 from 1154228 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.930490'\n",
       "\\item 'There were 1200825 total reads, 1200825 paired (1197536 properly mated), 2001 proper singles, 1288 improper reads (1204 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//4/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1197536 reads, 1197536 mapped, 0 unmapped, 1197536 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1985 reads, 1985 mapped, 0 unmapped, 1985 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1200825'\n",
       "\\item 'Total placed reads: 1200825'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16930'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1568.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15306701.076942'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//4/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//4/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//4/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//4/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.739542 from 1143486 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.934216'\n",
       "\\item 'There were 1190695 total reads, 1190695 paired (1186380 properly mated), 2967 proper singles, 1348 improper reads (1240 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//4/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1186380 reads, 1186380 mapped, 0 unmapped, 1186380 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1240 reads, 1240 mapped, 0 unmapped, 1240 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2949 reads, 2949 mapped, 0 unmapped, 2949 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1190695'\n",
       "\\item 'Total placed reads: 1190695'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11768'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1899.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14888272.486862'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//4/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//5/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//5/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//5/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.737512 from 1062958 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.924341'\n",
       "\\item 'There were 1105900 total reads, 1105900 paired (1102664 properly mated), 2398 proper singles, 838 improper reads (782 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//5/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1102664 reads, 1102664 mapped, 0 unmapped, 1102664 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 782 reads, 782 mapped, 0 unmapped, 782 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2392 reads, 2392 mapped, 0 unmapped, 2392 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1105900'\n",
       "\\item 'Total placed reads: 1105900'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15980'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1359.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -25110464.260699'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//5/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/map//5/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/assembly//5/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//5/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.753972 from 1049684 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.931368'\n",
       "\\item 'There were 1092573 total reads, 1092573 paired (1089006 properly mated), 2821 proper singles, 746 improper reads (694 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//5/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1089006 reads, 1089006 mapped, 0 unmapped, 1089006 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 694 reads, 694 mapped, 0 unmapped, 694 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2813 reads, 2813 mapped, 0 unmapped, 2813 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1092573'\n",
       "\\item 'Total placed reads: 1092573'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 10146'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1370.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -24422782.140910'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//5/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\end{enumerate}\n"
      ],
      "text/markdown": [
       "1. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.728772 from 1090912 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.916796'\n",
       "10. 'There were 1135136 total reads, 1135136 paired (1131604 properly mated), 2300 proper singles, 1232 improper reads (1190 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1131604 reads, 1131604 mapped, 0 unmapped, 1131604 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1190 reads, 1190 mapped, 0 unmapped, 1190 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2278 reads, 2278 mapped, 0 unmapped, 2278 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1135136'\n",
       "29. 'Total placed reads: 1135136'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 19449'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1725.100000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21341424.829069'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "2. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.732094 from 1076982 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.919110'\n",
       "10. 'There were 1121157 total reads, 1121157 paired (1117228 properly mated), 2707 proper singles, 1222 improper reads (1182 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1117228 reads, 1117228 mapped, 0 unmapped, 1117228 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1182 reads, 1182 mapped, 0 unmapped, 1182 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2705 reads, 2705 mapped, 0 unmapped, 2705 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1121157'\n",
       "29. 'Total placed reads: 1121157'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14176'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1816.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -20576586.864308'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "3. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.733571 from 928224 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.907996'\n",
       "9. 'There were 966493 total reads, 966493 paired (962814 properly mated), 3041 proper singles, 638 improper reads (592 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 962814 reads, 962814 mapped, 0 unmapped, 962814 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 592 reads, 592 mapped, 0 unmapped, 592 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 3031 reads, 3031 mapped, 0 unmapped, 3031 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 966493'\n",
       "27. 'Total placed reads: 966493'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 28327'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1641.300000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -25478193.858699'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "4. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.703599 from 913356 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.897229'\n",
       "9. 'There were 951219 total reads, 951219 paired (947314 properly mated), 3097 proper singles, 808 improper reads (760 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 947314 reads, 947314 mapped, 0 unmapped, 947314 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 29 placed, 1 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 760 reads, 760 mapped, 0 unmapped, 760 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 3093 reads, 3093 mapped, 0 unmapped, 3093 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 951219'\n",
       "27. 'Total placed reads: 951218'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 1'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 19554'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1253.001317'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -24557003.672984'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "5. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//3/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//3/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.723209 from 1264320 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.916738'\n",
       "10. 'There were 1314913 total reads, 1314913 paired (1311512 properly mated), 2789 proper singles, 612 improper reads (552 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1311512 reads, 1311512 mapped, 0 unmapped, 1311512 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 552 reads, 552 mapped, 0 unmapped, 552 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2769 reads, 2769 mapped, 0 unmapped, 2769 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1314913'\n",
       "29. 'Total placed reads: 1314913'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 33103'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1860.700000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -23376993.706206'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "6. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//3/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//3/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.700152 from 1250640 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.905588'\n",
       "10. 'There were 1300973 total reads, 1300973 paired (1297176 properly mated), 2867 proper singles, 930 improper reads (836 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1297176 reads, 1297176 mapped, 0 unmapped, 1297176 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2857 reads, 2857 mapped, 0 unmapped, 2857 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1300973'\n",
       "29. 'Total placed reads: 1300973'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 19826'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1405.300000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22645031.366919'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "7. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//4/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//4/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.733832 from 1154228 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.930490'\n",
       "10. 'There were 1200825 total reads, 1200825 paired (1197536 properly mated), 2001 proper singles, 1288 improper reads (1204 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1197536 reads, 1197536 mapped, 0 unmapped, 1197536 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1985 reads, 1985 mapped, 0 unmapped, 1985 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1200825'\n",
       "29. 'Total placed reads: 1200825'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 16930'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1568.700000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -15306701.076942'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "8. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//4/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//4/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.739542 from 1143486 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.934216'\n",
       "10. 'There were 1190695 total reads, 1190695 paired (1186380 properly mated), 2967 proper singles, 1348 improper reads (1240 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1186380 reads, 1186380 mapped, 0 unmapped, 1186380 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1240 reads, 1240 mapped, 0 unmapped, 1240 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2949 reads, 2949 mapped, 0 unmapped, 2949 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1190695'\n",
       "29. 'Total placed reads: 1190695'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 11768'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1899.400000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -14888272.486862'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "9. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//5/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//5/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.737512 from 1062958 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.924341'\n",
       "10. 'There were 1105900 total reads, 1105900 paired (1102664 properly mated), 2398 proper singles, 838 improper reads (782 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1102664 reads, 1102664 mapped, 0 unmapped, 1102664 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 782 reads, 782 mapped, 0 unmapped, 782 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2392 reads, 2392 mapped, 0 unmapped, 2392 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1105900'\n",
       "29. 'Total placed reads: 1105900'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 15980'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1359.800000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -25110464.260699'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "10. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//5/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//5/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.753972 from 1049684 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.931368'\n",
       "10. 'There were 1092573 total reads, 1092573 paired (1089006 properly mated), 2821 proper singles, 746 improper reads (694 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1089006 reads, 1089006 mapped, 0 unmapped, 1089006 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 694 reads, 694 mapped, 0 unmapped, 694 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2813 reads, 2813 mapped, 0 unmapped, 2813 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1092573'\n",
       "29. 'Total placed reads: 1092573'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 10146'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1370.700000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -24422782.140910'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[[1]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/megahit.bam\"                                                          \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/megahit/contigs_filtered.fasta\"                       \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt\"                                                   \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.728772 from 1090912 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.916796\"                                                                                                          \n",
       "[10] \"There were 1135136 total reads, 1135136 paired (1131604 properly mated), 2300 proper singles, 1232 improper reads (1190 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt.param\"                                  \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1131604 reads, 1131604 mapped, 0 unmapped, 1131604 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[18] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1190 reads, 1190 mapped, 0 unmapped, 1190 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2278 reads, 2278 mapped, 0 unmapped, 2278 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1135136\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1135136\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 19449\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1725.100000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -21341424.829069\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt\"                                                 \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[2]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//1/metaspades.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//1/metaspades/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.732094 from 1076982 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.919110\"                                                                                                          \n",
       "[10] \"There were 1121157 total reads, 1121157 paired (1117228 properly mated), 2707 proper singles, 1222 improper reads (1182 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1117228 reads, 1117228 mapped, 0 unmapped, 1117228 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[18] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1182 reads, 1182 mapped, 0 unmapped, 1182 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2705 reads, 2705 mapped, 0 unmapped, 2705 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                                \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1121157\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1121157\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 14176\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1816.600000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -20576586.864308\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[3]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/megahit.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/megahit/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.733571 from 928224 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.907996\"                                                                                                     \n",
       " [9] \"There were 966493 total reads, 966493 paired (962814 properly mated), 3041 proper singles, 638 improper reads (592 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt.param\"                             \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 962814 reads, 962814 mapped, 0 unmapped, 962814 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 592 reads, 592 mapped, 0 unmapped, 592 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 3031 reads, 3031 mapped, 0 unmapped, 3031 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 966493\"                                                                                                                           \n",
       "[27] \"Total placed reads: 966493\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 28327\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1641.300000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -25478193.858699\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt\"                                            \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[4]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//2/metaspades.bam\"                                                  \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//2/metaspades/contigs_filtered.fasta\"               \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt\"                                           \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.703599 from 913356 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.897229\"                                                                                                     \n",
       " [9] \"There were 951219 total reads, 951219 paired (947314 properly mated), 3097 proper singles, 808 improper reads (760 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt.param\"                          \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 947314 reads, 947314 mapped, 0 unmapped, 947314 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 29 placed, 1 mappedButNotPlaced\"                                                      \n",
       "[16] \"NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 760 reads, 760 mapped, 0 unmapped, 760 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 3093 reads, 3093 mapped, 0 unmapped, 3093 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 951219\"                                                                                                                           \n",
       "[27] \"Total placed reads: 951218\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 1\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 19554\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1253.001317\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -24557003.672984\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt\"                                         \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[5]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//3/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//3/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.723209 from 1264320 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.916738\"                                                                                                        \n",
       "[10] \"There were 1314913 total reads, 1314913 paired (1311512 properly mated), 2789 proper singles, 612 improper reads (552 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1311512 reads, 1311512 mapped, 0 unmapped, 1311512 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 552 reads, 552 mapped, 0 unmapped, 552 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2769 reads, 2769 mapped, 0 unmapped, 2769 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1314913\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1314913\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 33103\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1860.700000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -23376993.706206\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[6]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//3/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//3/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.700152 from 1250640 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.905588\"                                                                                                        \n",
       "[10] \"There were 1300973 total reads, 1300973 paired (1297176 properly mated), 2867 proper singles, 930 improper reads (836 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1297176 reads, 1297176 mapped, 0 unmapped, 1297176 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 68 reads, 68 mapped, 0 unmapped, 68 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2857 reads, 2857 mapped, 0 unmapped, 2857 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1300973\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1300973\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 19826\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1405.300000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -22645031.366919\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[7]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//4/megahit.bam\"                                                          \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//4/megahit/contigs_filtered.fasta\"                       \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt\"                                                   \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.733832 from 1154228 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.930490\"                                                                                                          \n",
       "[10] \"There were 1200825 total reads, 1200825 paired (1197536 properly mated), 2001 proper singles, 1288 improper reads (1204 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt.param\"                                  \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1197536 reads, 1197536 mapped, 0 unmapped, 1197536 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1985 reads, 1985 mapped, 0 unmapped, 1985 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1200825\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1200825\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 16930\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1568.700000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -15306701.076942\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/megahit.txt\"                                                 \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[8]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//4/metaspades.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//4/metaspades/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.739542 from 1143486 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.934216\"                                                                                                          \n",
       "[10] \"There were 1190695 total reads, 1190695 paired (1186380 properly mated), 2967 proper singles, 1348 improper reads (1240 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1186380 reads, 1186380 mapped, 0 unmapped, 1186380 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 82 reads, 82 mapped, 0 unmapped, 82 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1240 reads, 1240 mapped, 0 unmapped, 1240 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2949 reads, 2949 mapped, 0 unmapped, 2949 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1190695\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1190695\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 11768\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1899.400000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -14888272.486862\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//4/metaspades.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[9]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//5/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//5/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.737512 from 1062958 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.924341\"                                                                                                        \n",
       "[10] \"There were 1105900 total reads, 1105900 paired (1102664 properly mated), 2398 proper singles, 838 improper reads (782 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1102664 reads, 1102664 mapped, 0 unmapped, 1102664 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 782 reads, 782 mapped, 0 unmapped, 782 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2392 reads, 2392 mapped, 0 unmapped, 2392 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1105900\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1105900\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 15980\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1359.800000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -25110464.260699\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[10]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/map//5/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/assembly//5/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.753972 from 1049684 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.931368\"                                                                                                        \n",
       "[10] \"There were 1092573 total reads, 1092573 paired (1089006 properly mated), 2821 proper singles, 746 improper reads (694 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1089006 reads, 1089006 mapped, 0 unmapped, 1089006 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 694 reads, 694 mapped, 0 unmapped, 694 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2813 reads, 2813 mapped, 0 unmapped, 2813 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1092573\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1092573\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 10146\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1370.700000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -24422782.140910\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//5/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                      \n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# apply function on cluster\n",
    "job_ret = future_lapply(ALE_cmds, FUN = function(x) bash_job(x, conda_env=conda_env))\n",
    "job_ret"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Summarizing results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T11:01:09.456013Z",
     "start_time": "2019-07-23T10:57:34.761Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 10\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//1/metaspades.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//2/metaspades.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r5/ALE//3/metaspades.txt'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt'\n",
       "2. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt'\n",
       "3. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt'\n",
       "4. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt'\n",
       "5. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt'\n",
       "6. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/megahit.txt\"   \n",
       "[2] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//1/metaspades.txt\"\n",
       "[3] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/megahit.txt\"   \n",
       "[4] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//2/metaspades.txt\"\n",
       "[5] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/megahit.txt\"   \n",
       "[6] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//3/metaspades.txt\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "ALE_res_files = list.files(work_dir, '*.txt$', recursive=TRUE, full.names=TRUE)\n",
    "ALE_res_files %>% length %>% print\n",
    "ALE_res_files %>% head"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T11:01:09.488245Z",
     "start_time": "2019-07-23T10:57:37.809Z"
    }
   },
   "outputs": [],
   "source": [
    "read_ALE_files = function(in_file){\n",
    "    # input command\n",
    "    cmd = 'egrep -v \"^#\" {in_file}'\n",
    "    cmd = glue::glue(cmd, in_file=in_file)\n",
    "    # reading in table\n",
    "    df = fread(cmd, sep=' ', header=FALSE)\n",
    "    # formatting\n",
    "    colnames(df) = c('contig', 'position', 'depth', \n",
    "                     'ln__depthLike', 'ln__placeLike', \n",
    "                     'ln__insertLike', 'ln__kmerLike')\n",
    "    df$Assembler = in_file %>% basename %>% gsub('\\\\.txt$', '', .)\n",
    "    df$Rep = in_file %>% dirname %>% basename\n",
    "    return(df)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T11:05:20.976928Z",
     "start_time": "2019-07-23T10:57:38.226Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 295103567\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>contig</th><th scope=col>position</th><th scope=col>depth</th><th scope=col>ln__depthLike</th><th scope=col>ln__placeLike</th><th scope=col>ln__insertLike</th><th scope=col>ln__kmerLike</th><th scope=col>Assembler</th><th scope=col>Rep</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>0      </td><td>0      </td><td>1      </td><td>-2.601 </td><td>-0.004 </td><td>-1.169 </td><td>-0.020 </td><td>megahit</td><td>1      </td></tr>\n",
       "\t<tr><td>0      </td><td>1      </td><td>1      </td><td>-2.601 </td><td>-0.005 </td><td>-1.169 </td><td>-0.059 </td><td>megahit</td><td>1      </td></tr>\n",
       "\t<tr><td>0      </td><td>2      </td><td>1      </td><td>-2.601 </td><td>-0.004 </td><td>-1.169 </td><td>-0.062 </td><td>megahit</td><td>1      </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllllllll}\n",
       " contig & position & depth & ln\\_\\_depthLike & ln\\_\\_placeLike & ln\\_\\_insertLike & ln\\_\\_kmerLike & Assembler & Rep\\\\\n",
       "\\hline\n",
       "\t 0       & 0       & 1       & -2.601  & -0.004  & -1.169  & -0.020  & megahit & 1      \\\\\n",
       "\t 0       & 1       & 1       & -2.601  & -0.005  & -1.169  & -0.059  & megahit & 1      \\\\\n",
       "\t 0       & 2       & 1       & -2.601  & -0.004  & -1.169  & -0.062  & megahit & 1      \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "contig | position | depth | ln__depthLike | ln__placeLike | ln__insertLike | ln__kmerLike | Assembler | Rep | \n",
       "|---|---|---|\n",
       "| 0       | 0       | 1       | -2.601  | -0.004  | -1.169  | -0.020  | megahit | 1       | \n",
       "| 0       | 1       | 1       | -2.601  | -0.005  | -1.169  | -0.059  | megahit | 1       | \n",
       "| 0       | 2       | 1       | -2.601  | -0.004  | -1.169  | -0.062  | megahit | 1       | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  contig position depth ln__depthLike ln__placeLike ln__insertLike ln__kmerLike\n",
       "1 0      0        1     -2.601        -0.004        -1.169         -0.020      \n",
       "2 0      1        1     -2.601        -0.005        -1.169         -0.059      \n",
       "3 0      2        1     -2.601        -0.004        -1.169         -0.062      \n",
       "  Assembler Rep\n",
       "1 megahit   1  \n",
       "2 megahit   1  \n",
       "3 megahit   1  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "doParallel::registerDoParallel(threads)\n",
    "ALE_res = plyr::llply(as.list(ALE_res_files), read_ALE_files, .parallel=TRUE) %>%\n",
    "    do.call(rbind, .)\n",
    "\n",
    "ALE_res %>% nrow %>% print\n",
    "ALE_res %>% head(n=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Writing combined ALE results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T11:53:06.602859Z",
     "start_time": "2019-07-23T10:57:38.753Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File written: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r5/ALE//ALE_results.tsv \n"
     ]
    }
   ],
   "source": [
    "out_file = file.path(work_dir, 'ALE_results.tsv')\n",
    "ALE_res %>%\n",
    "    write.table(file=out_file, sep='\\t', quote=FALSE, row.names=FALSE)\n",
    "cat('File written:', out_file, '\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# sessionInfo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T11:55:23.782479Z",
     "start_time": "2019-07-23T11:55:19.740Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 3.4.1 (2017-06-30)\n",
       "Platform: x86_64-pc-linux-gnu (64-bit)\n",
       "Running under: Ubuntu 18.04.1 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS: /ebio/abt3_projects/software/miniconda3_gt4.4/envs/py3_physeq/lib/R/lib/libRblas.so\n",
       "LAPACK: /ebio/abt3_projects/software/miniconda3_gt4.4/envs/py3_physeq/lib/R/lib/libRlapack.so\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats     graphics  grDevices utils     datasets  methods  \n",
       "[8] base     \n",
       "\n",
       "other attached packages:\n",
       " [1] bindrcpp_0.2.2          future.apply_1.0.0      future.batchtools_0.7.1\n",
       " [4] future_1.9.0            doParallel_1.0.11       iterators_1.0.10       \n",
       " [7] foreach_1.4.4           data.table_1.11.4       ggplot2_3.0.0          \n",
       "[10] tidyr_0.8.1             dplyr_0.7.6            \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       " [1] pbdZMQ_0.3-2      progress_1.2.0    tidyselect_0.2.4  repr_0.15.0      \n",
       " [5] purrr_0.2.4       listenv_0.7.0     colorspace_1.3-2  htmltools_0.3.6  \n",
       " [9] base64enc_0.1-3   rlang_0.2.1       pillar_1.2.2      glue_1.3.0       \n",
       "[13] withr_2.1.1       rappdirs_0.3.1    uuid_0.1-2        bindr_0.1.1      \n",
       "[17] plyr_1.8.4        stringr_1.3.1     munsell_0.5.0     gtable_0.2.0     \n",
       "[21] codetools_0.2-15  evaluate_0.10.1   batchtools_0.9.11 IRdisplay_0.4.4  \n",
       "[25] Rcpp_0.12.17      scales_0.5.0      backports_1.1.2   checkmate_1.8.5  \n",
       "[29] IRkernel_0.8.12   jsonlite_1.5      fs_1.2.6          brew_1.0-6       \n",
       "[33] hms_0.4.2         digest_0.6.15     stringi_1.2.3     grid_3.4.1       \n",
       "[37] tools_3.4.1       magrittr_1.5      base64url_1.4     lazyeval_0.2.1   \n",
       "[41] tibble_1.4.2      crayon_1.3.4      pkgconfig_2.0.1   prettyunits_1.0.2\n",
       "[45] assertthat_0.2.0  R6_2.2.2          globals_0.12.2    compiler_3.4.1   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R [conda env:py3_physeq]",
   "language": "R",
   "name": "conda-env-py3_physeq-r"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.4.1"
  },
  "notify_time": "30",
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": true,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
