{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "toc": true
   },
   "source": [
    "<h1>Table of Contents<span class=\"tocSkip\"></span></h1>\n",
    "<div class=\"toc\"><ul class=\"toc-item\"><li><span><a href=\"#Goal\" data-toc-modified-id=\"Goal-1\"><span class=\"toc-item-num\">1&nbsp;&nbsp;</span>Goal</a></span></li><li><span><a href=\"#Var\" data-toc-modified-id=\"Var-2\"><span class=\"toc-item-num\">2&nbsp;&nbsp;</span>Var</a></span></li><li><span><a href=\"#Init\" data-toc-modified-id=\"Init-3\"><span class=\"toc-item-num\">3&nbsp;&nbsp;</span>Init</a></span></li><li><span><a href=\"#Load\" data-toc-modified-id=\"Load-4\"><span class=\"toc-item-num\">4&nbsp;&nbsp;</span>Load</a></span></li><li><span><a href=\"#Running-ALE\" data-toc-modified-id=\"Running-ALE-5\"><span class=\"toc-item-num\">5&nbsp;&nbsp;</span>Running ALE</a></span></li><li><span><a href=\"#Summarizing-results\" data-toc-modified-id=\"Summarizing-results-6\"><span class=\"toc-item-num\">6&nbsp;&nbsp;</span>Summarizing results</a></span><ul class=\"toc-item\"><li><span><a href=\"#Writing-combined-ALE-results\" data-toc-modified-id=\"Writing-combined-ALE-results-6.1\"><span class=\"toc-item-num\">6.1&nbsp;&nbsp;</span>Writing combined ALE results</a></span></li></ul></li><li><span><a href=\"#---WAITING---\" data-toc-modified-id=\"---WAITING----7\"><span class=\"toc-item-num\">7&nbsp;&nbsp;</span>-- WAITING --</a></span></li><li><span><a href=\"#sessionInfo\" data-toc-modified-id=\"sessionInfo-8\"><span class=\"toc-item-num\">8&nbsp;&nbsp;</span>sessionInfo</a></span></li></ul></div>"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Goal\n",
    "\n",
    "* run ALE on simulated test dataset for DeepMAsED \n",
    "  * the ground truth results can be used for selecting cutoffs for ALE and assessing accuracy"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Var"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:01:53.903984Z",
     "start_time": "2019-07-23T09:01:53.837Z"
    }
   },
   "outputs": [],
   "source": [
    "work_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE/'\n",
    "\n",
    "# MAGs for genome analysis\n",
    "asmbl_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly/'\n",
    "\n",
    "# bam file dir (mapping of metagenome reads to corresponding MAGs)\n",
    "map_dir = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map/'\n",
    "\n",
    "# ALE executable\n",
    "ALE_exe = '/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE'\n",
    "\n",
    "# params\n",
    "conda_env = 'base'\n",
    "threads = 12"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Init"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:01:11.977327Z",
     "start_time": "2019-07-23T09:01:10.135Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "Attaching package: ‘dplyr’\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    filter, lag\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    intersect, setdiff, setequal, union\n",
      "\n",
      "\n",
      "Attaching package: ‘data.table’\n",
      "\n",
      "The following objects are masked from ‘package:dplyr’:\n",
      "\n",
      "    between, first, last\n",
      "\n",
      "Loading required package: foreach\n",
      "Loading required package: iterators\n",
      "Loading required package: parallel\n",
      "\n",
      "Attaching package: ‘future.apply’\n",
      "\n",
      "The following object is masked from ‘package:future’:\n",
      "\n",
      "    future_lapply\n",
      "\n"
     ]
    }
   ],
   "source": [
    "library(dplyr)\n",
    "library(tidyr)\n",
    "library(ggplot2)\n",
    "library(data.table)\n",
    "library(doParallel)\n",
    "\n",
    "library(future)\n",
    "library(future.batchtools)\n",
    "library(future.apply)\n",
    "options(future.wait.interval = 2.0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Load"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:01:14.627569Z",
     "start_time": "2019-07-23T09:01:12.575Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 60\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                           </td><td>1                                                                                                                 </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta   </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                        </td><td>1                                                                                                                 </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta</td></tr>\n",
       "\t<tr><td>megahit                                                                                                           </td><td>10                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta  </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lll}\n",
       " Assembler & Rep & Fasta\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                      & 1                                                                                                                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/megahit/contigs\\_filtered.fasta   \\\\\n",
       "\t metaspades                                                                                                                   & 1                                                                                                                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/metaspades/contigs\\_filtered.fasta\\\\\n",
       "\t megahit                                                                                                                      & 10                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//10/megahit/contigs\\_filtered.fasta  \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                            | 1                                                                                                                  | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta    | \n",
       "| metaspades                                                                                                         | 1                                                                                                                  | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta | \n",
       "| megahit                                                                                                            | 10                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta   | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Fasta                                                                                                             \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# adding bam files\n",
    "fasta_files = list.files(asmbl_dir, '*.fasta$', recursive=TRUE, full.names=TRUE)\n",
    "fasta_files = data.frame(Assembler = fasta_files %>% dirname %>% basename,\n",
    "                         Rep = fasta_files %>% dirname %>% dirname %>% basename,\n",
    "                         Fasta = fasta_files)\n",
    "\n",
    "fasta_files %>% nrow %>% print\n",
    "fasta_files %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:01:56.814152Z",
     "start_time": "2019-07-23T09:01:56.727Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 58\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Bam</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                   </td><td>1                                                                                         </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam   </td></tr>\n",
       "\t<tr><td>metaspades                                                                                </td><td>1                                                                                         </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam</td></tr>\n",
       "\t<tr><td>megahit                                                                                   </td><td>10                                                                                        </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam  </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lll}\n",
       " Assembler & Rep & Bam\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                            & 1                                                                                                  & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/megahit.bam   \\\\\n",
       "\t metaspades                                                                                         & 1                                                                                                  & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/metaspades.bam\\\\\n",
       "\t megahit                                                                                            & 10                                                                                                 & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//10/megahit.bam  \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Bam | \n",
       "|---|---|---|\n",
       "| megahit                                                                                    | 1                                                                                          | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam    | \n",
       "| metaspades                                                                                 | 1                                                                                          | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam | \n",
       "| megahit                                                                                    | 10                                                                                         | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam   | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Bam                                                                                       \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# adding bam files\n",
    "bam_files = list.files(map_dir, '*.bam$', recursive=TRUE, full.names=TRUE)\n",
    "bam_files = data.frame(Assembler = bam_files %>% basename %>% gsub('\\\\.bam$', '', .),\n",
    "                       Rep = bam_files %>% dirname %>% basename,\n",
    "                       Bam = bam_files)\n",
    "\n",
    "bam_files %>% nrow %>% print\n",
    "bam_files %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:02:01.525784Z",
     "start_time": "2019-07-23T09:02:01.437Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning message:\n",
      "“Column `Rep` joining factors with different levels, coercing to character vector”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 58\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th><th scope=col>Bam</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                           </td><td>1                                                                                                                 </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam                           </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                        </td><td>1                                                                                                                 </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta</td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam                        </td></tr>\n",
       "\t<tr><td>megahit                                                                                                           </td><td>10                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam                          </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|llll}\n",
       " Assembler & Rep & Fasta & Bam\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                      & 1                                                                                                                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/megahit.bam                             \\\\\n",
       "\t metaspades                                                                                                                   & 1                                                                                                                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/metaspades/contigs\\_filtered.fasta & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/metaspades.bam                          \\\\\n",
       "\t megahit                                                                                                                      & 10                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//10/megahit/contigs\\_filtered.fasta   & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//10/megahit.bam                            \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | Bam | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                            | 1                                                                                                                  | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam                            | \n",
       "| metaspades                                                                                                         | 1                                                                                                                  | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam                         | \n",
       "| megahit                                                                                                            | 10                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam                           | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Fasta                                                                                                             \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta  \n",
       "  Bam                                                                                       \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# joining\n",
    "MAGs = fasta_files %>%\n",
    "    inner_join(bam_files, c('Assembler', 'Rep'))\n",
    "MAGs %>% nrow %>% print\n",
    "MAGs %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:02:06.668650Z",
     "start_time": "2019-07-23T09:02:06.534Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>Assembler</th><th scope=col>Rep</th><th scope=col>Fasta</th><th scope=col>Bam</th><th scope=col>ALE</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>megahit                                                                                                           </td><td>1                                                                                                                 </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta   </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam                           </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt                           </td></tr>\n",
       "\t<tr><td>metaspades                                                                                                        </td><td>1                                                                                                                 </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta</td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam                        </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt                        </td></tr>\n",
       "\t<tr><td>megahit                                                                                                           </td><td>10                                                                                                                </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta  </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam                          </td><td>/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt                          </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllll}\n",
       " Assembler & Rep & Fasta & Bam & ALE\\\\\n",
       "\\hline\n",
       "\t megahit                                                                                                                      & 1                                                                                                                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/megahit/contigs\\_filtered.fasta    & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/megahit.bam                              & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/megahit.txt                             \\\\\n",
       "\t metaspades                                                                                                                   & 1                                                                                                                            & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/metaspades/contigs\\_filtered.fasta & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/metaspades.bam                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/metaspades.txt                          \\\\\n",
       "\t megahit                                                                                                                      & 10                                                                                                                           & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//10/megahit/contigs\\_filtered.fasta   & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//10/megahit.bam                             & /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/megahit.txt                            \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "Assembler | Rep | Fasta | Bam | ALE | \n",
       "|---|---|---|\n",
       "| megahit                                                                                                            | 1                                                                                                                  | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta    | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam                            | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt                            | \n",
       "| metaspades                                                                                                         | 1                                                                                                                  | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam                         | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt                         | \n",
       "| megahit                                                                                                            | 10                                                                                                                 | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta   | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam                           | /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt                           | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  Assembler  Rep\n",
       "1 megahit    1  \n",
       "2 metaspades 1  \n",
       "3 megahit    10 \n",
       "  Fasta                                                                                                             \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta  \n",
       "  Bam                                                                                       \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam  \n",
       "  ALE                                                                                       \n",
       "1 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt   \n",
       "2 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt\n",
       "3 /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# ALE output file name\n",
    "MAGs = MAGs %>%\n",
    "    mutate(ALE = mapply(function(A, R) file.path(work_dir, R, paste0(A, '.txt')),\n",
    "                        A=Assembler, R=Rep))\n",
    "                        \n",
    "MAGs %>% head(n=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:02:25.855066Z",
     "start_time": "2019-07-23T09:02:25.805Z"
    }
   },
   "outputs": [],
   "source": [
    "# making output directories\n",
    "ret = MAGs$ALE %>% \n",
    "    dirname() %>%\n",
    "    unique() %>%\n",
    "    sapply(., function(x) dir.create(x, showWarnings=FALSE))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Running ALE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:02:30.240272Z",
     "start_time": "2019-07-23T09:02:30.165Z"
    }
   },
   "outputs": [],
   "source": [
    "# cluster resources\n",
    "resources = list(h_rt = '00:59:00',\n",
    "                 h_vmem = '8G',\n",
    "                 threads = '1',\n",
    "                 conda.env = 'py3_physeq')     # conda env with batchtools installed\n",
    "plan(batchtools_sge, resources=resources, workers=30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:02:34.339445Z",
     "start_time": "2019-07-23T09:02:34.156Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 58\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "'/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'"
      ],
      "text/latex": [
       "'/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/megahit.bam /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/megahit/contigs\\_filtered.fasta /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/megahit.txt'"
      ],
      "text/markdown": [
       "'/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'"
      ],
      "text/plain": [
       "[1] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/bin/ALE/src/ALE /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# creating commands for calling ALE \n",
    "ALE_params = function(x, exe){\n",
    "    cmd = '{exe} {bam} {fasta} {outfile}'\n",
    "    cmd = glue::glue(cmd, exe=exe, bam=x[['Bam']], fasta=x[['Fasta']], outfile=x[['ALE']])\n",
    "    return(cmd)\n",
    "}\n",
    "\n",
    "ALE_cmds = apply(MAGs, 1, ALE_params, exe=ALE_exe) %>%\n",
    "    as.list\n",
    "\n",
    "ALE_cmds %>% length %>% print\n",
    "ALE_cmds[[1]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:03:03.068197Z",
     "start_time": "2019-07-23T09:03:03.036Z"
    }
   },
   "outputs": [],
   "source": [
    "# general bash job function\n",
    "bash_job = function(cmd, conda_env, stdout=TRUE, stderr=FALSE){\n",
    "    # cmd : string; commandline job (eg., 'ls -thlc')\n",
    "    # conda_env : string; conda environment name\n",
    "    cmd = sprintf('. ~/.bashrc; conda activate %s; %s', conda_env, cmd)\n",
    "    cmd = sprintf('-c \"%s\"', cmd)\n",
    "    system2('bash', cmd, stdout=stdout, stderr=stderr)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:11:23.873506Z",
     "start_time": "2019-07-23T09:03:04.350Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.723707 from 1212220 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.925705'</li>\n",
       "\t<li>'There were 1260475 total reads, 1260475 paired (1257682 properly mated), 2073 proper singles, 720 improper reads (690 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1257682 reads, 1257682 mapped, 0 unmapped, 1257681 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 690 reads, 690 mapped, 0 unmapped, 690 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2043 reads, 2043 mapped, 0 unmapped, 2043 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1260475'</li>\n",
       "\t<li>'Total placed reads: 1260474'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17197'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1203.800955'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22697096.942980'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.732010 from 1201202 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.931224'</li>\n",
       "\t<li>'There were 1249353 total reads, 1249353 paired (1246230 properly mated), 2167 proper singles, 956 improper reads (912 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1246230 reads, 1246230 mapped, 0 unmapped, 1246230 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 912 reads, 912 mapped, 0 unmapped, 912 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2133 reads, 2133 mapped, 0 unmapped, 2133 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1249353'</li>\n",
       "\t<li>'Total placed reads: 1249353'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11647'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1221.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22172524.551273'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.735681 from 1270902 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.930199'</li>\n",
       "\t<li>'There were 1321605 total reads, 1321605 paired (1318686 properly mated), 1987 proper singles, 932 improper reads (880 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1318686 reads, 1318686 mapped, 0 unmapped, 1318686 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 880 reads, 880 mapped, 0 unmapped, 880 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1973 reads, 1973 mapped, 0 unmapped, 1973 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1321605'</li>\n",
       "\t<li>'Total placed reads: 1321605'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15898'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1278.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -19122156.831738'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.730003 from 1262138 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.927099'</li>\n",
       "\t<li>'There were 1312751 total reads, 1312751 paired (1309554 properly mated), 2309 proper singles, 888 improper reads (836 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1309554 reads, 1309554 mapped, 0 unmapped, 1309553 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2289 reads, 2289 mapped, 0 unmapped, 2289 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1312751'</li>\n",
       "\t<li>'Total placed reads: 1312750'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9767'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1267.800966'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -18755636.338068'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//11/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//11/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.769628 from 1229856 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.945784'</li>\n",
       "\t<li>'There were 1278594 total reads, 1278594 paired (1276032 properly mated), 1706 proper singles, 856 improper reads (778 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1276032 reads, 1276032 mapped, 0 unmapped, 1276031 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 778 reads, 778 mapped, 0 unmapped, 778 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1674 reads, 1674 mapped, 0 unmapped, 1674 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1278594'</li>\n",
       "\t<li>'Total placed reads: 1278593'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13435'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1076.300842'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12552137.564932'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//11/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//11/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.750257 from 1222898 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.940087'</li>\n",
       "\t<li>'There were 1271913 total reads, 1271913 paired (1268674 properly mated), 1953 proper singles, 1286 improper reads (1220 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1268674 reads, 1268674 mapped, 0 unmapped, 1268674 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 40 reads, 40 mapped, 0 unmapped, 39 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1220 reads, 1220 mapped, 0 unmapped, 1220 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1909 reads, 1909 mapped, 0 unmapped, 1909 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1271913'</li>\n",
       "\t<li>'Total placed reads: 1271912'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12144'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1243.700978'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -12235260.934613'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//12/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//12/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.769835 from 1441024 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.947323'</li>\n",
       "\t<li>'There were 1498680 total reads, 1498680 paired (1495500 properly mated), 2118 proper singles, 1062 improper reads (1004 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1495500 reads, 1495500 mapped, 0 unmapped, 1495495 placed, 5 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2072 reads, 2072 mapped, 0 unmapped, 2072 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1498680'</li>\n",
       "\t<li>'Total placed reads: 1498675'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 5'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13692'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1502.605013'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -17965987.100460'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//12/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//12/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.754574 from 1434354 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.941442'</li>\n",
       "\t<li>'There were 1491690 total reads, 1491690 paired (1488456 properly mated), 2248 proper singles, 986 improper reads (958 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1488456 reads, 1488456 mapped, 0 unmapped, 1488453 placed, 3 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 958 reads, 958 mapped, 0 unmapped, 958 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2228 reads, 2228 mapped, 0 unmapped, 2228 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1491690'</li>\n",
       "\t<li>'Total placed reads: 1491687'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 3'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11700'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1390.202796'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -17661706.357633'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//13/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//13/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.765196 from 1040338 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.932119'</li>\n",
       "\t<li>'There were 1082811 total reads, 1082811 paired (1079514 properly mated), 2755 proper singles, 542 improper reads (504 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1079514 reads, 1079514 mapped, 0 unmapped, 1079514 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 504 reads, 504 mapped, 0 unmapped, 504 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2753 reads, 2753 mapped, 0 unmapped, 2753 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1082811'</li>\n",
       "\t<li>'Total placed reads: 1082811'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 29053'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1845.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -29108120.735174'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//13/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//13/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.742836 from 1026902 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.920751'</li>\n",
       "\t<li>'There were 1068969 total reads, 1068969 paired (1065480 properly mated), 2609 proper singles, 880 improper reads (854 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1065480 reads, 1065480 mapped, 0 unmapped, 1065480 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 854 reads, 854 mapped, 0 unmapped, 854 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2609 reads, 2609 mapped, 0 unmapped, 2609 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1068969'</li>\n",
       "\t<li>'Total placed reads: 1068969'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17762'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1093.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -28177219.593090'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//14/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//14/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.779876 from 1026678 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.938274'</li>\n",
       "\t<li>'There were 1067657 total reads, 1067657 paired (1065468 properly mated), 1933 proper singles, 256 improper reads (246 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1065468 reads, 1065468 mapped, 0 unmapped, 1065468 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1929 reads, 1929 mapped, 0 unmapped, 1929 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1067657'</li>\n",
       "\t<li>'Total placed reads: 1067657'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19976'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1050.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -23112358.285221'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//14/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//14/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.775480 from 1017850 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.938990'</li>\n",
       "\t<li>'There were 1058508 total reads, 1058508 paired (1056270 properly mated), 1756 proper singles, 482 improper reads (448 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1056270 reads, 1056270 mapped, 0 unmapped, 1056270 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1750 reads, 1750 mapped, 0 unmapped, 1750 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1058508'</li>\n",
       "\t<li>'Total placed reads: 1058508'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12641'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 721.700000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22517914.298963'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//15/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//15/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.750549 from 1169866 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.930211'</li>\n",
       "\t<li>'There were 1216436 total reads, 1216436 paired (1213808 properly mated), 2268 proper singles, 360 improper reads (326 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1213808 reads, 1213808 mapped, 0 unmapped, 1213808 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2260 reads, 2260 mapped, 0 unmapped, 2260 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1216436'</li>\n",
       "\t<li>'Total placed reads: 1216436'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 23858'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1417.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -18167994.130481'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//15/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//15/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.726422 from 1160328 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.920425'</li>\n",
       "\t<li>'There were 1206622 total reads, 1206622 paired (1203812 properly mated), 2148 proper singles, 662 improper reads (608 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1203812 reads, 1203812 mapped, 0 unmapped, 1203811 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2140 reads, 2140 mapped, 0 unmapped, 2140 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1206622'</li>\n",
       "\t<li>'Total placed reads: 1206621'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14959'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1035.500858'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -17494904.019029'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//16/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//16/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.674769 from 1312060 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.906746'</li>\n",
       "\t<li>'There were 1363096 total reads, 1363096 paired (1360784 properly mated), 1860 proper singles, 452 improper reads (402 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1360784 reads, 1360784 mapped, 0 unmapped, 1360781 placed, 3 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1846 reads, 1846 mapped, 0 unmapped, 1846 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1363096'</li>\n",
       "\t<li>'Total placed reads: 1363093'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 3'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15590'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1039.902289'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -19575398.337141'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//16/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//16/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.665849 from 1304080 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.903611'</li>\n",
       "\t<li>'There were 1355039 total reads, 1355039 paired (1352438 properly mated), 1835 proper singles, 766 improper reads (710 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1352438 reads, 1352438 mapped, 0 unmapped, 1352438 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 710 reads, 710 mapped, 0 unmapped, 710 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1831 reads, 1831 mapped, 0 unmapped, 1831 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1355039'</li>\n",
       "\t<li>'Total placed reads: 1355039'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9252'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 966.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -19152471.917716'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//17/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//17/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.714297 from 1056452 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.917958'</li>\n",
       "\t<li>'There were 1098861 total reads, 1098861 paired (1095938 properly mated), 2365 proper singles, 558 improper reads (536 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1095938 reads, 1095938 mapped, 0 unmapped, 1095938 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 536 reads, 536 mapped, 0 unmapped, 536 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2357 reads, 2357 mapped, 0 unmapped, 2357 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1098861'</li>\n",
       "\t<li>'Total placed reads: 1098861'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 21615'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1217.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21855528.539573'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//17/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//17/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.708831 from 1047290 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.914692'</li>\n",
       "\t<li>'There were 1089261 total reads, 1089261 paired (1086416 properly mated), 2189 proper singles, 656 improper reads (628 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1086416 reads, 1086416 mapped, 0 unmapped, 1086416 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 628 reads, 628 mapped, 0 unmapped, 628 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2185 reads, 2185 mapped, 0 unmapped, 2185 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1089261'</li>\n",
       "\t<li>'Total placed reads: 1089261'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15593'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 904.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21295303.741731'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//18/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//18/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.784092 from 1032744 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.942105'</li>\n",
       "\t<li>'There were 1077664 total reads, 1077664 paired (1071728 properly mated), 5294 proper singles, 642 improper reads (624 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1071728 reads, 1071728 mapped, 0 unmapped, 1071728 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 624 reads, 624 mapped, 0 unmapped, 624 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 5288 reads, 5288 mapped, 0 unmapped, 5288 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1077664'</li>\n",
       "\t<li>'Total placed reads: 1077664'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 63004'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2727.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -30555470.646073'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//18/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//18/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.757966 from 994686 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.934278'</li>\n",
       "\t<li>'There were 1039080 total reads, 1039080 paired (1031998 properly mated), 6212 proper singles, 870 improper reads (846 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1031998 reads, 1031998 mapped, 0 unmapped, 1031998 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 846 reads, 846 mapped, 0 unmapped, 846 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 6210 reads, 6210 mapped, 0 unmapped, 6210 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1039080'</li>\n",
       "\t<li>'Total placed reads: 1039080'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 32109'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2378.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -28364110.245962'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//19/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//19/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.758838 from 1444970 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.947463'</li>\n",
       "\t<li>'There were 1503635 total reads, 1503635 paired (1499582 properly mated), 2255 proper singles, 1798 improper reads (1720 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1499582 reads, 1499582 mapped, 0 unmapped, 1499581 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1720 reads, 1720 mapped, 0 unmapped, 1720 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2219 reads, 2219 mapped, 0 unmapped, 2219 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1503635'</li>\n",
       "\t<li>'Total placed reads: 1503634'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11265'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1860.201237'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15588448.646139'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//19/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//19/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.744043 from 1430676 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.941927'</li>\n",
       "\t<li>'There were 1489966 total reads, 1489966 paired (1484608 properly mated), 3708 proper singles, 1650 improper reads (1564 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1484608 reads, 1484608 mapped, 0 unmapped, 1484608 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1564 reads, 1564 mapped, 0 unmapped, 1564 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3696 reads, 3696 mapped, 0 unmapped, 3696 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1489966'</li>\n",
       "\t<li>'Total placed reads: 1489966'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 7489'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2466.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15083716.468382'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//2/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//2/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.707842 from 1162692 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.911909'</li>\n",
       "\t<li>'There were 1209478 total reads, 1209478 paired (1206160 properly mated), 2218 proper singles, 1100 improper reads (1062 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1206160 reads, 1206160 mapped, 0 unmapped, 1206160 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1062 reads, 1062 mapped, 0 unmapped, 1062 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2210 reads, 2210 mapped, 0 unmapped, 2210 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1209478'</li>\n",
       "\t<li>'Total placed reads: 1209478'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16641'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1409.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21502554.168959'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//2/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//2/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.708035 from 1149766 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.911943'</li>\n",
       "\t<li>'There were 1196329 total reads, 1196329 paired (1192696 properly mated), 2801 proper singles, 832 improper reads (810 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1192696 reads, 1192696 mapped, 0 unmapped, 1192696 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 810 reads, 810 mapped, 0 unmapped, 810 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2801 reads, 2801 mapped, 0 unmapped, 2801 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1196329'</li>\n",
       "\t<li>'Total placed reads: 1196329'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 12478'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1527.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -20946198.398444'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//20/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//20/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.698839 from 929876 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.900362'</li>\n",
       "\t<li>'There were 966664 total reads, 966664 paired (964368 properly mated), 1996 proper singles, 300 improper reads (284 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 964368 reads, 964368 mapped, 0 unmapped, 964368 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 284 reads, 284 mapped, 0 unmapped, 284 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1984 reads, 1984 mapped, 0 unmapped, 1984 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 966664'</li>\n",
       "\t<li>'Total placed reads: 966664'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 18701'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1107.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -18099506.304873'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//20/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//20/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.691066 from 920578 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.895893'</li>\n",
       "\t<li>'There were 957156 total reads, 957156 paired (954662 properly mated), 1950 proper singles, 544 improper reads (520 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'FR orientation with 954662 reads, 954662 mapped, 0 unmapped, 954662 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 520 reads, 520 mapped, 0 unmapped, 520 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1928 reads, 1928 mapped, 0 unmapped, 1928 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 957156'</li>\n",
       "\t<li>'Total placed reads: 957156'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13037'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 838.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -17479146.406151'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//21/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//21/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.783618 from 1315482 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.951948'</li>\n",
       "\t<li>'There were 1367541 total reads, 1367541 paired (1365056 properly mated), 1629 proper singles, 856 improper reads (790 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1365056 reads, 1365056 mapped, 0 unmapped, 1365056 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 790 reads, 790 mapped, 0 unmapped, 790 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1623 reads, 1623 mapped, 0 unmapped, 1623 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1367541'</li>\n",
       "\t<li>'Total placed reads: 1367541'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11981'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1166.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22004557.470929'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//21/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//21/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.774182 from 1307486 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.947431'</li>\n",
       "\t<li>'There were 1360008 total reads, 1360008 paired (1356718 properly mated), 2242 proper singles, 1048 improper reads (1004 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1356718 reads, 1356718 mapped, 0 unmapped, 1356718 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2234 reads, 2234 mapped, 0 unmapped, 2234 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1360008'</li>\n",
       "\t<li>'Total placed reads: 1360008'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8679'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1424.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21546080.209347'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//22/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//22/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.758738 from 1107196 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.936306'</li>\n",
       "\t<li>'There were 1153199 total reads, 1153199 paired (1148874 properly mated), 2607 proper singles, 1718 improper reads (1640 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1148874 reads, 1148874 mapped, 0 unmapped, 1148874 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 37 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1640 reads, 1640 mapped, 0 unmapped, 1640 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2595 reads, 2595 mapped, 0 unmapped, 2595 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1153199'</li>\n",
       "\t<li>'Total placed reads: 1153198'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14542'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2070.801796'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -28105795.208026'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//22/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//22/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.750389 from 1088718 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.931532'</li>\n",
       "\t<li>'There were 1133938 total reads, 1133938 paired (1129594 properly mated), 3196 proper singles, 1148 improper reads (1110 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1129594 reads, 1129594 mapped, 0 unmapped, 1129593 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1110 reads, 1110 mapped, 0 unmapped, 1110 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3196 reads, 3196 mapped, 0 unmapped, 3196 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1133938'</li>\n",
       "\t<li>'Total placed reads: 1133937'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8482'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2082.501837'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -26800705.802743'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//23/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//23/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.717675 from 1100808 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.915271'</li>\n",
       "\t<li>'There were 1144566 total reads, 1144566 paired (1142120 properly mated), 1924 proper singles, 522 improper reads (462 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1142120 reads, 1142120 mapped, 0 unmapped, 1142120 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 462 reads, 462 mapped, 0 unmapped, 462 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1912 reads, 1912 mapped, 0 unmapped, 1912 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1144566'</li>\n",
       "\t<li>'Total placed reads: 1144566'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14963'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1172.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14831360.378148'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//23/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//23/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.718760 from 1092852 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.918076'</li>\n",
       "\t<li>'There were 1136709 total reads, 1136709 paired (1133854 properly mated), 1947 proper singles, 908 improper reads (798 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1133854 reads, 1133854 mapped, 0 unmapped, 1133854 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1941 reads, 1941 mapped, 0 unmapped, 1941 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1136709'</li>\n",
       "\t<li>'Total placed reads: 1136709'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 11931'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1125.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14386723.450540'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//24/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//24/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.751733 from 1288410 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.937254'</li>\n",
       "\t<li>'There were 1338708 total reads, 1338708 paired (1336816 properly mated), 1224 proper singles, 668 improper reads (604 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1336816 reads, 1336816 mapped, 0 unmapped, 1336816 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1338708'</li>\n",
       "\t<li>'Total placed reads: 1338708'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 8237'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 864.500000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11592032.231556'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//24/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//24/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.743318 from 1283372 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.934212'</li>\n",
       "\t<li>'There were 1333954 total reads, 1333954 paired (1331536 properly mated), 1730 proper singles, 688 improper reads (608 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1331536 reads, 1331536 mapped, 0 unmapped, 1331536 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 76 reads, 76 mapped, 0 unmapped, 76 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1724 reads, 1724 mapped, 0 unmapped, 1724 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1333954'</li>\n",
       "\t<li>'Total placed reads: 1333954'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 6248'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1017.100000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -11400899.152586'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//25/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//25/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.766508 from 1452908 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.947885'</li>\n",
       "\t<li>'There were 1511420 total reads, 1511420 paired (1507552 properly mated), 2286 proper singles, 1582 improper reads (1534 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1507552 reads, 1507552 mapped, 0 unmapped, 1507552 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1534 reads, 1534 mapped, 0 unmapped, 1534 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2266 reads, 2266 mapped, 0 unmapped, 2266 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1511420'</li>\n",
       "\t<li>'Total placed reads: 1511420'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14303'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1969.861129'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -40241263.112110'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//25/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//25/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.762489 from 1444882 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.944863'</li>\n",
       "\t<li>'There were 1503090 total reads, 1503090 paired (1499156 properly mated), 2700 proper singles, 1234 improper reads (1026 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1499156 reads, 1499156 mapped, 0 unmapped, 1499156 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 184 reads, 184 mapped, 0 unmapped, 184 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1026 reads, 1026 mapped, 0 unmapped, 1026 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2690 reads, 2690 mapped, 0 unmapped, 2690 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1503090'</li>\n",
       "\t<li>'Total placed reads: 1503090'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9385'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1992.625081'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -39926611.413909'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//26/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//26/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.761931 from 1237382 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.934695'</li>\n",
       "\t<li>'There were 1290127 total reads, 1290127 paired (1283976 properly mated), 3499 proper singles, 2652 improper reads (2600 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1283976 reads, 1283976 mapped, 0 unmapped, 1283976 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 23 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 2600 reads, 2600 mapped, 0 unmapped, 2600 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3485 reads, 3485 mapped, 0 unmapped, 3485 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1290127'</li>\n",
       "\t<li>'Total placed reads: 1290126'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 28963'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2734.102119'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21738734.283024'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//26/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//26/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.756475 from 1221778 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.932433'</li>\n",
       "\t<li>'There were 1274647 total reads, 1274647 paired (1267720 properly mated), 4839 proper singles, 2088 improper reads (2022 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1267720 reads, 1267720 mapped, 0 unmapped, 1267720 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 2022 reads, 2022 mapped, 0 unmapped, 2022 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 4827 reads, 4827 mapped, 0 unmapped, 4827 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1274647'</li>\n",
       "\t<li>'Total placed reads: 1274647'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19692'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2942.300000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21013802.246661'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//27/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//27/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.732755 from 1247126 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.932042'</li>\n",
       "\t<li>'There were 1296952 total reads, 1296952 paired (1294018 properly mated), 2364 proper singles, 570 improper reads (508 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1294018 reads, 1294018 mapped, 0 unmapped, 1294017 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 508 reads, 508 mapped, 0 unmapped, 508 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2346 reads, 2346 mapped, 0 unmapped, 2346 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1296952'</li>\n",
       "\t<li>'Total placed reads: 1296951'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 23201'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1404.801083'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -19509392.074483'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//27/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//27/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.722885 from 1237118 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.930657'</li>\n",
       "\t<li>'There were 1286793 total reads, 1286793 paired (1283584 properly mated), 2411 proper singles, 798 improper reads (722 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1283584 reads, 1283584 mapped, 0 unmapped, 1283583 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 40 placed, 2 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 722 reads, 722 mapped, 0 unmapped, 722 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2391 reads, 2391 mapped, 0 unmapped, 2391 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1286793'</li>\n",
       "\t<li>'Total placed reads: 1286790'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 3'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14491'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1206.802814'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -18952723.734257'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//29/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//29/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.788412 from 1228716 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.953392'</li>\n",
       "\t<li>'There were 1277577 total reads, 1277577 paired (1275142 properly mated), 1705 proper singles, 730 improper reads (654 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1275142 reads, 1275142 mapped, 0 unmapped, 1275142 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 35 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 654 reads, 654 mapped, 0 unmapped, 654 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1689 reads, 1689 mapped, 0 unmapped, 1689 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1277577'</li>\n",
       "\t<li>'Total placed reads: 1277576'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13429'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1114.200872'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16323526.728177'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//29/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//29/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.786097 from 1220526 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.954888'</li>\n",
       "\t<li>'There were 1269560 total reads, 1269560 paired (1266562 properly mated), 2030 proper singles, 968 improper reads (870 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1266562 reads, 1266562 mapped, 0 unmapped, 1266561 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 78 reads, 78 mapped, 0 unmapped, 78 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 870 reads, 870 mapped, 0 unmapped, 870 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2018 reads, 2018 mapped, 0 unmapped, 2018 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1269560'</li>\n",
       "\t<li>'Total placed reads: 1269559'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 9503'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1291.101017'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -15984512.357388'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//3/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//3/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.738839 from 1059914 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.917124'</li>\n",
       "\t<li>'There were 1104124 total reads, 1104124 paired (1099770 properly mated), 3128 proper singles, 1226 improper reads (1172 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1099770 reads, 1099770 mapped, 0 unmapped, 1099770 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1172 reads, 1172 mapped, 0 unmapped, 1172 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3074 reads, 3074 mapped, 0 unmapped, 3074 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1104124'</li>\n",
       "\t<li>'Total placed reads: 1104124'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 26445'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1913.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -23674874.444568'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//3/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//3/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.723814 from 1042270 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.912585'</li>\n",
       "\t<li>'There were 1086054 total reads, 1086054 paired (1081402 properly mated), 3390 proper singles, 1262 improper reads (1158 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1081402 reads, 1081402 mapped, 0 unmapped, 1081401 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1158 reads, 1158 mapped, 0 unmapped, 1158 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3368 reads, 3368 mapped, 0 unmapped, 3368 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1086054'</li>\n",
       "\t<li>'Total placed reads: 1086053'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 15980'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1774.101634'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22786161.840885'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//30/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//30/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.724625 from 983612 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.918317'</li>\n",
       "\t<li>'There were 1024938 total reads, 1024938 paired (1020408 properly mated), 3874 proper singles, 656 improper reads (600 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1020408 reads, 1020408 mapped, 0 unmapped, 1020407 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 600 reads, 600 mapped, 0 unmapped, 600 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3868 reads, 3868 mapped, 0 unmapped, 3868 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1024938'</li>\n",
       "\t<li>'Total placed reads: 1024937'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 39217'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 2030.501981'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -23545883.038263'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//30/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//30/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.710654 from 969954 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.912264'</li>\n",
       "\t<li>'There were 1011415 total reads, 1011415 paired (1006192 properly mated), 3897 proper singles, 1326 improper reads (1260 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1006192 reads, 1006192 mapped, 0 unmapped, 1006191 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1260 reads, 1260 mapped, 0 unmapped, 1260 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 3897 reads, 3897 mapped, 0 unmapped, 3897 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1011415'</li>\n",
       "\t<li>'Total placed reads: 1011414'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 21594'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1767.301747'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -22471465.103161'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//4/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//4/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.738952 from 1099898 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.923500'</li>\n",
       "\t<li>'There were 1143792 total reads, 1143792 paired (1141146 properly mated), 1752 proper singles, 894 improper reads (860 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1141146 reads, 1141146 mapped, 0 unmapped, 1141146 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1734 reads, 1734 mapped, 0 unmapped, 1734 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1143792'</li>\n",
       "\t<li>'Total placed reads: 1143792'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17197'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1344.800000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -14267214.886970'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//4/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//4/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.735895 from 1090712 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.924751'</li>\n",
       "\t<li>'There were 1134818 total reads, 1134818 paired (1131630 properly mated), 2128 proper singles, 1060 improper reads (962 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1131630 reads, 1131630 mapped, 0 unmapped, 1131630 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2120 reads, 2120 mapped, 0 unmapped, 2120 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1134818'</li>\n",
       "\t<li>'Total placed reads: 1134818'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13371'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1363.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -13782761.546427'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//5/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//5/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.700979 from 1150714 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.915817'</li>\n",
       "\t<li>'There were 1196310 total reads, 1196310 paired (1193536 properly mated), 2002 proper singles, 772 improper reads (702 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1193536 reads, 1193536 mapped, 0 unmapped, 1193536 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 702 reads, 702 mapped, 0 unmapped, 702 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 1986 reads, 1986 mapped, 0 unmapped, 1986 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1196310'</li>\n",
       "\t<li>'Total placed reads: 1196310'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19586'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1275.200000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -24820367.365228'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//5/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//5/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.704902 from 1140808 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.917229'</li>\n",
       "\t<li>'There were 1186595 total reads, 1186595 paired (1183274 properly mated), 2393 proper singles, 928 improper reads (798 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1183274 reads, 1183274 mapped, 0 unmapped, 1183273 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 116 reads, 116 mapped, 0 unmapped, 116 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2385 reads, 2385 mapped, 0 unmapped, 2385 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1186595'</li>\n",
       "\t<li>'Total placed reads: 1186594'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 13384'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1198.801010'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -24439262.268919'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//6/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//6/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.740631 from 1200058 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.923067'</li>\n",
       "\t<li>'There were 1248604 total reads, 1248604 paired (1245142 properly mated), 2540 proper singles, 922 improper reads (882 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1245142 reads, 1245142 mapped, 0 unmapped, 1245142 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 882 reads, 882 mapped, 0 unmapped, 882 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2538 reads, 2538 mapped, 0 unmapped, 2538 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1248604'</li>\n",
       "\t<li>'Total placed reads: 1248604'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 25160'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1599.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21668021.782460'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//6/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//6/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.738174 from 1188020 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.923106'</li>\n",
       "\t<li>'There were 1236593 total reads, 1236593 paired (1232608 properly mated), 2827 proper singles, 1158 improper reads (1124 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1232608 reads, 1232608 mapped, 0 unmapped, 1232608 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1124 reads, 1124 mapped, 0 unmapped, 1124 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2825 reads, 2825 mapped, 0 unmapped, 2825 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1236593'</li>\n",
       "\t<li>'Total placed reads: 1236593'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 17827'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1662.400000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -21045290.276025'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//7/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//7/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.731806 from 1270148 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.931106'</li>\n",
       "\t<li>'There were 1321322 total reads, 1321322 paired (1317878 properly mated), 2596 proper singles, 848 improper reads (724 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1317878 reads, 1317878 mapped, 0 unmapped, 1317877 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 60 reads, 60 mapped, 0 unmapped, 60 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 724 reads, 724 mapped, 0 unmapped, 724 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2558 reads, 2558 mapped, 0 unmapped, 2558 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1321322'</li>\n",
       "\t<li>'Total placed reads: 1321321'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 22187'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1369.501036'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16725120.051489'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//7/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//7/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.739212 from 1261184 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.935542'</li>\n",
       "\t<li>'There were 1312608 total reads, 1312608 paired (1308642 properly mated), 2796 proper singles, 1170 improper reads (1078 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1308642 reads, 1308642 mapped, 0 unmapped, 1308642 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 1078 reads, 1078 mapped, 0 unmapped, 1078 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2762 reads, 2762 mapped, 0 unmapped, 2762 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1312608'</li>\n",
       "\t<li>'Total placed reads: 1312608'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 16374'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1600.900000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -16404897.603518'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//8/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//8/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.716050 from 1056608 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.913943'</li>\n",
       "\t<li>'There were 1099743 total reads, 1099743 paired (1096198 properly mated), 2915 proper singles, 630 improper reads (572 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1096198 reads, 1096198 mapped, 0 unmapped, 1096197 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 572 reads, 572 mapped, 0 unmapped, 572 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2915 reads, 2915 mapped, 0 unmapped, 2915 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1099743'</li>\n",
       "\t<li>'Total placed reads: 1099742'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 30760'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1541.201401'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -27720412.514541'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//8/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//8/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.711122 from 1042420 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.913675'</li>\n",
       "\t<li>'There were 1085136 total reads, 1085136 paired (1081470 properly mated), 2656 proper singles, 1010 improper reads (962 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1081470 reads, 1081470 mapped, 0 unmapped, 1081469 placed, 1 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2646 reads, 2646 mapped, 0 unmapped, 2646 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1085136'</li>\n",
       "\t<li>'Total placed reads: 1085135'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 1'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 19772'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1166.901075'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -26982645.501738'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//9/megahit.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//9/megahit/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.733004 from 1150100 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.920223'</li>\n",
       "\t<li>'There were 1195606 total reads, 1195606 paired (1193070 properly mated), 2166 proper singles, 370 improper reads (354 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1193070 reads, 1193070 mapped, 0 unmapped, 1193070 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2160 reads, 2160 mapped, 0 unmapped, 2160 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1195606'</li>\n",
       "\t<li>'Total placed reads: 1195606'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 22245'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 1177.000000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -19579897.957545'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "\t<li><ol class=list-inline>\n",
       "\t<li>'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//9/metaspades.bam'</li>\n",
       "\t<li>'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//9/metaspades/contigs_filtered.fasta'</li>\n",
       "\t<li>'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt'</li>\n",
       "\t<li>'Reading in assembly...'</li>\n",
       "\t<li>'Reading in the map and computing statistics...'</li>\n",
       "\t<li>'Insert length and std not given, will be calculated from input map.'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'Found FR sample avg insert length to be 253.713040 from 1139810 mapped reads'</li>\n",
       "\t<li>'Found FR sample insert length std to be 9.913492'</li>\n",
       "\t<li>'There were 1185114 total reads, 1185114 paired (1182324 properly mated), 2034 proper singles, 756 improper reads (748 chimeric). (0 reads were unmapped)'</li>\n",
       "\t<li>'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt.param'</li>\n",
       "\t<li>'Computing read placements and depths'</li>\n",
       "\t<li>'Read 1000000 reads...'</li>\n",
       "\t<li>'FR orientation with 1182324 reads, 1182324 mapped, 0 unmapped, 1182324 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'CHIMER orientation with 748 reads, 748 mapped, 0 unmapped, 748 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_UNMAPPED_MATE orientation with 2034 reads, 2034 mapped, 0 unmapped, 2034 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'</li>\n",
       "\t<li>'Total mapped reads: 1185114'</li>\n",
       "\t<li>'Total placed reads: 1185114'</li>\n",
       "\t<li>'Total unmapped reads: 0'</li>\n",
       "\t<li>'Total unplaced reads: 0'</li>\n",
       "\t<li>'Computing k-mer statistics...'</li>\n",
       "\t<li>'Done computing k-mer statistics.'</li>\n",
       "\t<li>'Computing depth statistics...'</li>\n",
       "\t<li>'bases with no GC depth correction metric (small contigs): 0'</li>\n",
       "\t<li>'unmappable bases: 0 across 0 different regions'</li>\n",
       "\t<li>'bases with 0 coverage: 14551'</li>\n",
       "\t<li>'Done computing depth statistics.'</li>\n",
       "\t<li>'Computing statistics on expected missing...'</li>\n",
       "\t<li>'Expected extra length: 864.600000'</li>\n",
       "\t<li>'Done computing statistics on expected missing.'</li>\n",
       "\t<li>'Set minLogLike to: -120.0'</li>\n",
       "\t<li>'Total ALE Score: -18995038.934677'</li>\n",
       "\t<li>'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt'</li>\n",
       "\t<li>'Closing input file'</li>\n",
       "</ol>\n",
       "</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate}\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.723707 from 1212220 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.925705'\n",
       "\\item 'There were 1260475 total reads, 1260475 paired (1257682 properly mated), 2073 proper singles, 720 improper reads (690 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1257682 reads, 1257682 mapped, 0 unmapped, 1257681 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 690 reads, 690 mapped, 0 unmapped, 690 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2043 reads, 2043 mapped, 0 unmapped, 2043 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1260475'\n",
       "\\item 'Total placed reads: 1260474'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17197'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1203.800955'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22697096.942980'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//1/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//1/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.732010 from 1201202 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.931224'\n",
       "\\item 'There were 1249353 total reads, 1249353 paired (1246230 properly mated), 2167 proper singles, 956 improper reads (912 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1246230 reads, 1246230 mapped, 0 unmapped, 1246230 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 912 reads, 912 mapped, 0 unmapped, 912 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2133 reads, 2133 mapped, 0 unmapped, 2133 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1249353'\n",
       "\\item 'Total placed reads: 1249353'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11647'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1221.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22172524.551273'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//10/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//10/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.735681 from 1270902 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.930199'\n",
       "\\item 'There were 1321605 total reads, 1321605 paired (1318686 properly mated), 1987 proper singles, 932 improper reads (880 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1318686 reads, 1318686 mapped, 0 unmapped, 1318686 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 880 reads, 880 mapped, 0 unmapped, 880 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1973 reads, 1973 mapped, 0 unmapped, 1973 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1321605'\n",
       "\\item 'Total placed reads: 1321605'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15898'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1278.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -19122156.831738'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//10/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//10/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.730003 from 1262138 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.927099'\n",
       "\\item 'There were 1312751 total reads, 1312751 paired (1309554 properly mated), 2309 proper singles, 888 improper reads (836 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1309554 reads, 1309554 mapped, 0 unmapped, 1309553 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2289 reads, 2289 mapped, 0 unmapped, 2289 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1312751'\n",
       "\\item 'Total placed reads: 1312750'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9767'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1267.800966'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -18755636.338068'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//11/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//11/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.769628 from 1229856 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.945784'\n",
       "\\item 'There were 1278594 total reads, 1278594 paired (1276032 properly mated), 1706 proper singles, 856 improper reads (778 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1276032 reads, 1276032 mapped, 0 unmapped, 1276031 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 778 reads, 778 mapped, 0 unmapped, 778 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1674 reads, 1674 mapped, 0 unmapped, 1674 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1278594'\n",
       "\\item 'Total placed reads: 1278593'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13435'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1076.300842'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12552137.564932'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//11/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//11/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.750257 from 1222898 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.940087'\n",
       "\\item 'There were 1271913 total reads, 1271913 paired (1268674 properly mated), 1953 proper singles, 1286 improper reads (1220 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1268674 reads, 1268674 mapped, 0 unmapped, 1268674 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 40 reads, 40 mapped, 0 unmapped, 39 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1220 reads, 1220 mapped, 0 unmapped, 1220 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1909 reads, 1909 mapped, 0 unmapped, 1909 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1271913'\n",
       "\\item 'Total placed reads: 1271912'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12144'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1243.700978'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -12235260.934613'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//12/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//12/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//12/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.769835 from 1441024 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.947323'\n",
       "\\item 'There were 1498680 total reads, 1498680 paired (1495500 properly mated), 2118 proper singles, 1062 improper reads (1004 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//12/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1495500 reads, 1495500 mapped, 0 unmapped, 1495495 placed, 5 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2072 reads, 2072 mapped, 0 unmapped, 2072 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1498680'\n",
       "\\item 'Total placed reads: 1498675'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 5'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13692'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1502.605013'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -17965987.100460'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//12/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//12/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//12/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//12/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.754574 from 1434354 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.941442'\n",
       "\\item 'There were 1491690 total reads, 1491690 paired (1488456 properly mated), 2248 proper singles, 986 improper reads (958 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//12/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1488456 reads, 1488456 mapped, 0 unmapped, 1488453 placed, 3 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 958 reads, 958 mapped, 0 unmapped, 958 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2228 reads, 2228 mapped, 0 unmapped, 2228 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1491690'\n",
       "\\item 'Total placed reads: 1491687'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 3'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11700'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1390.202796'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -17661706.357633'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//12/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//13/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//13/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//13/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.765196 from 1040338 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.932119'\n",
       "\\item 'There were 1082811 total reads, 1082811 paired (1079514 properly mated), 2755 proper singles, 542 improper reads (504 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//13/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1079514 reads, 1079514 mapped, 0 unmapped, 1079514 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 504 reads, 504 mapped, 0 unmapped, 504 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2753 reads, 2753 mapped, 0 unmapped, 2753 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1082811'\n",
       "\\item 'Total placed reads: 1082811'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 29053'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1845.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -29108120.735174'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//13/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//13/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//13/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//13/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.742836 from 1026902 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.920751'\n",
       "\\item 'There were 1068969 total reads, 1068969 paired (1065480 properly mated), 2609 proper singles, 880 improper reads (854 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//13/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1065480 reads, 1065480 mapped, 0 unmapped, 1065480 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 854 reads, 854 mapped, 0 unmapped, 854 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2609 reads, 2609 mapped, 0 unmapped, 2609 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1068969'\n",
       "\\item 'Total placed reads: 1068969'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17762'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1093.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -28177219.593090'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//13/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//14/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//14/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//14/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.779876 from 1026678 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.938274'\n",
       "\\item 'There were 1067657 total reads, 1067657 paired (1065468 properly mated), 1933 proper singles, 256 improper reads (246 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//14/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1065468 reads, 1065468 mapped, 0 unmapped, 1065468 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1929 reads, 1929 mapped, 0 unmapped, 1929 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1067657'\n",
       "\\item 'Total placed reads: 1067657'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19976'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1050.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -23112358.285221'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//14/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//14/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//14/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//14/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.775480 from 1017850 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.938990'\n",
       "\\item 'There were 1058508 total reads, 1058508 paired (1056270 properly mated), 1756 proper singles, 482 improper reads (448 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//14/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1056270 reads, 1056270 mapped, 0 unmapped, 1056270 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1750 reads, 1750 mapped, 0 unmapped, 1750 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1058508'\n",
       "\\item 'Total placed reads: 1058508'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12641'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 721.700000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22517914.298963'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//14/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//15/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//15/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//15/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.750549 from 1169866 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.930211'\n",
       "\\item 'There were 1216436 total reads, 1216436 paired (1213808 properly mated), 2268 proper singles, 360 improper reads (326 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//15/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1213808 reads, 1213808 mapped, 0 unmapped, 1213808 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2260 reads, 2260 mapped, 0 unmapped, 2260 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1216436'\n",
       "\\item 'Total placed reads: 1216436'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 23858'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1417.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -18167994.130481'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//15/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//15/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//15/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//15/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.726422 from 1160328 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.920425'\n",
       "\\item 'There were 1206622 total reads, 1206622 paired (1203812 properly mated), 2148 proper singles, 662 improper reads (608 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//15/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1203812 reads, 1203812 mapped, 0 unmapped, 1203811 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2140 reads, 2140 mapped, 0 unmapped, 2140 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1206622'\n",
       "\\item 'Total placed reads: 1206621'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14959'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1035.500858'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -17494904.019029'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//15/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//16/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//16/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//16/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.674769 from 1312060 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.906746'\n",
       "\\item 'There were 1363096 total reads, 1363096 paired (1360784 properly mated), 1860 proper singles, 452 improper reads (402 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//16/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1360784 reads, 1360784 mapped, 0 unmapped, 1360781 placed, 3 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1846 reads, 1846 mapped, 0 unmapped, 1846 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1363096'\n",
       "\\item 'Total placed reads: 1363093'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 3'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15590'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1039.902289'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -19575398.337141'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//16/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//16/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//16/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//16/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.665849 from 1304080 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.903611'\n",
       "\\item 'There were 1355039 total reads, 1355039 paired (1352438 properly mated), 1835 proper singles, 766 improper reads (710 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//16/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1352438 reads, 1352438 mapped, 0 unmapped, 1352438 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 710 reads, 710 mapped, 0 unmapped, 710 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1831 reads, 1831 mapped, 0 unmapped, 1831 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1355039'\n",
       "\\item 'Total placed reads: 1355039'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9252'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 966.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -19152471.917716'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//16/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//17/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//17/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//17/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.714297 from 1056452 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.917958'\n",
       "\\item 'There were 1098861 total reads, 1098861 paired (1095938 properly mated), 2365 proper singles, 558 improper reads (536 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//17/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1095938 reads, 1095938 mapped, 0 unmapped, 1095938 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 536 reads, 536 mapped, 0 unmapped, 536 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2357 reads, 2357 mapped, 0 unmapped, 2357 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1098861'\n",
       "\\item 'Total placed reads: 1098861'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 21615'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1217.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21855528.539573'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//17/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//17/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//17/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//17/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.708831 from 1047290 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.914692'\n",
       "\\item 'There were 1089261 total reads, 1089261 paired (1086416 properly mated), 2189 proper singles, 656 improper reads (628 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//17/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1086416 reads, 1086416 mapped, 0 unmapped, 1086416 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 628 reads, 628 mapped, 0 unmapped, 628 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2185 reads, 2185 mapped, 0 unmapped, 2185 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1089261'\n",
       "\\item 'Total placed reads: 1089261'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15593'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 904.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21295303.741731'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//17/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//18/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//18/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//18/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.784092 from 1032744 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.942105'\n",
       "\\item 'There were 1077664 total reads, 1077664 paired (1071728 properly mated), 5294 proper singles, 642 improper reads (624 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//18/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1071728 reads, 1071728 mapped, 0 unmapped, 1071728 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 624 reads, 624 mapped, 0 unmapped, 624 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 5288 reads, 5288 mapped, 0 unmapped, 5288 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1077664'\n",
       "\\item 'Total placed reads: 1077664'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 63004'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2727.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -30555470.646073'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//18/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//18/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//18/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//18/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.757966 from 994686 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.934278'\n",
       "\\item 'There were 1039080 total reads, 1039080 paired (1031998 properly mated), 6212 proper singles, 870 improper reads (846 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//18/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1031998 reads, 1031998 mapped, 0 unmapped, 1031998 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 846 reads, 846 mapped, 0 unmapped, 846 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 6210 reads, 6210 mapped, 0 unmapped, 6210 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1039080'\n",
       "\\item 'Total placed reads: 1039080'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 32109'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2378.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -28364110.245962'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//18/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//19/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//19/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//19/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.758838 from 1444970 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.947463'\n",
       "\\item 'There were 1503635 total reads, 1503635 paired (1499582 properly mated), 2255 proper singles, 1798 improper reads (1720 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//19/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1499582 reads, 1499582 mapped, 0 unmapped, 1499581 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1720 reads, 1720 mapped, 0 unmapped, 1720 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2219 reads, 2219 mapped, 0 unmapped, 2219 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1503635'\n",
       "\\item 'Total placed reads: 1503634'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11265'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1860.201237'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15588448.646139'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//19/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//19/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//19/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//19/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.744043 from 1430676 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.941927'\n",
       "\\item 'There were 1489966 total reads, 1489966 paired (1484608 properly mated), 3708 proper singles, 1650 improper reads (1564 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//19/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1484608 reads, 1484608 mapped, 0 unmapped, 1484608 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1564 reads, 1564 mapped, 0 unmapped, 1564 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3696 reads, 3696 mapped, 0 unmapped, 3696 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1489966'\n",
       "\\item 'Total placed reads: 1489966'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 7489'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2466.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15083716.468382'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//19/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//2/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//2/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//2/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.707842 from 1162692 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.911909'\n",
       "\\item 'There were 1209478 total reads, 1209478 paired (1206160 properly mated), 2218 proper singles, 1100 improper reads (1062 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//2/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1206160 reads, 1206160 mapped, 0 unmapped, 1206160 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1062 reads, 1062 mapped, 0 unmapped, 1062 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2210 reads, 2210 mapped, 0 unmapped, 2210 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1209478'\n",
       "\\item 'Total placed reads: 1209478'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16641'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1409.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21502554.168959'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//2/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//2/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//2/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//2/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.708035 from 1149766 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.911943'\n",
       "\\item 'There were 1196329 total reads, 1196329 paired (1192696 properly mated), 2801 proper singles, 832 improper reads (810 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//2/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1192696 reads, 1192696 mapped, 0 unmapped, 1192696 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 810 reads, 810 mapped, 0 unmapped, 810 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2801 reads, 2801 mapped, 0 unmapped, 2801 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1196329'\n",
       "\\item 'Total placed reads: 1196329'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 12478'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1527.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -20946198.398444'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//2/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//20/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//20/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//20/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.698839 from 929876 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.900362'\n",
       "\\item 'There were 966664 total reads, 966664 paired (964368 properly mated), 1996 proper singles, 300 improper reads (284 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//20/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 964368 reads, 964368 mapped, 0 unmapped, 964368 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 284 reads, 284 mapped, 0 unmapped, 284 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1984 reads, 1984 mapped, 0 unmapped, 1984 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 966664'\n",
       "\\item 'Total placed reads: 966664'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 18701'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1107.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -18099506.304873'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//20/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//20/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//20/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//20/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Found FR sample avg insert length to be 253.691066 from 920578 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.895893'\n",
       "\\item 'There were 957156 total reads, 957156 paired (954662 properly mated), 1950 proper singles, 544 improper reads (520 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//20/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'FR orientation with 954662 reads, 954662 mapped, 0 unmapped, 954662 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 520 reads, 520 mapped, 0 unmapped, 520 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1928 reads, 1928 mapped, 0 unmapped, 1928 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 957156'\n",
       "\\item 'Total placed reads: 957156'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13037'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 838.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -17479146.406151'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//20/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//21/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//21/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//21/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.783618 from 1315482 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.951948'\n",
       "\\item 'There were 1367541 total reads, 1367541 paired (1365056 properly mated), 1629 proper singles, 856 improper reads (790 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//21/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1365056 reads, 1365056 mapped, 0 unmapped, 1365056 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 790 reads, 790 mapped, 0 unmapped, 790 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1623 reads, 1623 mapped, 0 unmapped, 1623 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1367541'\n",
       "\\item 'Total placed reads: 1367541'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11981'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1166.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22004557.470929'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//21/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//21/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//21/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//21/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.774182 from 1307486 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.947431'\n",
       "\\item 'There were 1360008 total reads, 1360008 paired (1356718 properly mated), 2242 proper singles, 1048 improper reads (1004 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//21/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1356718 reads, 1356718 mapped, 0 unmapped, 1356718 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2234 reads, 2234 mapped, 0 unmapped, 2234 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1360008'\n",
       "\\item 'Total placed reads: 1360008'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8679'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1424.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21546080.209347'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//21/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//22/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//22/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//22/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.758738 from 1107196 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.936306'\n",
       "\\item 'There were 1153199 total reads, 1153199 paired (1148874 properly mated), 2607 proper singles, 1718 improper reads (1640 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//22/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1148874 reads, 1148874 mapped, 0 unmapped, 1148874 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 38 reads, 38 mapped, 0 unmapped, 37 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1640 reads, 1640 mapped, 0 unmapped, 1640 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2595 reads, 2595 mapped, 0 unmapped, 2595 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1153199'\n",
       "\\item 'Total placed reads: 1153198'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14542'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2070.801796'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -28105795.208026'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//22/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//22/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//22/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//22/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.750389 from 1088718 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.931532'\n",
       "\\item 'There were 1133938 total reads, 1133938 paired (1129594 properly mated), 3196 proper singles, 1148 improper reads (1110 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//22/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1129594 reads, 1129594 mapped, 0 unmapped, 1129593 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1110 reads, 1110 mapped, 0 unmapped, 1110 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3196 reads, 3196 mapped, 0 unmapped, 3196 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1133938'\n",
       "\\item 'Total placed reads: 1133937'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8482'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2082.501837'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -26800705.802743'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//22/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//23/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//23/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//23/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.717675 from 1100808 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.915271'\n",
       "\\item 'There were 1144566 total reads, 1144566 paired (1142120 properly mated), 1924 proper singles, 522 improper reads (462 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//23/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1142120 reads, 1142120 mapped, 0 unmapped, 1142120 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 462 reads, 462 mapped, 0 unmapped, 462 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1912 reads, 1912 mapped, 0 unmapped, 1912 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1144566'\n",
       "\\item 'Total placed reads: 1144566'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14963'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1172.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14831360.378148'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//23/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//23/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//23/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//23/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.718760 from 1092852 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.918076'\n",
       "\\item 'There were 1136709 total reads, 1136709 paired (1133854 properly mated), 1947 proper singles, 908 improper reads (798 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//23/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1133854 reads, 1133854 mapped, 0 unmapped, 1133854 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1941 reads, 1941 mapped, 0 unmapped, 1941 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1136709'\n",
       "\\item 'Total placed reads: 1136709'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 11931'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1125.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14386723.450540'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//23/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//24/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//24/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//24/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.751733 from 1288410 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.937254'\n",
       "\\item 'There were 1338708 total reads, 1338708 paired (1336816 properly mated), 1224 proper singles, 668 improper reads (604 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//24/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1336816 reads, 1336816 mapped, 0 unmapped, 1336816 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1338708'\n",
       "\\item 'Total placed reads: 1338708'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 8237'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 864.500000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11592032.231556'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//24/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//24/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//24/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//24/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.743318 from 1283372 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.934212'\n",
       "\\item 'There were 1333954 total reads, 1333954 paired (1331536 properly mated), 1730 proper singles, 688 improper reads (608 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//24/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1331536 reads, 1331536 mapped, 0 unmapped, 1331536 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 76 reads, 76 mapped, 0 unmapped, 76 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1724 reads, 1724 mapped, 0 unmapped, 1724 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1333954'\n",
       "\\item 'Total placed reads: 1333954'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 6248'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1017.100000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -11400899.152586'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//24/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//25/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//25/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//25/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.766508 from 1452908 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.947885'\n",
       "\\item 'There were 1511420 total reads, 1511420 paired (1507552 properly mated), 2286 proper singles, 1582 improper reads (1534 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//25/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1507552 reads, 1507552 mapped, 0 unmapped, 1507552 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1534 reads, 1534 mapped, 0 unmapped, 1534 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2266 reads, 2266 mapped, 0 unmapped, 2266 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1511420'\n",
       "\\item 'Total placed reads: 1511420'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14303'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1969.861129'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -40241263.112110'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//25/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//25/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//25/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//25/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.762489 from 1444882 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.944863'\n",
       "\\item 'There were 1503090 total reads, 1503090 paired (1499156 properly mated), 2700 proper singles, 1234 improper reads (1026 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//25/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1499156 reads, 1499156 mapped, 0 unmapped, 1499156 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 184 reads, 184 mapped, 0 unmapped, 184 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1026 reads, 1026 mapped, 0 unmapped, 1026 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2690 reads, 2690 mapped, 0 unmapped, 2690 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1503090'\n",
       "\\item 'Total placed reads: 1503090'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9385'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1992.625081'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -39926611.413909'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//25/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//26/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//26/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//26/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.761931 from 1237382 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.934695'\n",
       "\\item 'There were 1290127 total reads, 1290127 paired (1283976 properly mated), 3499 proper singles, 2652 improper reads (2600 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//26/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1283976 reads, 1283976 mapped, 0 unmapped, 1283976 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 24 reads, 24 mapped, 0 unmapped, 23 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 2600 reads, 2600 mapped, 0 unmapped, 2600 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3485 reads, 3485 mapped, 0 unmapped, 3485 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1290127'\n",
       "\\item 'Total placed reads: 1290126'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 28963'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2734.102119'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21738734.283024'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//26/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//26/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//26/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//26/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.756475 from 1221778 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.932433'\n",
       "\\item 'There were 1274647 total reads, 1274647 paired (1267720 properly mated), 4839 proper singles, 2088 improper reads (2022 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//26/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1267720 reads, 1267720 mapped, 0 unmapped, 1267720 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 2022 reads, 2022 mapped, 0 unmapped, 2022 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 4827 reads, 4827 mapped, 0 unmapped, 4827 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1274647'\n",
       "\\item 'Total placed reads: 1274647'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19692'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2942.300000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21013802.246661'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//26/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//27/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//27/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//27/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.732755 from 1247126 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.932042'\n",
       "\\item 'There were 1296952 total reads, 1296952 paired (1294018 properly mated), 2364 proper singles, 570 improper reads (508 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//27/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1294018 reads, 1294018 mapped, 0 unmapped, 1294017 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 508 reads, 508 mapped, 0 unmapped, 508 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2346 reads, 2346 mapped, 0 unmapped, 2346 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1296952'\n",
       "\\item 'Total placed reads: 1296951'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 23201'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1404.801083'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -19509392.074483'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//27/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//27/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//27/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//27/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.722885 from 1237118 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.930657'\n",
       "\\item 'There were 1286793 total reads, 1286793 paired (1283584 properly mated), 2411 proper singles, 798 improper reads (722 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//27/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1283584 reads, 1283584 mapped, 0 unmapped, 1283583 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 42 reads, 42 mapped, 0 unmapped, 40 placed, 2 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 722 reads, 722 mapped, 0 unmapped, 722 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2391 reads, 2391 mapped, 0 unmapped, 2391 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1286793'\n",
       "\\item 'Total placed reads: 1286790'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 3'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14491'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1206.802814'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -18952723.734257'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//27/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//29/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//29/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//29/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.788412 from 1228716 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.953392'\n",
       "\\item 'There were 1277577 total reads, 1277577 paired (1275142 properly mated), 1705 proper singles, 730 improper reads (654 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//29/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1275142 reads, 1275142 mapped, 0 unmapped, 1275142 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 36 reads, 36 mapped, 0 unmapped, 35 placed, 1 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 654 reads, 654 mapped, 0 unmapped, 654 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1689 reads, 1689 mapped, 0 unmapped, 1689 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1277577'\n",
       "\\item 'Total placed reads: 1277576'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13429'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1114.200872'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16323526.728177'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//29/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//29/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//29/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//29/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.786097 from 1220526 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.954888'\n",
       "\\item 'There were 1269560 total reads, 1269560 paired (1266562 properly mated), 2030 proper singles, 968 improper reads (870 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//29/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1266562 reads, 1266562 mapped, 0 unmapped, 1266561 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 78 reads, 78 mapped, 0 unmapped, 78 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 870 reads, 870 mapped, 0 unmapped, 870 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2018 reads, 2018 mapped, 0 unmapped, 2018 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1269560'\n",
       "\\item 'Total placed reads: 1269559'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 9503'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1291.101017'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -15984512.357388'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//29/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//3/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//3/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//3/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.738839 from 1059914 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.917124'\n",
       "\\item 'There were 1104124 total reads, 1104124 paired (1099770 properly mated), 3128 proper singles, 1226 improper reads (1172 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//3/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1099770 reads, 1099770 mapped, 0 unmapped, 1099770 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1172 reads, 1172 mapped, 0 unmapped, 1172 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3074 reads, 3074 mapped, 0 unmapped, 3074 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1104124'\n",
       "\\item 'Total placed reads: 1104124'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 26445'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1913.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -23674874.444568'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//3/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//3/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//3/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//3/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.723814 from 1042270 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.912585'\n",
       "\\item 'There were 1086054 total reads, 1086054 paired (1081402 properly mated), 3390 proper singles, 1262 improper reads (1158 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//3/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1081402 reads, 1081402 mapped, 0 unmapped, 1081401 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1158 reads, 1158 mapped, 0 unmapped, 1158 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3368 reads, 3368 mapped, 0 unmapped, 3368 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1086054'\n",
       "\\item 'Total placed reads: 1086053'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 15980'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1774.101634'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22786161.840885'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//3/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//30/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//30/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//30/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.724625 from 983612 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.918317'\n",
       "\\item 'There were 1024938 total reads, 1024938 paired (1020408 properly mated), 3874 proper singles, 656 improper reads (600 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//30/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1020408 reads, 1020408 mapped, 0 unmapped, 1020407 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 600 reads, 600 mapped, 0 unmapped, 600 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3868 reads, 3868 mapped, 0 unmapped, 3868 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1024938'\n",
       "\\item 'Total placed reads: 1024937'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 39217'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 2030.501981'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -23545883.038263'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//30/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//30/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//30/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//30/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.710654 from 969954 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.912264'\n",
       "\\item 'There were 1011415 total reads, 1011415 paired (1006192 properly mated), 3897 proper singles, 1326 improper reads (1260 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//30/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1006192 reads, 1006192 mapped, 0 unmapped, 1006191 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1260 reads, 1260 mapped, 0 unmapped, 1260 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 3897 reads, 3897 mapped, 0 unmapped, 3897 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1011415'\n",
       "\\item 'Total placed reads: 1011414'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 21594'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1767.301747'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -22471465.103161'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//30/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//4/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//4/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//4/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.738952 from 1099898 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.923500'\n",
       "\\item 'There were 1143792 total reads, 1143792 paired (1141146 properly mated), 1752 proper singles, 894 improper reads (860 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//4/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1141146 reads, 1141146 mapped, 0 unmapped, 1141146 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1734 reads, 1734 mapped, 0 unmapped, 1734 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1143792'\n",
       "\\item 'Total placed reads: 1143792'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17197'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1344.800000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -14267214.886970'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//4/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//4/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//4/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//4/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.735895 from 1090712 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.924751'\n",
       "\\item 'There were 1134818 total reads, 1134818 paired (1131630 properly mated), 2128 proper singles, 1060 improper reads (962 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//4/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1131630 reads, 1131630 mapped, 0 unmapped, 1131630 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2120 reads, 2120 mapped, 0 unmapped, 2120 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1134818'\n",
       "\\item 'Total placed reads: 1134818'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13371'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1363.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -13782761.546427'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//4/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//5/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//5/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//5/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.700979 from 1150714 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.915817'\n",
       "\\item 'There were 1196310 total reads, 1196310 paired (1193536 properly mated), 2002 proper singles, 772 improper reads (702 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//5/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1193536 reads, 1193536 mapped, 0 unmapped, 1193536 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 702 reads, 702 mapped, 0 unmapped, 702 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 1986 reads, 1986 mapped, 0 unmapped, 1986 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1196310'\n",
       "\\item 'Total placed reads: 1196310'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19586'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1275.200000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -24820367.365228'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//5/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//5/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//5/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//5/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.704902 from 1140808 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.917229'\n",
       "\\item 'There were 1186595 total reads, 1186595 paired (1183274 properly mated), 2393 proper singles, 928 improper reads (798 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//5/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1183274 reads, 1183274 mapped, 0 unmapped, 1183273 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 116 reads, 116 mapped, 0 unmapped, 116 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2385 reads, 2385 mapped, 0 unmapped, 2385 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1186595'\n",
       "\\item 'Total placed reads: 1186594'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 13384'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1198.801010'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -24439262.268919'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//5/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//6/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//6/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//6/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.740631 from 1200058 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.923067'\n",
       "\\item 'There were 1248604 total reads, 1248604 paired (1245142 properly mated), 2540 proper singles, 922 improper reads (882 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//6/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1245142 reads, 1245142 mapped, 0 unmapped, 1245142 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 882 reads, 882 mapped, 0 unmapped, 882 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2538 reads, 2538 mapped, 0 unmapped, 2538 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1248604'\n",
       "\\item 'Total placed reads: 1248604'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 25160'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1599.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21668021.782460'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//6/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//6/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//6/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//6/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.738174 from 1188020 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.923106'\n",
       "\\item 'There were 1236593 total reads, 1236593 paired (1232608 properly mated), 2827 proper singles, 1158 improper reads (1124 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//6/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1232608 reads, 1232608 mapped, 0 unmapped, 1232608 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1124 reads, 1124 mapped, 0 unmapped, 1124 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2825 reads, 2825 mapped, 0 unmapped, 2825 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1236593'\n",
       "\\item 'Total placed reads: 1236593'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 17827'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1662.400000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -21045290.276025'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//6/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//7/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//7/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//7/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.731806 from 1270148 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.931106'\n",
       "\\item 'There were 1321322 total reads, 1321322 paired (1317878 properly mated), 2596 proper singles, 848 improper reads (724 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//7/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1317878 reads, 1317878 mapped, 0 unmapped, 1317877 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 60 reads, 60 mapped, 0 unmapped, 60 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 724 reads, 724 mapped, 0 unmapped, 724 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2558 reads, 2558 mapped, 0 unmapped, 2558 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1321322'\n",
       "\\item 'Total placed reads: 1321321'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 22187'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1369.501036'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16725120.051489'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//7/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//7/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//7/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//7/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.739212 from 1261184 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.935542'\n",
       "\\item 'There were 1312608 total reads, 1312608 paired (1308642 properly mated), 2796 proper singles, 1170 improper reads (1078 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//7/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1308642 reads, 1308642 mapped, 0 unmapped, 1308642 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 1078 reads, 1078 mapped, 0 unmapped, 1078 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2762 reads, 2762 mapped, 0 unmapped, 2762 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1312608'\n",
       "\\item 'Total placed reads: 1312608'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 16374'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1600.900000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -16404897.603518'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//7/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//8/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//8/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//8/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.716050 from 1056608 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.913943'\n",
       "\\item 'There were 1099743 total reads, 1099743 paired (1096198 properly mated), 2915 proper singles, 630 improper reads (572 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//8/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1096198 reads, 1096198 mapped, 0 unmapped, 1096197 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 572 reads, 572 mapped, 0 unmapped, 572 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2915 reads, 2915 mapped, 0 unmapped, 2915 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1099743'\n",
       "\\item 'Total placed reads: 1099742'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 30760'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1541.201401'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -27720412.514541'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//8/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//8/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//8/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//8/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.711122 from 1042420 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.913675'\n",
       "\\item 'There were 1085136 total reads, 1085136 paired (1081470 properly mated), 2656 proper singles, 1010 improper reads (962 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//8/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1081470 reads, 1081470 mapped, 0 unmapped, 1081469 placed, 1 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2646 reads, 2646 mapped, 0 unmapped, 2646 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1085136'\n",
       "\\item 'Total placed reads: 1085135'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 1'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 19772'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1166.901075'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -26982645.501738'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//8/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//9/megahit.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//9/megahit/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//9/megahit.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.733004 from 1150100 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.920223'\n",
       "\\item 'There were 1195606 total reads, 1195606 paired (1193070 properly mated), 2166 proper singles, 370 improper reads (354 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//9/megahit.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1193070 reads, 1193070 mapped, 0 unmapped, 1193070 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2160 reads, 2160 mapped, 0 unmapped, 2160 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1195606'\n",
       "\\item 'Total placed reads: 1195606'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 22245'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 1177.000000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -19579897.957545'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//9/megahit.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item \\begin{enumerate*}\n",
       "\\item 'BAM file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/map//9/metaspades.bam'\n",
       "\\item 'Assembly fasta file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/assembly//9/metaspades/contigs\\_filtered.fasta'\n",
       "\\item 'ALE Output file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//9/metaspades.txt'\n",
       "\\item 'Reading in assembly...'\n",
       "\\item 'Reading in the map and computing statistics...'\n",
       "\\item 'Insert length and std not given, will be calculated from input map.'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'Found FR sample avg insert length to be 253.713040 from 1139810 mapped reads'\n",
       "\\item 'Found FR sample insert length std to be 9.913492'\n",
       "\\item 'There were 1185114 total reads, 1185114 paired (1182324 properly mated), 2034 proper singles, 756 improper reads (748 chimeric). (0 reads were unmapped)'\n",
       "\\item 'Saved library parameters to /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//9/metaspades.txt.param'\n",
       "\\item 'Computing read placements and depths'\n",
       "\\item 'Read 1000000 reads...'\n",
       "\\item 'FR orientation with 1182324 reads, 1182324 mapped, 0 unmapped, 1182324 placed, 0 mappedButNotPlaced'\n",
       "\\item 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NOT\\_PROPER\\_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'CHIMER orientation with 748 reads, 748 mapped, 0 unmapped, 748 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_UNMAPPED\\_MATE orientation with 2034 reads, 2034 mapped, 0 unmapped, 2034 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'HALF\\_VALID\\_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'SINGLE\\_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'UNMAPPED\\_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'NO\\_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "\\item 'Total mapped reads: 1185114'\n",
       "\\item 'Total placed reads: 1185114'\n",
       "\\item 'Total unmapped reads: 0'\n",
       "\\item 'Total unplaced reads: 0'\n",
       "\\item 'Computing k-mer statistics...'\n",
       "\\item 'Done computing k-mer statistics.'\n",
       "\\item 'Computing depth statistics...'\n",
       "\\item 'bases with no GC depth correction metric (small contigs): 0'\n",
       "\\item 'unmappable bases: 0 across 0 different regions'\n",
       "\\item 'bases with 0 coverage: 14551'\n",
       "\\item 'Done computing depth statistics.'\n",
       "\\item 'Computing statistics on expected missing...'\n",
       "\\item 'Expected extra length: 864.600000'\n",
       "\\item 'Done computing statistics on expected missing.'\n",
       "\\item 'Set minLogLike to: -120.0'\n",
       "\\item 'Total ALE Score: -18995038.934677'\n",
       "\\item 'Output is in file: /ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//9/metaspades.txt'\n",
       "\\item 'Closing input file'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\end{enumerate}\n"
      ],
      "text/markdown": [
       "1. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.723707 from 1212220 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.925705'\n",
       "10. 'There were 1260475 total reads, 1260475 paired (1257682 properly mated), 2073 proper singles, 720 improper reads (690 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1257682 reads, 1257682 mapped, 0 unmapped, 1257681 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 690 reads, 690 mapped, 0 unmapped, 690 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2043 reads, 2043 mapped, 0 unmapped, 2043 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1260475'\n",
       "29. 'Total placed reads: 1260474'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 17197'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1203.800955'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22697096.942980'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "2. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.732010 from 1201202 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.931224'\n",
       "10. 'There were 1249353 total reads, 1249353 paired (1246230 properly mated), 2167 proper singles, 956 improper reads (912 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1246230 reads, 1246230 mapped, 0 unmapped, 1246230 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 912 reads, 912 mapped, 0 unmapped, 912 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2133 reads, 2133 mapped, 0 unmapped, 2133 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1249353'\n",
       "29. 'Total placed reads: 1249353'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 11647'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1221.800000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22172524.551273'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "3. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.735681 from 1270902 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.930199'\n",
       "10. 'There were 1321605 total reads, 1321605 paired (1318686 properly mated), 1987 proper singles, 932 improper reads (880 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1318686 reads, 1318686 mapped, 0 unmapped, 1318686 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 880 reads, 880 mapped, 0 unmapped, 880 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1973 reads, 1973 mapped, 0 unmapped, 1973 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1321605'\n",
       "29. 'Total placed reads: 1321605'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 15898'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1278.400000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -19122156.831738'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "4. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.730003 from 1262138 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.927099'\n",
       "10. 'There were 1312751 total reads, 1312751 paired (1309554 properly mated), 2309 proper singles, 888 improper reads (836 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1309554 reads, 1309554 mapped, 0 unmapped, 1309553 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2289 reads, 2289 mapped, 0 unmapped, 2289 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1312751'\n",
       "29. 'Total placed reads: 1312750'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 9767'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1267.800966'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -18755636.338068'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "5. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//11/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//11/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.769628 from 1229856 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.945784'\n",
       "10. 'There were 1278594 total reads, 1278594 paired (1276032 properly mated), 1706 proper singles, 856 improper reads (778 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1276032 reads, 1276032 mapped, 0 unmapped, 1276031 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 778 reads, 778 mapped, 0 unmapped, 778 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1674 reads, 1674 mapped, 0 unmapped, 1674 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1278594'\n",
       "29. 'Total placed reads: 1278593'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 13435'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1076.300842'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -12552137.564932'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "6. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//11/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//11/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.750257 from 1222898 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.940087'\n",
       "10. 'There were 1271913 total reads, 1271913 paired (1268674 properly mated), 1953 proper singles, 1286 improper reads (1220 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1268674 reads, 1268674 mapped, 0 unmapped, 1268674 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 40 reads, 40 mapped, 0 unmapped, 39 placed, 1 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1220 reads, 1220 mapped, 0 unmapped, 1220 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1909 reads, 1909 mapped, 0 unmapped, 1909 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1271913'\n",
       "29. 'Total placed reads: 1271912'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 12144'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1243.700978'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -12235260.934613'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "7. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//12/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//12/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.769835 from 1441024 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.947323'\n",
       "10. 'There were 1498680 total reads, 1498680 paired (1495500 properly mated), 2118 proper singles, 1062 improper reads (1004 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1495500 reads, 1495500 mapped, 0 unmapped, 1495495 placed, 5 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2072 reads, 2072 mapped, 0 unmapped, 2072 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1498680'\n",
       "29. 'Total placed reads: 1498675'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 5'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 13692'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1502.605013'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -17965987.100460'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "8. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//12/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//12/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.754574 from 1434354 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.941442'\n",
       "10. 'There were 1491690 total reads, 1491690 paired (1488456 properly mated), 2248 proper singles, 986 improper reads (958 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1488456 reads, 1488456 mapped, 0 unmapped, 1488453 placed, 3 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 958 reads, 958 mapped, 0 unmapped, 958 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2228 reads, 2228 mapped, 0 unmapped, 2228 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1491690'\n",
       "29. 'Total placed reads: 1491687'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 3'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 11700'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1390.202796'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -17661706.357633'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "9. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//13/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//13/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.765196 from 1040338 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.932119'\n",
       "10. 'There were 1082811 total reads, 1082811 paired (1079514 properly mated), 2755 proper singles, 542 improper reads (504 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1079514 reads, 1079514 mapped, 0 unmapped, 1079514 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 504 reads, 504 mapped, 0 unmapped, 504 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2753 reads, 2753 mapped, 0 unmapped, 2753 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1082811'\n",
       "29. 'Total placed reads: 1082811'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 29053'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1845.300000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -29108120.735174'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "10. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//13/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//13/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.742836 from 1026902 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.920751'\n",
       "10. 'There were 1068969 total reads, 1068969 paired (1065480 properly mated), 2609 proper singles, 880 improper reads (854 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1065480 reads, 1065480 mapped, 0 unmapped, 1065480 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 854 reads, 854 mapped, 0 unmapped, 854 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2609 reads, 2609 mapped, 0 unmapped, 2609 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1068969'\n",
       "29. 'Total placed reads: 1068969'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 17762'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1093.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -28177219.593090'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "11. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//14/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//14/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.779876 from 1026678 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.938274'\n",
       "10. 'There were 1067657 total reads, 1067657 paired (1065468 properly mated), 1933 proper singles, 256 improper reads (246 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1065468 reads, 1065468 mapped, 0 unmapped, 1065468 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1929 reads, 1929 mapped, 0 unmapped, 1929 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1067657'\n",
       "29. 'Total placed reads: 1067657'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 19976'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1050.300000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -23112358.285221'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "12. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//14/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//14/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.775480 from 1017850 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.938990'\n",
       "10. 'There were 1058508 total reads, 1058508 paired (1056270 properly mated), 1756 proper singles, 482 improper reads (448 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1056270 reads, 1056270 mapped, 0 unmapped, 1056270 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1750 reads, 1750 mapped, 0 unmapped, 1750 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1058508'\n",
       "29. 'Total placed reads: 1058508'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 12641'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 721.700000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22517914.298963'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "13. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//15/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//15/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.750549 from 1169866 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.930211'\n",
       "10. 'There were 1216436 total reads, 1216436 paired (1213808 properly mated), 2268 proper singles, 360 improper reads (326 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1213808 reads, 1213808 mapped, 0 unmapped, 1213808 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2260 reads, 2260 mapped, 0 unmapped, 2260 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1216436'\n",
       "29. 'Total placed reads: 1216436'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 23858'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1417.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -18167994.130481'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "14. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//15/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//15/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.726422 from 1160328 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.920425'\n",
       "10. 'There were 1206622 total reads, 1206622 paired (1203812 properly mated), 2148 proper singles, 662 improper reads (608 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1203812 reads, 1203812 mapped, 0 unmapped, 1203811 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2140 reads, 2140 mapped, 0 unmapped, 2140 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1206622'\n",
       "29. 'Total placed reads: 1206621'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14959'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1035.500858'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -17494904.019029'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "15. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//16/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//16/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.674769 from 1312060 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.906746'\n",
       "10. 'There were 1363096 total reads, 1363096 paired (1360784 properly mated), 1860 proper singles, 452 improper reads (402 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1360784 reads, 1360784 mapped, 0 unmapped, 1360781 placed, 3 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1846 reads, 1846 mapped, 0 unmapped, 1846 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1363096'\n",
       "29. 'Total placed reads: 1363093'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 3'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 15590'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1039.902289'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -19575398.337141'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "16. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//16/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//16/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.665849 from 1304080 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.903611'\n",
       "10. 'There were 1355039 total reads, 1355039 paired (1352438 properly mated), 1835 proper singles, 766 improper reads (710 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1352438 reads, 1352438 mapped, 0 unmapped, 1352438 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 710 reads, 710 mapped, 0 unmapped, 710 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1831 reads, 1831 mapped, 0 unmapped, 1831 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1355039'\n",
       "29. 'Total placed reads: 1355039'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 9252'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 966.500000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -19152471.917716'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "17. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//17/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//17/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.714297 from 1056452 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.917958'\n",
       "10. 'There were 1098861 total reads, 1098861 paired (1095938 properly mated), 2365 proper singles, 558 improper reads (536 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1095938 reads, 1095938 mapped, 0 unmapped, 1095938 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 536 reads, 536 mapped, 0 unmapped, 536 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2357 reads, 2357 mapped, 0 unmapped, 2357 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1098861'\n",
       "29. 'Total placed reads: 1098861'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 21615'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1217.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21855528.539573'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "18. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//17/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//17/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.708831 from 1047290 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.914692'\n",
       "10. 'There were 1089261 total reads, 1089261 paired (1086416 properly mated), 2189 proper singles, 656 improper reads (628 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1086416 reads, 1086416 mapped, 0 unmapped, 1086416 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 628 reads, 628 mapped, 0 unmapped, 628 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2185 reads, 2185 mapped, 0 unmapped, 2185 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1089261'\n",
       "29. 'Total placed reads: 1089261'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 15593'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 904.000000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21295303.741731'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "19. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//18/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//18/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.784092 from 1032744 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.942105'\n",
       "10. 'There were 1077664 total reads, 1077664 paired (1071728 properly mated), 5294 proper singles, 642 improper reads (624 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1071728 reads, 1071728 mapped, 0 unmapped, 1071728 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 624 reads, 624 mapped, 0 unmapped, 624 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 5288 reads, 5288 mapped, 0 unmapped, 5288 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1077664'\n",
       "29. 'Total placed reads: 1077664'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 63004'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2727.500000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -30555470.646073'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "20. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//18/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//18/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.757966 from 994686 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.934278'\n",
       "10. 'There were 1039080 total reads, 1039080 paired (1031998 properly mated), 6212 proper singles, 870 improper reads (846 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1031998 reads, 1031998 mapped, 0 unmapped, 1031998 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 846 reads, 846 mapped, 0 unmapped, 846 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 6210 reads, 6210 mapped, 0 unmapped, 6210 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1039080'\n",
       "29. 'Total placed reads: 1039080'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 32109'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2378.400000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -28364110.245962'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "21. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//19/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//19/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.758838 from 1444970 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.947463'\n",
       "10. 'There were 1503635 total reads, 1503635 paired (1499582 properly mated), 2255 proper singles, 1798 improper reads (1720 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1499582 reads, 1499582 mapped, 0 unmapped, 1499581 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1720 reads, 1720 mapped, 0 unmapped, 1720 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2219 reads, 2219 mapped, 0 unmapped, 2219 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1503635'\n",
       "29. 'Total placed reads: 1503634'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 11265'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1860.201237'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -15588448.646139'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "22. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//19/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//19/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.744043 from 1430676 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.941927'\n",
       "10. 'There were 1489966 total reads, 1489966 paired (1484608 properly mated), 3708 proper singles, 1650 improper reads (1564 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1484608 reads, 1484608 mapped, 0 unmapped, 1484608 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1564 reads, 1564 mapped, 0 unmapped, 1564 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3696 reads, 3696 mapped, 0 unmapped, 3696 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1489966'\n",
       "29. 'Total placed reads: 1489966'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 7489'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2466.400000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -15083716.468382'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "23. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//2/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//2/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.707842 from 1162692 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.911909'\n",
       "10. 'There were 1209478 total reads, 1209478 paired (1206160 properly mated), 2218 proper singles, 1100 improper reads (1062 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1206160 reads, 1206160 mapped, 0 unmapped, 1206160 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1062 reads, 1062 mapped, 0 unmapped, 1062 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2210 reads, 2210 mapped, 0 unmapped, 2210 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1209478'\n",
       "29. 'Total placed reads: 1209478'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 16641'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1409.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21502554.168959'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "24. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//2/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//2/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.708035 from 1149766 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.911943'\n",
       "10. 'There were 1196329 total reads, 1196329 paired (1192696 properly mated), 2801 proper singles, 832 improper reads (810 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1192696 reads, 1192696 mapped, 0 unmapped, 1192696 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 810 reads, 810 mapped, 0 unmapped, 810 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2801 reads, 2801 mapped, 0 unmapped, 2801 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1196329'\n",
       "29. 'Total placed reads: 1196329'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 12478'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1527.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -20946198.398444'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "25. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//20/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//20/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.698839 from 929876 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.900362'\n",
       "9. 'There were 966664 total reads, 966664 paired (964368 properly mated), 1996 proper singles, 300 improper reads (284 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 964368 reads, 964368 mapped, 0 unmapped, 964368 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 284 reads, 284 mapped, 0 unmapped, 284 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1984 reads, 1984 mapped, 0 unmapped, 1984 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 966664'\n",
       "27. 'Total placed reads: 966664'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 18701'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 1107.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -18099506.304873'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "26. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//20/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//20/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Found FR sample avg insert length to be 253.691066 from 920578 mapped reads'\n",
       "8. 'Found FR sample insert length std to be 9.895893'\n",
       "9. 'There were 957156 total reads, 957156 paired (954662 properly mated), 1950 proper singles, 544 improper reads (520 chimeric). (0 reads were unmapped)'\n",
       "10. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt.param'\n",
       "11. 'Computing read placements and depths'\n",
       "12. 'FR orientation with 954662 reads, 954662 mapped, 0 unmapped, 954662 placed, 0 mappedButNotPlaced'\n",
       "13. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "14. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "15. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "19. 'CHIMER orientation with 520 reads, 520 mapped, 0 unmapped, 520 placed, 0 mappedButNotPlaced'\n",
       "20. 'SINGLE_UNMAPPED_MATE orientation with 1928 reads, 1928 mapped, 0 unmapped, 1928 placed, 0 mappedButNotPlaced'\n",
       "21. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "22. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "23. 'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "24. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'Total mapped reads: 957156'\n",
       "27. 'Total placed reads: 957156'\n",
       "28. 'Total unmapped reads: 0'\n",
       "29. 'Total unplaced reads: 0'\n",
       "30. 'Computing k-mer statistics...'\n",
       "31. 'Done computing k-mer statistics.'\n",
       "32. 'Computing depth statistics...'\n",
       "33. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "34. 'unmappable bases: 0 across 0 different regions'\n",
       "35. 'bases with 0 coverage: 13037'\n",
       "36. 'Done computing depth statistics.'\n",
       "37. 'Computing statistics on expected missing...'\n",
       "38. 'Expected extra length: 838.500000'\n",
       "39. 'Done computing statistics on expected missing.'\n",
       "40. 'Set minLogLike to: -120.0'\n",
       "41. 'Total ALE Score: -17479146.406151'\n",
       "42. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt'\n",
       "43. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "27. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//21/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//21/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.783618 from 1315482 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.951948'\n",
       "10. 'There were 1367541 total reads, 1367541 paired (1365056 properly mated), 1629 proper singles, 856 improper reads (790 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1365056 reads, 1365056 mapped, 0 unmapped, 1365056 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 790 reads, 790 mapped, 0 unmapped, 790 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1623 reads, 1623 mapped, 0 unmapped, 1623 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1367541'\n",
       "29. 'Total placed reads: 1367541'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 11981'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1166.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22004557.470929'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "28. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//21/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//21/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.774182 from 1307486 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.947431'\n",
       "10. 'There were 1360008 total reads, 1360008 paired (1356718 properly mated), 2242 proper singles, 1048 improper reads (1004 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1356718 reads, 1356718 mapped, 0 unmapped, 1356718 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2234 reads, 2234 mapped, 0 unmapped, 2234 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1360008'\n",
       "29. 'Total placed reads: 1360008'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 8679'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1424.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21546080.209347'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "29. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//22/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//22/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.758738 from 1107196 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.936306'\n",
       "10. 'There were 1153199 total reads, 1153199 paired (1148874 properly mated), 2607 proper singles, 1718 improper reads (1640 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1148874 reads, 1148874 mapped, 0 unmapped, 1148874 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 37 placed, 1 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1640 reads, 1640 mapped, 0 unmapped, 1640 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2595 reads, 2595 mapped, 0 unmapped, 2595 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1153199'\n",
       "29. 'Total placed reads: 1153198'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14542'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2070.801796'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -28105795.208026'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "30. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//22/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//22/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.750389 from 1088718 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.931532'\n",
       "10. 'There were 1133938 total reads, 1133938 paired (1129594 properly mated), 3196 proper singles, 1148 improper reads (1110 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1129594 reads, 1129594 mapped, 0 unmapped, 1129593 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1110 reads, 1110 mapped, 0 unmapped, 1110 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3196 reads, 3196 mapped, 0 unmapped, 3196 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1133938'\n",
       "29. 'Total placed reads: 1133937'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 8482'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2082.501837'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -26800705.802743'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "31. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//23/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//23/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.717675 from 1100808 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.915271'\n",
       "10. 'There were 1144566 total reads, 1144566 paired (1142120 properly mated), 1924 proper singles, 522 improper reads (462 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1142120 reads, 1142120 mapped, 0 unmapped, 1142120 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 462 reads, 462 mapped, 0 unmapped, 462 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1912 reads, 1912 mapped, 0 unmapped, 1912 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1144566'\n",
       "29. 'Total placed reads: 1144566'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14963'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1172.300000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -14831360.378148'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "32. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//23/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//23/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.718760 from 1092852 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.918076'\n",
       "10. 'There were 1136709 total reads, 1136709 paired (1133854 properly mated), 1947 proper singles, 908 improper reads (798 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1133854 reads, 1133854 mapped, 0 unmapped, 1133854 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1941 reads, 1941 mapped, 0 unmapped, 1941 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1136709'\n",
       "29. 'Total placed reads: 1136709'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 11931'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1125.200000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -14386723.450540'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "33. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//24/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//24/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.751733 from 1288410 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.937254'\n",
       "10. 'There were 1338708 total reads, 1338708 paired (1336816 properly mated), 1224 proper singles, 668 improper reads (604 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1336816 reads, 1336816 mapped, 0 unmapped, 1336816 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1338708'\n",
       "29. 'Total placed reads: 1338708'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 8237'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 864.500000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -11592032.231556'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "34. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//24/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//24/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.743318 from 1283372 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.934212'\n",
       "10. 'There were 1333954 total reads, 1333954 paired (1331536 properly mated), 1730 proper singles, 688 improper reads (608 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1331536 reads, 1331536 mapped, 0 unmapped, 1331536 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 76 reads, 76 mapped, 0 unmapped, 76 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1724 reads, 1724 mapped, 0 unmapped, 1724 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1333954'\n",
       "29. 'Total placed reads: 1333954'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 6248'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1017.100000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -11400899.152586'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "35. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//25/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//25/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.766508 from 1452908 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.947885'\n",
       "10. 'There were 1511420 total reads, 1511420 paired (1507552 properly mated), 2286 proper singles, 1582 improper reads (1534 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1507552 reads, 1507552 mapped, 0 unmapped, 1507552 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1534 reads, 1534 mapped, 0 unmapped, 1534 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2266 reads, 2266 mapped, 0 unmapped, 2266 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1511420'\n",
       "29. 'Total placed reads: 1511420'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14303'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1969.861129'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -40241263.112110'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "36. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//25/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//25/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.762489 from 1444882 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.944863'\n",
       "10. 'There were 1503090 total reads, 1503090 paired (1499156 properly mated), 2700 proper singles, 1234 improper reads (1026 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1499156 reads, 1499156 mapped, 0 unmapped, 1499156 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 184 reads, 184 mapped, 0 unmapped, 184 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1026 reads, 1026 mapped, 0 unmapped, 1026 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2690 reads, 2690 mapped, 0 unmapped, 2690 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1503090'\n",
       "29. 'Total placed reads: 1503090'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 9385'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1992.625081'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -39926611.413909'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "37. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//26/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//26/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.761931 from 1237382 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.934695'\n",
       "10. 'There were 1290127 total reads, 1290127 paired (1283976 properly mated), 3499 proper singles, 2652 improper reads (2600 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1283976 reads, 1283976 mapped, 0 unmapped, 1283976 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 23 placed, 1 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 2600 reads, 2600 mapped, 0 unmapped, 2600 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3485 reads, 3485 mapped, 0 unmapped, 3485 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1290127'\n",
       "29. 'Total placed reads: 1290126'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 28963'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2734.102119'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21738734.283024'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "38. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//26/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//26/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.756475 from 1221778 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.932433'\n",
       "10. 'There were 1274647 total reads, 1274647 paired (1267720 properly mated), 4839 proper singles, 2088 improper reads (2022 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1267720 reads, 1267720 mapped, 0 unmapped, 1267720 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 2022 reads, 2022 mapped, 0 unmapped, 2022 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 4827 reads, 4827 mapped, 0 unmapped, 4827 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1274647'\n",
       "29. 'Total placed reads: 1274647'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 19692'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2942.300000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21013802.246661'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "39. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//27/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//27/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.732755 from 1247126 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.932042'\n",
       "10. 'There were 1296952 total reads, 1296952 paired (1294018 properly mated), 2364 proper singles, 570 improper reads (508 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1294018 reads, 1294018 mapped, 0 unmapped, 1294017 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 508 reads, 508 mapped, 0 unmapped, 508 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2346 reads, 2346 mapped, 0 unmapped, 2346 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1296952'\n",
       "29. 'Total placed reads: 1296951'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 23201'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1404.801083'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -19509392.074483'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "40. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//27/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//27/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.722885 from 1237118 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.930657'\n",
       "10. 'There were 1286793 total reads, 1286793 paired (1283584 properly mated), 2411 proper singles, 798 improper reads (722 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1283584 reads, 1283584 mapped, 0 unmapped, 1283583 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 40 placed, 2 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 722 reads, 722 mapped, 0 unmapped, 722 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2391 reads, 2391 mapped, 0 unmapped, 2391 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1286793'\n",
       "29. 'Total placed reads: 1286790'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 3'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14491'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1206.802814'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -18952723.734257'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "41. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//29/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//29/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.788412 from 1228716 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.953392'\n",
       "10. 'There were 1277577 total reads, 1277577 paired (1275142 properly mated), 1705 proper singles, 730 improper reads (654 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1275142 reads, 1275142 mapped, 0 unmapped, 1275142 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 35 placed, 1 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 654 reads, 654 mapped, 0 unmapped, 654 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1689 reads, 1689 mapped, 0 unmapped, 1689 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1277577'\n",
       "29. 'Total placed reads: 1277576'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 13429'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1114.200872'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -16323526.728177'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "42. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//29/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//29/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.786097 from 1220526 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.954888'\n",
       "10. 'There were 1269560 total reads, 1269560 paired (1266562 properly mated), 2030 proper singles, 968 improper reads (870 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1266562 reads, 1266562 mapped, 0 unmapped, 1266561 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 78 reads, 78 mapped, 0 unmapped, 78 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 870 reads, 870 mapped, 0 unmapped, 870 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2018 reads, 2018 mapped, 0 unmapped, 2018 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1269560'\n",
       "29. 'Total placed reads: 1269559'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 9503'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1291.101017'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -15984512.357388'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "43. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//3/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//3/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.738839 from 1059914 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.917124'\n",
       "10. 'There were 1104124 total reads, 1104124 paired (1099770 properly mated), 3128 proper singles, 1226 improper reads (1172 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1099770 reads, 1099770 mapped, 0 unmapped, 1099770 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1172 reads, 1172 mapped, 0 unmapped, 1172 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3074 reads, 3074 mapped, 0 unmapped, 3074 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1104124'\n",
       "29. 'Total placed reads: 1104124'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 26445'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1913.000000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -23674874.444568'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "44. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//3/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//3/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.723814 from 1042270 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.912585'\n",
       "10. 'There were 1086054 total reads, 1086054 paired (1081402 properly mated), 3390 proper singles, 1262 improper reads (1158 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1081402 reads, 1081402 mapped, 0 unmapped, 1081401 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1158 reads, 1158 mapped, 0 unmapped, 1158 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3368 reads, 3368 mapped, 0 unmapped, 3368 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1086054'\n",
       "29. 'Total placed reads: 1086053'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 15980'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1774.101634'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22786161.840885'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "45. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//30/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//30/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.724625 from 983612 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.918317'\n",
       "10. 'There were 1024938 total reads, 1024938 paired (1020408 properly mated), 3874 proper singles, 656 improper reads (600 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1020408 reads, 1020408 mapped, 0 unmapped, 1020407 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 600 reads, 600 mapped, 0 unmapped, 600 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3868 reads, 3868 mapped, 0 unmapped, 3868 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1024938'\n",
       "29. 'Total placed reads: 1024937'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 39217'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 2030.501981'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -23545883.038263'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "46. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//30/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//30/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.710654 from 969954 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.912264'\n",
       "10. 'There were 1011415 total reads, 1011415 paired (1006192 properly mated), 3897 proper singles, 1326 improper reads (1260 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1006192 reads, 1006192 mapped, 0 unmapped, 1006191 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1260 reads, 1260 mapped, 0 unmapped, 1260 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 3897 reads, 3897 mapped, 0 unmapped, 3897 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1011415'\n",
       "29. 'Total placed reads: 1011414'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 21594'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1767.301747'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -22471465.103161'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "47. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//4/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//4/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.738952 from 1099898 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.923500'\n",
       "10. 'There were 1143792 total reads, 1143792 paired (1141146 properly mated), 1752 proper singles, 894 improper reads (860 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1141146 reads, 1141146 mapped, 0 unmapped, 1141146 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1734 reads, 1734 mapped, 0 unmapped, 1734 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1143792'\n",
       "29. 'Total placed reads: 1143792'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 17197'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1344.800000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -14267214.886970'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "48. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//4/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//4/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.735895 from 1090712 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.924751'\n",
       "10. 'There were 1134818 total reads, 1134818 paired (1131630 properly mated), 2128 proper singles, 1060 improper reads (962 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1131630 reads, 1131630 mapped, 0 unmapped, 1131630 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2120 reads, 2120 mapped, 0 unmapped, 2120 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1134818'\n",
       "29. 'Total placed reads: 1134818'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 13371'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1363.000000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -13782761.546427'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "49. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//5/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//5/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.700979 from 1150714 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.915817'\n",
       "10. 'There were 1196310 total reads, 1196310 paired (1193536 properly mated), 2002 proper singles, 772 improper reads (702 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1193536 reads, 1193536 mapped, 0 unmapped, 1193536 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 702 reads, 702 mapped, 0 unmapped, 702 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 1986 reads, 1986 mapped, 0 unmapped, 1986 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1196310'\n",
       "29. 'Total placed reads: 1196310'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 19586'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1275.200000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -24820367.365228'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "50. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//5/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//5/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.704902 from 1140808 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.917229'\n",
       "10. 'There were 1186595 total reads, 1186595 paired (1183274 properly mated), 2393 proper singles, 928 improper reads (798 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1183274 reads, 1183274 mapped, 0 unmapped, 1183273 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 116 reads, 116 mapped, 0 unmapped, 116 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2385 reads, 2385 mapped, 0 unmapped, 2385 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1186595'\n",
       "29. 'Total placed reads: 1186594'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 13384'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1198.801010'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -24439262.268919'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "51. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//6/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//6/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.740631 from 1200058 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.923067'\n",
       "10. 'There were 1248604 total reads, 1248604 paired (1245142 properly mated), 2540 proper singles, 922 improper reads (882 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1245142 reads, 1245142 mapped, 0 unmapped, 1245142 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 882 reads, 882 mapped, 0 unmapped, 882 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2538 reads, 2538 mapped, 0 unmapped, 2538 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1248604'\n",
       "29. 'Total placed reads: 1248604'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 25160'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1599.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21668021.782460'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "52. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//6/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//6/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.738174 from 1188020 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.923106'\n",
       "10. 'There were 1236593 total reads, 1236593 paired (1232608 properly mated), 2827 proper singles, 1158 improper reads (1124 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1232608 reads, 1232608 mapped, 0 unmapped, 1232608 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1124 reads, 1124 mapped, 0 unmapped, 1124 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2825 reads, 2825 mapped, 0 unmapped, 2825 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1236593'\n",
       "29. 'Total placed reads: 1236593'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 17827'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1662.400000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -21045290.276025'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "53. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//7/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//7/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.731806 from 1270148 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.931106'\n",
       "10. 'There were 1321322 total reads, 1321322 paired (1317878 properly mated), 2596 proper singles, 848 improper reads (724 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1317878 reads, 1317878 mapped, 0 unmapped, 1317877 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 60 reads, 60 mapped, 0 unmapped, 60 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 724 reads, 724 mapped, 0 unmapped, 724 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2558 reads, 2558 mapped, 0 unmapped, 2558 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1321322'\n",
       "29. 'Total placed reads: 1321321'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 22187'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1369.501036'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -16725120.051489'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "54. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//7/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//7/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.739212 from 1261184 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.935542'\n",
       "10. 'There were 1312608 total reads, 1312608 paired (1308642 properly mated), 2796 proper singles, 1170 improper reads (1078 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1308642 reads, 1308642 mapped, 0 unmapped, 1308642 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 1078 reads, 1078 mapped, 0 unmapped, 1078 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2762 reads, 2762 mapped, 0 unmapped, 2762 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1312608'\n",
       "29. 'Total placed reads: 1312608'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 16374'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1600.900000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -16404897.603518'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "55. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//8/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//8/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.716050 from 1056608 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.913943'\n",
       "10. 'There were 1099743 total reads, 1099743 paired (1096198 properly mated), 2915 proper singles, 630 improper reads (572 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1096198 reads, 1096198 mapped, 0 unmapped, 1096197 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 572 reads, 572 mapped, 0 unmapped, 572 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2915 reads, 2915 mapped, 0 unmapped, 2915 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1099743'\n",
       "29. 'Total placed reads: 1099742'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 30760'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1541.201401'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -27720412.514541'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "56. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//8/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//8/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.711122 from 1042420 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.913675'\n",
       "10. 'There were 1085136 total reads, 1085136 paired (1081470 properly mated), 2656 proper singles, 1010 improper reads (962 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1081470 reads, 1081470 mapped, 0 unmapped, 1081469 placed, 1 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2646 reads, 2646 mapped, 0 unmapped, 2646 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1085136'\n",
       "29. 'Total placed reads: 1085135'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 1'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 19772'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1166.901075'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -26982645.501738'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "57. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//9/megahit.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//9/megahit/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.733004 from 1150100 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.920223'\n",
       "10. 'There were 1195606 total reads, 1195606 paired (1193070 properly mated), 2166 proper singles, 370 improper reads (354 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1193070 reads, 1193070 mapped, 0 unmapped, 1193070 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2160 reads, 2160 mapped, 0 unmapped, 2160 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1195606'\n",
       "29. 'Total placed reads: 1195606'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 22245'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 1177.000000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -19579897.957545'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "58. 1. 'BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//9/metaspades.bam'\n",
       "2. 'Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//9/metaspades/contigs_filtered.fasta'\n",
       "3. 'ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt'\n",
       "4. 'Reading in assembly...'\n",
       "5. 'Reading in the map and computing statistics...'\n",
       "6. 'Insert length and std not given, will be calculated from input map.'\n",
       "7. 'Read 1000000 reads...'\n",
       "8. 'Found FR sample avg insert length to be 253.713040 from 1139810 mapped reads'\n",
       "9. 'Found FR sample insert length std to be 9.913492'\n",
       "10. 'There were 1185114 total reads, 1185114 paired (1182324 properly mated), 2034 proper singles, 756 improper reads (748 chimeric). (0 reads were unmapped)'\n",
       "11. 'Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt.param'\n",
       "12. 'Computing read placements and depths'\n",
       "13. 'Read 1000000 reads...'\n",
       "14. 'FR orientation with 1182324 reads, 1182324 mapped, 0 unmapped, 1182324 placed, 0 mappedButNotPlaced'\n",
       "15. 'RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "16. 'FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "17. 'NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "18. 'NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced'\n",
       "19. 'NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced'\n",
       "20. 'UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "21. 'CHIMER orientation with 748 reads, 748 mapped, 0 unmapped, 748 placed, 0 mappedButNotPlaced'\n",
       "22. 'SINGLE_UNMAPPED_MATE orientation with 2034 reads, 2034 mapped, 0 unmapped, 2034 placed, 0 mappedButNotPlaced'\n",
       "23. 'UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "24. 'HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "25. 'SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "26. 'UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "27. 'NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced'\n",
       "28. 'Total mapped reads: 1185114'\n",
       "29. 'Total placed reads: 1185114'\n",
       "30. 'Total unmapped reads: 0'\n",
       "31. 'Total unplaced reads: 0'\n",
       "32. 'Computing k-mer statistics...'\n",
       "33. 'Done computing k-mer statistics.'\n",
       "34. 'Computing depth statistics...'\n",
       "35. 'bases with no GC depth correction metric (small contigs): 0'\n",
       "36. 'unmappable bases: 0 across 0 different regions'\n",
       "37. 'bases with 0 coverage: 14551'\n",
       "38. 'Done computing depth statistics.'\n",
       "39. 'Computing statistics on expected missing...'\n",
       "40. 'Expected extra length: 864.600000'\n",
       "41. 'Done computing statistics on expected missing.'\n",
       "42. 'Set minLogLike to: -120.0'\n",
       "43. 'Total ALE Score: -18995038.934677'\n",
       "44. 'Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt'\n",
       "45. 'Closing input file'\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[[1]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.723707 from 1212220 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.925705\"                                                                                                        \n",
       "[10] \"There were 1260475 total reads, 1260475 paired (1257682 properly mated), 2073 proper singles, 720 improper reads (690 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1257682 reads, 1257682 mapped, 0 unmapped, 1257681 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 690 reads, 690 mapped, 0 unmapped, 690 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2043 reads, 2043 mapped, 0 unmapped, 2043 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1260475\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1260474\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 17197\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1203.800955\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -22697096.942980\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[2]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//1/metaspades.bam\"                                                    \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//1/metaspades/contigs_filtered.fasta\"                 \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt\"                                             \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.732010 from 1201202 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.931224\"                                                                                                        \n",
       "[10] \"There were 1249353 total reads, 1249353 paired (1246230 properly mated), 2167 proper singles, 956 improper reads (912 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt.param\"                            \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1246230 reads, 1246230 mapped, 0 unmapped, 1246230 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 912 reads, 912 mapped, 0 unmapped, 912 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2133 reads, 2133 mapped, 0 unmapped, 2133 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1249353\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1249353\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 11647\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1221.800000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -22172524.551273\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt\"                                           \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[3]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.735681 from 1270902 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.930199\"                                                                                                        \n",
       "[10] \"There were 1321605 total reads, 1321605 paired (1318686 properly mated), 1987 proper singles, 932 improper reads (880 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1318686 reads, 1318686 mapped, 0 unmapped, 1318686 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 880 reads, 880 mapped, 0 unmapped, 880 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1973 reads, 1973 mapped, 0 unmapped, 1973 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1321605\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1321605\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 15898\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1278.400000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -19122156.831738\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[4]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//10/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//10/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.730003 from 1262138 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.927099\"                                                                                                        \n",
       "[10] \"There were 1312751 total reads, 1312751 paired (1309554 properly mated), 2309 proper singles, 888 improper reads (836 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1309554 reads, 1309554 mapped, 0 unmapped, 1309553 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 836 reads, 836 mapped, 0 unmapped, 836 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2289 reads, 2289 mapped, 0 unmapped, 2289 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1312751\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1312750\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 9767\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1267.800966\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -18755636.338068\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[5]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//11/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//11/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.769628 from 1229856 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.945784\"                                                                                                        \n",
       "[10] \"There were 1278594 total reads, 1278594 paired (1276032 properly mated), 1706 proper singles, 856 improper reads (778 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1276032 reads, 1276032 mapped, 0 unmapped, 1276031 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 778 reads, 778 mapped, 0 unmapped, 778 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1674 reads, 1674 mapped, 0 unmapped, 1674 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 32 reads, 32 mapped, 0 unmapped, 32 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1278594\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1278593\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 13435\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1076.300842\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -12552137.564932\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[6]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//11/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//11/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.750257 from 1222898 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.940087\"                                                                                                          \n",
       "[10] \"There were 1271913 total reads, 1271913 paired (1268674 properly mated), 1953 proper singles, 1286 improper reads (1220 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1268674 reads, 1268674 mapped, 0 unmapped, 1268674 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 40 reads, 40 mapped, 0 unmapped, 39 placed, 1 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 26 reads, 26 mapped, 0 unmapped, 26 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1220 reads, 1220 mapped, 0 unmapped, 1220 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1909 reads, 1909 mapped, 0 unmapped, 1909 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1271913\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1271912\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 12144\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1243.700978\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -12235260.934613\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[7]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//12/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//12/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.769835 from 1441024 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.947323\"                                                                                                          \n",
       "[10] \"There were 1498680 total reads, 1498680 paired (1495500 properly mated), 2118 proper singles, 1062 improper reads (1004 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1495500 reads, 1495500 mapped, 0 unmapped, 1495495 placed, 5 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2072 reads, 2072 mapped, 0 unmapped, 2072 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1498680\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1498675\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 5\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 13692\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1502.605013\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -17965987.100460\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[8]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//12/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//12/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.754574 from 1434354 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.941442\"                                                                                                        \n",
       "[10] \"There were 1491690 total reads, 1491690 paired (1488456 properly mated), 2248 proper singles, 986 improper reads (958 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1488456 reads, 1488456 mapped, 0 unmapped, 1488453 placed, 3 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 958 reads, 958 mapped, 0 unmapped, 958 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2228 reads, 2228 mapped, 0 unmapped, 2228 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1491690\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1491687\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 3\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 11700\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1390.202796\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -17661706.357633\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//12/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[9]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//13/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//13/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.765196 from 1040338 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.932119\"                                                                                                        \n",
       "[10] \"There were 1082811 total reads, 1082811 paired (1079514 properly mated), 2755 proper singles, 542 improper reads (504 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1079514 reads, 1079514 mapped, 0 unmapped, 1079514 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 504 reads, 504 mapped, 0 unmapped, 504 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2753 reads, 2753 mapped, 0 unmapped, 2753 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1082811\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1082811\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 29053\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1845.300000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -29108120.735174\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[10]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//13/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//13/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.742836 from 1026902 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.920751\"                                                                                                        \n",
       "[10] \"There were 1068969 total reads, 1068969 paired (1065480 properly mated), 2609 proper singles, 880 improper reads (854 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1065480 reads, 1065480 mapped, 0 unmapped, 1065480 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 854 reads, 854 mapped, 0 unmapped, 854 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2609 reads, 2609 mapped, 0 unmapped, 2609 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1068969\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1068969\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 17762\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1093.600000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -28177219.593090\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//13/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[11]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//14/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//14/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.779876 from 1026678 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.938274\"                                                                                                        \n",
       "[10] \"There were 1067657 total reads, 1067657 paired (1065468 properly mated), 1933 proper singles, 256 improper reads (246 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1065468 reads, 1065468 mapped, 0 unmapped, 1065468 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[18] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 246 reads, 246 mapped, 0 unmapped, 246 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1929 reads, 1929 mapped, 0 unmapped, 1929 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1067657\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1067657\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 19976\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1050.300000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -23112358.285221\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[12]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//14/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//14/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.775480 from 1017850 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.938990\"                                                                                                        \n",
       "[10] \"There were 1058508 total reads, 1058508 paired (1056270 properly mated), 1756 proper singles, 482 improper reads (448 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1056270 reads, 1056270 mapped, 0 unmapped, 1056270 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 448 reads, 448 mapped, 0 unmapped, 448 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1750 reads, 1750 mapped, 0 unmapped, 1750 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1058508\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1058508\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 12641\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 721.700000\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -22517914.298963\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//14/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[13]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//15/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//15/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.750549 from 1169866 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.930211\"                                                                                                        \n",
       "[10] \"There were 1216436 total reads, 1216436 paired (1213808 properly mated), 2268 proper singles, 360 improper reads (326 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1213808 reads, 1213808 mapped, 0 unmapped, 1213808 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 326 reads, 326 mapped, 0 unmapped, 326 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2260 reads, 2260 mapped, 0 unmapped, 2260 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1216436\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1216436\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 23858\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1417.600000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -18167994.130481\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[14]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//15/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//15/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.726422 from 1160328 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.920425\"                                                                                                        \n",
       "[10] \"There were 1206622 total reads, 1206622 paired (1203812 properly mated), 2148 proper singles, 662 improper reads (608 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1203812 reads, 1203812 mapped, 0 unmapped, 1203811 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2140 reads, 2140 mapped, 0 unmapped, 2140 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1206622\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1206621\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 14959\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1035.500858\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -17494904.019029\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//15/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[15]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//16/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//16/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.674769 from 1312060 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.906746\"                                                                                                        \n",
       "[10] \"There were 1363096 total reads, 1363096 paired (1360784 properly mated), 1860 proper singles, 452 improper reads (402 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1360784 reads, 1360784 mapped, 0 unmapped, 1360781 placed, 3 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 402 reads, 402 mapped, 0 unmapped, 402 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1846 reads, 1846 mapped, 0 unmapped, 1846 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1363096\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1363093\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 3\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 15590\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1039.902289\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -19575398.337141\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[16]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//16/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//16/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.665849 from 1304080 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.903611\"                                                                                                        \n",
       "[10] \"There were 1355039 total reads, 1355039 paired (1352438 properly mated), 1835 proper singles, 766 improper reads (710 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1352438 reads, 1352438 mapped, 0 unmapped, 1352438 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 710 reads, 710 mapped, 0 unmapped, 710 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1831 reads, 1831 mapped, 0 unmapped, 1831 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1355039\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1355039\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 9252\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 966.500000\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -19152471.917716\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//16/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[17]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//17/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//17/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.714297 from 1056452 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.917958\"                                                                                                        \n",
       "[10] \"There were 1098861 total reads, 1098861 paired (1095938 properly mated), 2365 proper singles, 558 improper reads (536 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1095938 reads, 1095938 mapped, 0 unmapped, 1095938 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 536 reads, 536 mapped, 0 unmapped, 536 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2357 reads, 2357 mapped, 0 unmapped, 2357 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1098861\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1098861\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 21615\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1217.600000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -21855528.539573\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[18]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//17/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//17/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.708831 from 1047290 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.914692\"                                                                                                        \n",
       "[10] \"There were 1089261 total reads, 1089261 paired (1086416 properly mated), 2189 proper singles, 656 improper reads (628 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1086416 reads, 1086416 mapped, 0 unmapped, 1086416 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 628 reads, 628 mapped, 0 unmapped, 628 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2185 reads, 2185 mapped, 0 unmapped, 2185 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1089261\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1089261\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 15593\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 904.000000\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -21295303.741731\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//17/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[19]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//18/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//18/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.784092 from 1032744 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.942105\"                                                                                                        \n",
       "[10] \"There were 1077664 total reads, 1077664 paired (1071728 properly mated), 5294 proper singles, 642 improper reads (624 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1071728 reads, 1071728 mapped, 0 unmapped, 1071728 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[18] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 624 reads, 624 mapped, 0 unmapped, 624 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 5288 reads, 5288 mapped, 0 unmapped, 5288 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1077664\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1077664\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 63004\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 2727.500000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -30555470.646073\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[20]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//18/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//18/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.757966 from 994686 mapped reads\"                                                                             \n",
       " [9] \"Found FR sample insert length std to be 9.934278\"                                                                                                        \n",
       "[10] \"There were 1039080 total reads, 1039080 paired (1031998 properly mated), 6212 proper singles, 870 improper reads (846 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1031998 reads, 1031998 mapped, 0 unmapped, 1031998 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 846 reads, 846 mapped, 0 unmapped, 846 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 6210 reads, 6210 mapped, 0 unmapped, 6210 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1039080\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1039080\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 32109\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 2378.400000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -28364110.245962\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//18/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[21]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//19/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//19/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.758838 from 1444970 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.947463\"                                                                                                          \n",
       "[10] \"There were 1503635 total reads, 1503635 paired (1499582 properly mated), 2255 proper singles, 1798 improper reads (1720 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1499582 reads, 1499582 mapped, 0 unmapped, 1499581 placed, 1 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1720 reads, 1720 mapped, 0 unmapped, 1720 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2219 reads, 2219 mapped, 0 unmapped, 2219 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1503635\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1503634\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 11265\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1860.201237\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -15588448.646139\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[22]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//19/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//19/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.744043 from 1430676 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.941927\"                                                                                                          \n",
       "[10] \"There were 1489966 total reads, 1489966 paired (1484608 properly mated), 3708 proper singles, 1650 improper reads (1564 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1484608 reads, 1484608 mapped, 0 unmapped, 1484608 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1564 reads, 1564 mapped, 0 unmapped, 1564 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3696 reads, 3696 mapped, 0 unmapped, 3696 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1489966\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1489966\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 7489\"                                                                                                                               \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 2466.400000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -15083716.468382\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//19/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[23]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//2/megahit.bam\"                                                         \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//2/megahit/contigs_filtered.fasta\"                      \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt\"                                                  \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.707842 from 1162692 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.911909\"                                                                                                          \n",
       "[10] \"There were 1209478 total reads, 1209478 paired (1206160 properly mated), 2218 proper singles, 1100 improper reads (1062 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt.param\"                                 \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1206160 reads, 1206160 mapped, 0 unmapped, 1206160 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[18] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1062 reads, 1062 mapped, 0 unmapped, 1062 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2210 reads, 2210 mapped, 0 unmapped, 2210 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                                \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1209478\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1209478\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 16641\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1409.600000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -21502554.168959\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/megahit.txt\"                                                \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[24]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//2/metaspades.bam\"                                                    \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//2/metaspades/contigs_filtered.fasta\"                 \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt\"                                             \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.708035 from 1149766 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.911943\"                                                                                                        \n",
       "[10] \"There were 1196329 total reads, 1196329 paired (1192696 properly mated), 2801 proper singles, 832 improper reads (810 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt.param\"                            \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1192696 reads, 1192696 mapped, 0 unmapped, 1192696 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[18] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 810 reads, 810 mapped, 0 unmapped, 810 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2801 reads, 2801 mapped, 0 unmapped, 2801 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1196329\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1196329\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 12478\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1527.600000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -20946198.398444\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//2/metaspades.txt\"                                           \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[25]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//20/megahit.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//20/megahit/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.698839 from 929876 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.900362\"                                                                                                     \n",
       " [9] \"There were 966664 total reads, 966664 paired (964368 properly mated), 1996 proper singles, 300 improper reads (284 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt.param\"                           \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 964368 reads, 964368 mapped, 0 unmapped, 964368 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 284 reads, 284 mapped, 0 unmapped, 284 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1984 reads, 1984 mapped, 0 unmapped, 1984 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 966664\"                                                                                                                           \n",
       "[27] \"Total placed reads: 966664\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 18701\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 1107.500000\"                                                                                                                   \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -18099506.304873\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/megahit.txt\"                                          \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[26]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//20/metaspades.bam\"                                                \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//20/metaspades/contigs_filtered.fasta\"             \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt\"                                         \n",
       " [4] \"Reading in assembly...\"                                                                                                                               \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                       \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                  \n",
       " [7] \"Found FR sample avg insert length to be 253.691066 from 920578 mapped reads\"                                                                          \n",
       " [8] \"Found FR sample insert length std to be 9.895893\"                                                                                                     \n",
       " [9] \"There were 957156 total reads, 957156 paired (954662 properly mated), 1950 proper singles, 544 improper reads (520 chimeric). (0 reads were unmapped)\"\n",
       "[10] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt.param\"                        \n",
       "[11] \"Computing read placements and depths\"                                                                                                                 \n",
       "[12] \"FR orientation with 954662 reads, 954662 mapped, 0 unmapped, 954662 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[13] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[14] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                    \n",
       "[15] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[16] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[17] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"CHIMER orientation with 520 reads, 520 mapped, 0 unmapped, 520 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[20] \"SINGLE_UNMAPPED_MATE orientation with 1928 reads, 1928 mapped, 0 unmapped, 1928 placed, 0 mappedButNotPlaced\"                                         \n",
       "[21] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[22] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[23] \"SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[24] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[25] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"Total mapped reads: 957156\"                                                                                                                           \n",
       "[27] \"Total placed reads: 957156\"                                                                                                                           \n",
       "[28] \"Total unmapped reads: 0\"                                                                                                                              \n",
       "[29] \"Total unplaced reads: 0\"                                                                                                                              \n",
       "[30] \"Computing k-mer statistics...\"                                                                                                                        \n",
       "[31] \"Done computing k-mer statistics.\"                                                                                                                     \n",
       "[32] \"Computing depth statistics...\"                                                                                                                        \n",
       "[33] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                          \n",
       "[34] \"unmappable bases: 0 across 0 different regions\"                                                                                                       \n",
       "[35] \"bases with 0 coverage: 13037\"                                                                                                                         \n",
       "[36] \"Done computing depth statistics.\"                                                                                                                     \n",
       "[37] \"Computing statistics on expected missing...\"                                                                                                          \n",
       "[38] \"Expected extra length: 838.500000\"                                                                                                                    \n",
       "[39] \"Done computing statistics on expected missing.\"                                                                                                       \n",
       "[40] \"Set minLogLike to: -120.0\"                                                                                                                            \n",
       "[41] \"Total ALE Score: -17479146.406151\"                                                                                                                    \n",
       "[42] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//20/metaspades.txt\"                                       \n",
       "[43] \"Closing input file\"                                                                                                                                   \n",
       "\n",
       "[[27]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//21/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//21/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.783618 from 1315482 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.951948\"                                                                                                        \n",
       "[10] \"There were 1367541 total reads, 1367541 paired (1365056 properly mated), 1629 proper singles, 856 improper reads (790 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1365056 reads, 1365056 mapped, 0 unmapped, 1365056 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 790 reads, 790 mapped, 0 unmapped, 790 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1623 reads, 1623 mapped, 0 unmapped, 1623 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1367541\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1367541\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 11981\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1166.600000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -22004557.470929\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[28]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//21/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//21/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.774182 from 1307486 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.947431\"                                                                                                          \n",
       "[10] \"There were 1360008 total reads, 1360008 paired (1356718 properly mated), 2242 proper singles, 1048 improper reads (1004 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1356718 reads, 1356718 mapped, 0 unmapped, 1356718 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1004 reads, 1004 mapped, 0 unmapped, 1004 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2234 reads, 2234 mapped, 0 unmapped, 2234 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                                \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1360008\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1360008\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 8679\"                                                                                                                               \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1424.600000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -21546080.209347\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//21/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[29]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//22/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//22/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.758738 from 1107196 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.936306\"                                                                                                          \n",
       "[10] \"There were 1153199 total reads, 1153199 paired (1148874 properly mated), 2607 proper singles, 1718 improper reads (1640 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1148874 reads, 1148874 mapped, 0 unmapped, 1148874 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 38 reads, 38 mapped, 0 unmapped, 37 placed, 1 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1640 reads, 1640 mapped, 0 unmapped, 1640 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2595 reads, 2595 mapped, 0 unmapped, 2595 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1153199\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1153198\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 14542\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 2070.801796\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -28105795.208026\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[30]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//22/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//22/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.750389 from 1088718 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.931532\"                                                                                                          \n",
       "[10] \"There were 1133938 total reads, 1133938 paired (1129594 properly mated), 3196 proper singles, 1148 improper reads (1110 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1129594 reads, 1129594 mapped, 0 unmapped, 1129593 placed, 1 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[18] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1110 reads, 1110 mapped, 0 unmapped, 1110 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3196 reads, 3196 mapped, 0 unmapped, 3196 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1133938\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1133937\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 8482\"                                                                                                                               \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 2082.501837\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -26800705.802743\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//22/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[31]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//23/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//23/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.717675 from 1100808 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.915271\"                                                                                                        \n",
       "[10] \"There were 1144566 total reads, 1144566 paired (1142120 properly mated), 1924 proper singles, 522 improper reads (462 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1142120 reads, 1142120 mapped, 0 unmapped, 1142120 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 462 reads, 462 mapped, 0 unmapped, 462 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1912 reads, 1912 mapped, 0 unmapped, 1912 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1144566\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1144566\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 14963\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1172.300000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -14831360.378148\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[32]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//23/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//23/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.718760 from 1092852 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.918076\"                                                                                                        \n",
       "[10] \"There were 1136709 total reads, 1136709 paired (1133854 properly mated), 1947 proper singles, 908 improper reads (798 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1133854 reads, 1133854 mapped, 0 unmapped, 1133854 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1941 reads, 1941 mapped, 0 unmapped, 1941 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1136709\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1136709\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 11931\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1125.200000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -14386723.450540\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//23/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[33]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//24/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//24/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.751733 from 1288410 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.937254\"                                                                                                        \n",
       "[10] \"There were 1338708 total reads, 1338708 paired (1336816 properly mated), 1224 proper singles, 668 improper reads (604 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1336816 reads, 1336816 mapped, 0 unmapped, 1336816 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 52 reads, 52 mapped, 0 unmapped, 52 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 604 reads, 604 mapped, 0 unmapped, 604 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1204 reads, 1204 mapped, 0 unmapped, 1204 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1338708\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1338708\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 8237\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 864.500000\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -11592032.231556\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[34]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//24/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//24/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.743318 from 1283372 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.934212\"                                                                                                        \n",
       "[10] \"There were 1333954 total reads, 1333954 paired (1331536 properly mated), 1730 proper singles, 688 improper reads (608 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1331536 reads, 1331536 mapped, 0 unmapped, 1331536 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[18] \"NOT_PROPER_RF orientation with 76 reads, 76 mapped, 0 unmapped, 76 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 608 reads, 608 mapped, 0 unmapped, 608 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1724 reads, 1724 mapped, 0 unmapped, 1724 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1333954\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1333954\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 6248\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1017.100000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -11400899.152586\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//24/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[35]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//25/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//25/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.766508 from 1452908 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.947885\"                                                                                                          \n",
       "[10] \"There were 1511420 total reads, 1511420 paired (1507552 properly mated), 2286 proper singles, 1582 improper reads (1534 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1507552 reads, 1507552 mapped, 0 unmapped, 1507552 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1534 reads, 1534 mapped, 0 unmapped, 1534 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2266 reads, 2266 mapped, 0 unmapped, 2266 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1511420\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1511420\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 14303\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1969.861129\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -40241263.112110\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[36]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//25/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//25/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.762489 from 1444882 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.944863\"                                                                                                          \n",
       "[10] \"There were 1503090 total reads, 1503090 paired (1499156 properly mated), 2700 proper singles, 1234 improper reads (1026 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1499156 reads, 1499156 mapped, 0 unmapped, 1499156 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 24 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 184 reads, 184 mapped, 0 unmapped, 184 placed, 0 mappedButNotPlaced\"                                                        \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1026 reads, 1026 mapped, 0 unmapped, 1026 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2690 reads, 2690 mapped, 0 unmapped, 2690 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1503090\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1503090\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 9385\"                                                                                                                               \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1992.625081\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -39926611.413909\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//25/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[37]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//26/megahit.bam\"                                                        \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//26/megahit/contigs_filtered.fasta\"                     \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt\"                                                 \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.761931 from 1237382 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.934695\"                                                                                                          \n",
       "[10] \"There were 1290127 total reads, 1290127 paired (1283976 properly mated), 3499 proper singles, 2652 improper reads (2600 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt.param\"                                \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1283976 reads, 1283976 mapped, 0 unmapped, 1283976 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 24 reads, 24 mapped, 0 unmapped, 23 placed, 1 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 2600 reads, 2600 mapped, 0 unmapped, 2600 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3485 reads, 3485 mapped, 0 unmapped, 3485 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1290127\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1290126\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 28963\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 2734.102119\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -21738734.283024\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/megahit.txt\"                                               \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[38]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//26/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//26/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.756475 from 1221778 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.932433\"                                                                                                          \n",
       "[10] \"There were 1274647 total reads, 1274647 paired (1267720 properly mated), 4839 proper singles, 2088 improper reads (2022 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1267720 reads, 1267720 mapped, 0 unmapped, 1267720 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 2022 reads, 2022 mapped, 0 unmapped, 2022 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 4827 reads, 4827 mapped, 0 unmapped, 4827 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1274647\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1274647\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 19692\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 2942.300000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -21013802.246661\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//26/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[39]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//27/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//27/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.732755 from 1247126 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.932042\"                                                                                                        \n",
       "[10] \"There were 1296952 total reads, 1296952 paired (1294018 properly mated), 2364 proper singles, 570 improper reads (508 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1294018 reads, 1294018 mapped, 0 unmapped, 1294017 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 508 reads, 508 mapped, 0 unmapped, 508 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2346 reads, 2346 mapped, 0 unmapped, 2346 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1296952\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1296951\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 23201\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1404.801083\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -19509392.074483\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[40]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//27/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//27/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.722885 from 1237118 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.930657\"                                                                                                        \n",
       "[10] \"There were 1286793 total reads, 1286793 paired (1283584 properly mated), 2411 proper singles, 798 improper reads (722 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1283584 reads, 1283584 mapped, 0 unmapped, 1283583 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 40 placed, 2 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 722 reads, 722 mapped, 0 unmapped, 722 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2391 reads, 2391 mapped, 0 unmapped, 2391 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1286793\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1286790\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 3\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 14491\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1206.802814\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -18952723.734257\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//27/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[41]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//29/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//29/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.788412 from 1228716 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.953392\"                                                                                                        \n",
       "[10] \"There were 1277577 total reads, 1277577 paired (1275142 properly mated), 1705 proper singles, 730 improper reads (654 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1275142 reads, 1275142 mapped, 0 unmapped, 1275142 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 35 placed, 1 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 654 reads, 654 mapped, 0 unmapped, 654 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1689 reads, 1689 mapped, 0 unmapped, 1689 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1277577\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1277576\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 13429\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1114.200872\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -16323526.728177\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[42]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//29/metaspades.bam\"                                                   \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//29/metaspades/contigs_filtered.fasta\"                \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt\"                                            \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.786097 from 1220526 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.954888\"                                                                                                        \n",
       "[10] \"There were 1269560 total reads, 1269560 paired (1266562 properly mated), 2030 proper singles, 968 improper reads (870 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt.param\"                           \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1266562 reads, 1266562 mapped, 0 unmapped, 1266561 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 78 reads, 78 mapped, 0 unmapped, 78 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 870 reads, 870 mapped, 0 unmapped, 870 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2018 reads, 2018 mapped, 0 unmapped, 2018 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1269560\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1269559\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 9503\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1291.101017\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -15984512.357388\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//29/metaspades.txt\"                                          \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[43]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//3/megahit.bam\"                                                         \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//3/megahit/contigs_filtered.fasta\"                      \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt\"                                                  \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.738839 from 1059914 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.917124\"                                                                                                          \n",
       "[10] \"There were 1104124 total reads, 1104124 paired (1099770 properly mated), 3128 proper singles, 1226 improper reads (1172 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt.param\"                                 \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1099770 reads, 1099770 mapped, 0 unmapped, 1099770 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1172 reads, 1172 mapped, 0 unmapped, 1172 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3074 reads, 3074 mapped, 0 unmapped, 3074 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 54 reads, 54 mapped, 0 unmapped, 54 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1104124\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1104124\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 26445\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1913.000000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -23674874.444568\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/megahit.txt\"                                                \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[44]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//3/metaspades.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//3/metaspades/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.723814 from 1042270 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.912585\"                                                                                                          \n",
       "[10] \"There were 1086054 total reads, 1086054 paired (1081402 properly mated), 3390 proper singles, 1262 improper reads (1158 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1081402 reads, 1081402 mapped, 0 unmapped, 1081401 placed, 1 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 62 reads, 62 mapped, 0 unmapped, 62 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1158 reads, 1158 mapped, 0 unmapped, 1158 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3368 reads, 3368 mapped, 0 unmapped, 3368 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1086054\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1086053\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 15980\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1774.101634\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -22786161.840885\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//3/metaspades.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[45]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//30/megahit.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//30/megahit/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.724625 from 983612 mapped reads\"                                                                             \n",
       " [9] \"Found FR sample insert length std to be 9.918317\"                                                                                                        \n",
       "[10] \"There were 1024938 total reads, 1024938 paired (1020408 properly mated), 3874 proper singles, 656 improper reads (600 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1020408 reads, 1020408 mapped, 0 unmapped, 1020407 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 42 reads, 42 mapped, 0 unmapped, 42 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 600 reads, 600 mapped, 0 unmapped, 600 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3868 reads, 3868 mapped, 0 unmapped, 3868 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1024938\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1024937\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 39217\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 2030.501981\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -23545883.038263\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/megahit.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[46]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//30/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//30/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.710654 from 969954 mapped reads\"                                                                               \n",
       " [9] \"Found FR sample insert length std to be 9.912264\"                                                                                                          \n",
       "[10] \"There were 1011415 total reads, 1011415 paired (1006192 properly mated), 3897 proper singles, 1326 improper reads (1260 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1006192 reads, 1006192 mapped, 0 unmapped, 1006191 placed, 1 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1260 reads, 1260 mapped, 0 unmapped, 1260 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 3897 reads, 3897 mapped, 0 unmapped, 3897 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1011415\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1011414\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 21594\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1767.301747\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -22471465.103161\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//30/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[47]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//4/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//4/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.738952 from 1099898 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.923500\"                                                                                                        \n",
       "[10] \"There were 1143792 total reads, 1143792 paired (1141146 properly mated), 1752 proper singles, 894 improper reads (860 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1141146 reads, 1141146 mapped, 0 unmapped, 1141146 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 12 reads, 12 mapped, 0 unmapped, 12 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 860 reads, 860 mapped, 0 unmapped, 860 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1734 reads, 1734 mapped, 0 unmapped, 1734 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1143792\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1143792\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 17197\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1344.800000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -14267214.886970\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[48]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//4/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//4/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                   \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                           \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                      \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                    \n",
       " [8] \"Found FR sample avg insert length to be 253.735895 from 1090712 mapped reads\"                                                                             \n",
       " [9] \"Found FR sample insert length std to be 9.924751\"                                                                                                         \n",
       "[10] \"There were 1134818 total reads, 1134818 paired (1131630 properly mated), 2128 proper singles, 1060 improper reads (962 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                     \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                    \n",
       "[14] \"FR orientation with 1131630 reads, 1131630 mapped, 0 unmapped, 1131630 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                        \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                        \n",
       "[17] \"NOT_PROPER_FR orientation with 40 reads, 40 mapped, 0 unmapped, 40 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[18] \"NOT_PROPER_RF orientation with 58 reads, 58 mapped, 0 unmapped, 58 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[21] \"CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2120 reads, 2120 mapped, 0 unmapped, 2120 placed, 0 mappedButNotPlaced\"                                             \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                               \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[28] \"Total mapped reads: 1134818\"                                                                                                                              \n",
       "[29] \"Total placed reads: 1134818\"                                                                                                                              \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                  \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                  \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                            \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                         \n",
       "[34] \"Computing depth statistics...\"                                                                                                                            \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                              \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                           \n",
       "[37] \"bases with 0 coverage: 13371\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                         \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                              \n",
       "[40] \"Expected extra length: 1363.000000\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                           \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                \n",
       "[43] \"Total ALE Score: -13782761.546427\"                                                                                                                        \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//4/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                       \n",
       "\n",
       "[[49]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//5/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//5/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.700979 from 1150714 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.915817\"                                                                                                        \n",
       "[10] \"There were 1196310 total reads, 1196310 paired (1193536 properly mated), 2002 proper singles, 772 improper reads (702 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1193536 reads, 1193536 mapped, 0 unmapped, 1193536 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 36 reads, 36 mapped, 0 unmapped, 36 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 28 reads, 28 mapped, 0 unmapped, 28 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 702 reads, 702 mapped, 0 unmapped, 702 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 1986 reads, 1986 mapped, 0 unmapped, 1986 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 16 reads, 16 mapped, 0 unmapped, 16 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1196310\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1196310\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 19586\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1275.200000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -24820367.365228\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[50]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//5/metaspades.bam\"                                                    \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//5/metaspades/contigs_filtered.fasta\"                 \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt\"                                             \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.704902 from 1140808 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.917229\"                                                                                                        \n",
       "[10] \"There were 1186595 total reads, 1186595 paired (1183274 properly mated), 2393 proper singles, 928 improper reads (798 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt.param\"                            \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1183274 reads, 1183274 mapped, 0 unmapped, 1183273 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 116 reads, 116 mapped, 0 unmapped, 116 placed, 0 mappedButNotPlaced\"                                                      \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 798 reads, 798 mapped, 0 unmapped, 798 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2385 reads, 2385 mapped, 0 unmapped, 2385 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 8 reads, 8 mapped, 0 unmapped, 8 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1186595\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1186594\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 13384\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1198.801010\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -24439262.268919\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//5/metaspades.txt\"                                           \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[51]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//6/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//6/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.740631 from 1200058 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.923067\"                                                                                                        \n",
       "[10] \"There were 1248604 total reads, 1248604 paired (1245142 properly mated), 2540 proper singles, 922 improper reads (882 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1245142 reads, 1245142 mapped, 0 unmapped, 1245142 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 882 reads, 882 mapped, 0 unmapped, 882 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2538 reads, 2538 mapped, 0 unmapped, 2538 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1248604\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1248604\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 25160\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1599.600000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -21668021.782460\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[52]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//6/metaspades.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//6/metaspades/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.738174 from 1188020 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.923106\"                                                                                                          \n",
       "[10] \"There were 1236593 total reads, 1236593 paired (1232608 properly mated), 2827 proper singles, 1158 improper reads (1124 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1232608 reads, 1232608 mapped, 0 unmapped, 1232608 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 20 reads, 20 mapped, 0 unmapped, 20 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1124 reads, 1124 mapped, 0 unmapped, 1124 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2825 reads, 2825 mapped, 0 unmapped, 2825 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                                \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1236593\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1236593\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 17827\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1662.400000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -21045290.276025\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//6/metaspades.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[53]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//7/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//7/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.731806 from 1270148 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.931106\"                                                                                                        \n",
       "[10] \"There were 1321322 total reads, 1321322 paired (1317878 properly mated), 2596 proper singles, 848 improper reads (724 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1317878 reads, 1317878 mapped, 0 unmapped, 1317877 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 60 reads, 60 mapped, 0 unmapped, 60 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 50 reads, 50 mapped, 0 unmapped, 50 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 14 reads, 14 mapped, 0 unmapped, 14 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 724 reads, 724 mapped, 0 unmapped, 724 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2558 reads, 2558 mapped, 0 unmapped, 2558 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 38 reads, 38 mapped, 0 unmapped, 38 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1321322\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1321321\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 22187\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1369.501036\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -16725120.051489\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[54]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//7/metaspades.bam\"                                                      \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//7/metaspades/contigs_filtered.fasta\"                   \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt\"                                               \n",
       " [4] \"Reading in assembly...\"                                                                                                                                    \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                            \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                       \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                     \n",
       " [8] \"Found FR sample avg insert length to be 253.739212 from 1261184 mapped reads\"                                                                              \n",
       " [9] \"Found FR sample insert length std to be 9.935542\"                                                                                                          \n",
       "[10] \"There were 1312608 total reads, 1312608 paired (1308642 properly mated), 2796 proper singles, 1170 improper reads (1078 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt.param\"                              \n",
       "[12] \"Computing read placements and depths\"                                                                                                                      \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                     \n",
       "[14] \"FR orientation with 1308642 reads, 1308642 mapped, 0 unmapped, 1308642 placed, 0 mappedButNotPlaced\"                                                       \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                         \n",
       "[17] \"NOT_PROPER_FR orientation with 46 reads, 46 mapped, 0 unmapped, 46 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[18] \"NOT_PROPER_RF orientation with 44 reads, 44 mapped, 0 unmapped, 44 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[21] \"CHIMER orientation with 1078 reads, 1078 mapped, 0 unmapped, 1078 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2762 reads, 2762 mapped, 0 unmapped, 2762 placed, 0 mappedButNotPlaced\"                                              \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[25] \"SINGLE_READ orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                   \n",
       "[28] \"Total mapped reads: 1312608\"                                                                                                                               \n",
       "[29] \"Total placed reads: 1312608\"                                                                                                                               \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                   \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                   \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                             \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                          \n",
       "[34] \"Computing depth statistics...\"                                                                                                                             \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                               \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                            \n",
       "[37] \"bases with 0 coverage: 16374\"                                                                                                                              \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                          \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                               \n",
       "[40] \"Expected extra length: 1600.900000\"                                                                                                                        \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                            \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                 \n",
       "[43] \"Total ALE Score: -16404897.603518\"                                                                                                                         \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//7/metaspades.txt\"                                             \n",
       "[45] \"Closing input file\"                                                                                                                                        \n",
       "\n",
       "[[55]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//8/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//8/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.716050 from 1056608 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.913943\"                                                                                                        \n",
       "[10] \"There were 1099743 total reads, 1099743 paired (1096198 properly mated), 2915 proper singles, 630 improper reads (572 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1096198 reads, 1096198 mapped, 0 unmapped, 1096197 placed, 1 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 34 reads, 34 mapped, 0 unmapped, 34 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[18] \"NOT_PROPER_RF orientation with 22 reads, 22 mapped, 0 unmapped, 22 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 572 reads, 572 mapped, 0 unmapped, 572 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2915 reads, 2915 mapped, 0 unmapped, 2915 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1099743\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1099742\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 30760\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1541.201401\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -27720412.514541\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[56]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//8/metaspades.bam\"                                                     \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//8/metaspades/contigs_filtered.fasta\"                  \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt\"                                              \n",
       " [4] \"Reading in assembly...\"                                                                                                                                   \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                           \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                      \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                    \n",
       " [8] \"Found FR sample avg insert length to be 253.711122 from 1042420 mapped reads\"                                                                             \n",
       " [9] \"Found FR sample insert length std to be 9.913675\"                                                                                                         \n",
       "[10] \"There were 1085136 total reads, 1085136 paired (1081470 properly mated), 2656 proper singles, 1010 improper reads (962 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt.param\"                             \n",
       "[12] \"Computing read placements and depths\"                                                                                                                     \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                    \n",
       "[14] \"FR orientation with 1081470 reads, 1081470 mapped, 0 unmapped, 1081469 placed, 1 mappedButNotPlaced\"                                                      \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                        \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                        \n",
       "[17] \"NOT_PROPER_FR orientation with 18 reads, 18 mapped, 0 unmapped, 18 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[18] \"NOT_PROPER_RF orientation with 30 reads, 30 mapped, 0 unmapped, 30 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[19] \"NOT_PROPER_FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[21] \"CHIMER orientation with 962 reads, 962 mapped, 0 unmapped, 962 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2646 reads, 2646 mapped, 0 unmapped, 2646 placed, 0 mappedButNotPlaced\"                                             \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[25] \"SINGLE_READ orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                           \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                  \n",
       "[28] \"Total mapped reads: 1085136\"                                                                                                                              \n",
       "[29] \"Total placed reads: 1085135\"                                                                                                                              \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                  \n",
       "[31] \"Total unplaced reads: 1\"                                                                                                                                  \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                            \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                         \n",
       "[34] \"Computing depth statistics...\"                                                                                                                            \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                              \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                           \n",
       "[37] \"bases with 0 coverage: 19772\"                                                                                                                             \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                         \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                              \n",
       "[40] \"Expected extra length: 1166.901075\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                           \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                                \n",
       "[43] \"Total ALE Score: -26982645.501738\"                                                                                                                        \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//8/metaspades.txt\"                                            \n",
       "[45] \"Closing input file\"                                                                                                                                       \n",
       "\n",
       "[[57]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//9/megahit.bam\"                                                       \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//9/megahit/contigs_filtered.fasta\"                    \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt\"                                                \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.733004 from 1150100 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.920223\"                                                                                                        \n",
       "[10] \"There were 1195606 total reads, 1195606 paired (1193070 properly mated), 2166 proper singles, 370 improper reads (354 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt.param\"                               \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1193070 reads, 1193070 mapped, 0 unmapped, 1193070 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[18] \"NOT_PROPER_RF orientation with 10 reads, 10 mapped, 0 unmapped, 10 placed, 0 mappedButNotPlaced\"                                                         \n",
       "[19] \"NOT_PROPER_FF orientation with 4 reads, 4 mapped, 0 unmapped, 4 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 354 reads, 354 mapped, 0 unmapped, 354 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2160 reads, 2160 mapped, 0 unmapped, 2160 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1195606\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1195606\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 22245\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 1177.000000\"                                                                                                                      \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -19579897.957545\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/megahit.txt\"                                              \n",
       "[45] \"Closing input file\"                                                                                                                                      \n",
       "\n",
       "[[58]]\n",
       " [1] \"BAM file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/map//9/metaspades.bam\"                                                    \n",
       " [2] \"Assembly fasta file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/assembly//9/metaspades/contigs_filtered.fasta\"                 \n",
       " [3] \"ALE Output file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt\"                                             \n",
       " [4] \"Reading in assembly...\"                                                                                                                                  \n",
       " [5] \"Reading in the map and computing statistics...\"                                                                                                          \n",
       " [6] \"Insert length and std not given, will be calculated from input map.\"                                                                                     \n",
       " [7] \"Read 1000000 reads...\"                                                                                                                                   \n",
       " [8] \"Found FR sample avg insert length to be 253.713040 from 1139810 mapped reads\"                                                                            \n",
       " [9] \"Found FR sample insert length std to be 9.913492\"                                                                                                        \n",
       "[10] \"There were 1185114 total reads, 1185114 paired (1182324 properly mated), 2034 proper singles, 756 improper reads (748 chimeric). (0 reads were unmapped)\"\n",
       "[11] \"Saved library parameters to /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt.param\"                            \n",
       "[12] \"Computing read placements and depths\"                                                                                                                    \n",
       "[13] \"Read 1000000 reads...\"                                                                                                                                   \n",
       "[14] \"FR orientation with 1182324 reads, 1182324 mapped, 0 unmapped, 1182324 placed, 0 mappedButNotPlaced\"                                                     \n",
       "[15] \"RF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[16] \"FF orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                       \n",
       "[17] \"NOT_PROPER_FR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[18] \"NOT_PROPER_RF orientation with 6 reads, 6 mapped, 0 unmapped, 6 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[19] \"NOT_PROPER_FF orientation with 2 reads, 2 mapped, 0 unmapped, 2 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[20] \"UNMAPPED_PAIR orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[21] \"CHIMER orientation with 748 reads, 748 mapped, 0 unmapped, 748 placed, 0 mappedButNotPlaced\"                                                             \n",
       "[22] \"SINGLE_UNMAPPED_MATE orientation with 2034 reads, 2034 mapped, 0 unmapped, 2034 placed, 0 mappedButNotPlaced\"                                            \n",
       "[23] \"UNMAPPED_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                            \n",
       "[24] \"HALF_VALID_MATE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[25] \"SINGLE_READ orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                              \n",
       "[26] \"UNMAPPED_SINGLE orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                          \n",
       "[27] \"NO_READS orientation with 0 reads, 0 mapped, 0 unmapped, 0 placed, 0 mappedButNotPlaced\"                                                                 \n",
       "[28] \"Total mapped reads: 1185114\"                                                                                                                             \n",
       "[29] \"Total placed reads: 1185114\"                                                                                                                             \n",
       "[30] \"Total unmapped reads: 0\"                                                                                                                                 \n",
       "[31] \"Total unplaced reads: 0\"                                                                                                                                 \n",
       "[32] \"Computing k-mer statistics...\"                                                                                                                           \n",
       "[33] \"Done computing k-mer statistics.\"                                                                                                                        \n",
       "[34] \"Computing depth statistics...\"                                                                                                                           \n",
       "[35] \"bases with no GC depth correction metric (small contigs): 0\"                                                                                             \n",
       "[36] \"unmappable bases: 0 across 0 different regions\"                                                                                                          \n",
       "[37] \"bases with 0 coverage: 14551\"                                                                                                                            \n",
       "[38] \"Done computing depth statistics.\"                                                                                                                        \n",
       "[39] \"Computing statistics on expected missing...\"                                                                                                             \n",
       "[40] \"Expected extra length: 864.600000\"                                                                                                                       \n",
       "[41] \"Done computing statistics on expected missing.\"                                                                                                          \n",
       "[42] \"Set minLogLike to: -120.0\"                                                                                                                               \n",
       "[43] \"Total ALE Score: -18995038.934677\"                                                                                                                       \n",
       "[44] \"Output is in file: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//9/metaspades.txt\"                                           \n",
       "[45] \"Closing input file\"                                                                                                                                      \n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# apply function on cluster\n",
    "job_ret = future_lapply(ALE_cmds, FUN = function(x) bash_job(x, conda_env=conda_env))\n",
    "job_ret"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Summarizing results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:11:23.908214Z",
     "start_time": "2019-07-23T09:03:18.515Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 58\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt'</li>\n",
       "\t<li>'/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//1/metaspades.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//10/metaspades.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/megahit.txt'\n",
       "\\item '/ebio/abt3\\_projects/databases\\_no-backup/DeepMAsED/test\\_runs/n100\\_r25/ALE//11/metaspades.txt'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt'\n",
       "2. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt'\n",
       "3. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt'\n",
       "4. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt'\n",
       "5. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt'\n",
       "6. '/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/megahit.txt\"    \n",
       "[2] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//1/metaspades.txt\" \n",
       "[3] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/megahit.txt\"   \n",
       "[4] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//10/metaspades.txt\"\n",
       "[5] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/megahit.txt\"   \n",
       "[6] \"/ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//11/metaspades.txt\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "ALE_res_files = list.files(work_dir, '*.txt$', recursive=TRUE, full.names=TRUE)\n",
    "ALE_res_files %>% length %>% print\n",
    "ALE_res_files %>% head"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:11:23.939023Z",
     "start_time": "2019-07-23T09:03:18.899Z"
    }
   },
   "outputs": [],
   "source": [
    "read_ALE_files = function(in_file){\n",
    "    # input command\n",
    "    cmd = 'egrep -v \"^#\" {in_file}'\n",
    "    cmd = glue::glue(cmd, in_file=in_file)\n",
    "    # reading in table\n",
    "    df = fread(cmd, sep=' ', header=FALSE)\n",
    "    # formatting\n",
    "    colnames(df) = c('contig', 'position', 'depth', \n",
    "                     'ln__depthLike', 'ln__placeLike', \n",
    "                     'ln__insertLike', 'ln__kmerLike')\n",
    "    df$Assembler = in_file %>% basename %>% gsub('\\\\.txt$', '', .)\n",
    "    df$Rep = in_file %>% dirname %>% basename\n",
    "    return(df)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T09:33:26.669564Z",
     "start_time": "2019-07-23T09:03:20.170Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 1641203714\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<thead><tr><th scope=col>contig</th><th scope=col>position</th><th scope=col>depth</th><th scope=col>ln__depthLike</th><th scope=col>ln__placeLike</th><th scope=col>ln__insertLike</th><th scope=col>ln__kmerLike</th><th scope=col>Assembler</th><th scope=col>Rep</th></tr></thead>\n",
       "<tbody>\n",
       "\t<tr><td>0      </td><td>0      </td><td>1      </td><td>-2.601 </td><td>-0.003 </td><td>0.044  </td><td>0.001  </td><td>megahit</td><td>1      </td></tr>\n",
       "\t<tr><td>0      </td><td>1      </td><td>1      </td><td>-2.601 </td><td>-0.003 </td><td>0.044  </td><td>0.076  </td><td>megahit</td><td>1      </td></tr>\n",
       "\t<tr><td>0      </td><td>2      </td><td>1      </td><td>-2.601 </td><td>-0.003 </td><td>0.044  </td><td>0.246  </td><td>megahit</td><td>1      </td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "\\begin{tabular}{r|lllllllll}\n",
       " contig & position & depth & ln\\_\\_depthLike & ln\\_\\_placeLike & ln\\_\\_insertLike & ln\\_\\_kmerLike & Assembler & Rep\\\\\n",
       "\\hline\n",
       "\t 0       & 0       & 1       & -2.601  & -0.003  & 0.044   & 0.001   & megahit & 1      \\\\\n",
       "\t 0       & 1       & 1       & -2.601  & -0.003  & 0.044   & 0.076   & megahit & 1      \\\\\n",
       "\t 0       & 2       & 1       & -2.601  & -0.003  & 0.044   & 0.246   & megahit & 1      \\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "contig | position | depth | ln__depthLike | ln__placeLike | ln__insertLike | ln__kmerLike | Assembler | Rep | \n",
       "|---|---|---|\n",
       "| 0       | 0       | 1       | -2.601  | -0.003  | 0.044   | 0.001   | megahit | 1       | \n",
       "| 0       | 1       | 1       | -2.601  | -0.003  | 0.044   | 0.076   | megahit | 1       | \n",
       "| 0       | 2       | 1       | -2.601  | -0.003  | 0.044   | 0.246   | megahit | 1       | \n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "  contig position depth ln__depthLike ln__placeLike ln__insertLike ln__kmerLike\n",
       "1 0      0        1     -2.601        -0.003        0.044          0.001       \n",
       "2 0      1        1     -2.601        -0.003        0.044          0.076       \n",
       "3 0      2        1     -2.601        -0.003        0.044          0.246       \n",
       "  Assembler Rep\n",
       "1 megahit   1  \n",
       "2 megahit   1  \n",
       "3 megahit   1  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "doParallel::registerDoParallel(threads)\n",
    "ALE_res = plyr::llply(as.list(ALE_res_files), read_ALE_files, .parallel=TRUE) %>%\n",
    "    do.call(rbind, .)\n",
    "\n",
    "ALE_res %>% nrow %>% print\n",
    "ALE_res %>% head(n=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Writing combined ALE results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T13:58:10.700283Z",
     "start_time": "2019-07-23T09:03:20.586Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File written: /ebio/abt3_projects/databases_no-backup/DeepMAsED/test_runs/n100_r25/ALE//ALE_results.tsv \n"
     ]
    }
   ],
   "source": [
    "out_file = file.path(work_dir, 'ALE_results.tsv')\n",
    "ALE_res %>%\n",
    "    write.table(file=out_file, sep='\\t', quote=FALSE, row.names=FALSE)\n",
    "cat('File written:', out_file, '\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# sessionInfo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-07-23T13:59:15.007040Z",
     "start_time": "2019-07-23T13:58:50.184Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "R version 3.4.1 (2017-06-30)\n",
       "Platform: x86_64-pc-linux-gnu (64-bit)\n",
       "Running under: Ubuntu 18.04.1 LTS\n",
       "\n",
       "Matrix products: default\n",
       "BLAS: /ebio/abt3_projects/software/miniconda3_gt4.4/envs/py3_physeq/lib/R/lib/libRblas.so\n",
       "LAPACK: /ebio/abt3_projects/software/miniconda3_gt4.4/envs/py3_physeq/lib/R/lib/libRlapack.so\n",
       "\n",
       "locale:\n",
       " [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              \n",
       " [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    \n",
       " [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   \n",
       " [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 \n",
       " [9] LC_ADDRESS=C               LC_TELEPHONE=C            \n",
       "[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       \n",
       "\n",
       "attached base packages:\n",
       "[1] parallel  stats     graphics  grDevices utils     datasets  methods  \n",
       "[8] base     \n",
       "\n",
       "other attached packages:\n",
       " [1] bindrcpp_0.2.2          future.apply_1.0.0      future.batchtools_0.7.1\n",
       " [4] future_1.9.0            doParallel_1.0.11       iterators_1.0.10       \n",
       " [7] foreach_1.4.4           data.table_1.11.4       ggplot2_3.0.0          \n",
       "[10] tidyr_0.8.1             dplyr_0.7.6            \n",
       "\n",
       "loaded via a namespace (and not attached):\n",
       " [1] pbdZMQ_0.3-2      progress_1.2.0    tidyselect_0.2.4  repr_0.15.0      \n",
       " [5] purrr_0.2.4       listenv_0.7.0     colorspace_1.3-2  htmltools_0.3.6  \n",
       " [9] base64enc_0.1-3   rlang_0.2.1       pillar_1.2.2      glue_1.3.0       \n",
       "[13] withr_2.1.1       rappdirs_0.3.1    uuid_0.1-2        bindr_0.1.1      \n",
       "[17] plyr_1.8.4        stringr_1.3.1     munsell_0.5.0     gtable_0.2.0     \n",
       "[21] codetools_0.2-15  evaluate_0.10.1   batchtools_0.9.11 IRdisplay_0.4.4  \n",
       "[25] Rcpp_0.12.17      scales_0.5.0      backports_1.1.2   checkmate_1.8.5  \n",
       "[29] IRkernel_0.8.12   jsonlite_1.5      fs_1.2.6          brew_1.0-6       \n",
       "[33] hms_0.4.2         digest_0.6.15     stringi_1.2.3     grid_3.4.1       \n",
       "[37] tools_3.4.1       magrittr_1.5      base64url_1.4     lazyeval_0.2.1   \n",
       "[41] tibble_1.4.2      crayon_1.3.4      pkgconfig_2.0.1   prettyunits_1.0.2\n",
       "[45] assertthat_0.2.0  R6_2.2.2          globals_0.12.2    compiler_3.4.1   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sessionInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R [conda env:py3_physeq]",
   "language": "R",
   "name": "conda-env-py3_physeq-r"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.4.1"
  },
  "notify_time": "30",
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": true,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
